<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="David R. Slochower" />
  <meta name="author" content="Niel M. Henriksen" />
  <meta name="author" content="Lee-Ping Wang" />
  <meta name="author" content="John D. Chodera" />
  <meta name="author" content="David L. Mobley" />
  <meta name="author" content="Michael K. Gilson" />
  <meta name="dcterms.date" content="2019-07-19" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>Binding thermodynamics of host-guest systems with SMIRNOFF99Frosst 1.0.5 from the Open Force Field Initiative</title>
  <style>
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Binding thermodynamics of host-guest systems with SMIRNOFF99Frosst 1.0.5 from the Open Force Field Initiative</h1>
</header>
<p><small><em> This manuscript (<a href="https://slochower.github.io/smirnoff-host-guest-manuscript/v/fbd19ff34930efed0625af0d9cf9eb84dca5d7d8/">permalink</a>) was automatically generated from <a href="https://github.com/slochower/smirnoff-host-guest-manuscript/tree/fbd19ff34930efed0625af0d9cf9eb84dca5d7d8">slochower/smirnoff-host-guest-manuscript@fbd19ff</a> on July 19, 2019. </em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>David R. Slochower</strong><br> <img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" /> <a href="https://orcid.org/0000-0003-3928-5050">0000-0003-3928-5050</a> · <img src="images/github.svg" class="inline_icon" alt="GitHub icon" /> <a href="https://github.com/slochower">slochower</a> · <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" /> <a href="https://twitter.com/drslochower">drslochower</a><br> <small> Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, La Jolla, CA 92093, USA </small></p></li>
<li><p><strong>Niel M. Henriksen</strong><br> <img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" /> <a href="https://orcid.org/0000-0002-7916-0757">0000-0002-7916-0757</a> · <img src="images/github.svg" class="inline_icon" alt="GitHub icon" /> <a href="https://github.com/nhenriksen">nhenriksen</a><br> <small> Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, La Jolla, CA 92093, USA </small></p></li>
<li><p><strong>Lee-Ping Wang</strong><br> <img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" /> <a href="https://orcid.org/0000-0003-3072-9946">0000-0003-3072-9946</a> · <img src="images/github.svg" class="inline_icon" alt="GitHub icon" /> <a href="https://github.com/leeping">leeping</a><br> <small> Department of Chemistry, University of California, Davis, CA 95616, USA </small></p></li>
<li><p><strong>John D. Chodera</strong><br> <img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" /> <a href="https://orcid.org/0000-0003-0542-119X">0000-0003-0542-119X</a> · <img src="images/github.svg" class="inline_icon" alt="GitHub icon" /> <a href="https://github.com/jchodera">jchodera</a> · <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" /> <a href="https://twitter.com/jchodera">jchodera</a><br> <small> Computational and Systems Biology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY 10065 </small></p></li>
<li><p><strong>David L. Mobley</strong><br> <img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" /> <a href="https://orcid.org/0000-0002-1083-5533">0000-0002-1083-5533</a> · <img src="images/github.svg" class="inline_icon" alt="GitHub icon" /> <a href="https://github.com/davidlmobley">davidlmobley</a> · <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" /> <a href="https://twitter.com/davidlmobley">davidlmobley</a><br> <small> Department of Pharmaceutical Sciences and Department of Chemistry, University of California, Irvine, CA 92697, USA </small></p></li>
<li><p><strong>Michael K. Gilson</strong><br> <img src="images/orcid.svg" class="inline_icon" alt="ORCID icon" /> <a href="https://orcid.org/0000-0002-3375-1738">0000-0002-3375-1738</a> · <img src="images/twitter.svg" class="inline_icon" alt="Twitter icon" /> <a href="https://twitter.com/michaelkgilson">michaelkgilson</a><br> <small> Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, La Jolla, CA 92093, USA </small></p></li>
</ul>
<h2 id="outstanding-issues">Outstanding Issues</h2>
<ul>
<li><a href="https://github.com/slochower/smirnoff-host-guest-manuscript/issues/42">Update tables with CSV link and SMILES #42</a></li>
<li><a href="https://github.com/slochower/smirnoff-host-guest-manuscript/issues/46">Update figure legends and labels with additional information #46</a></li>
<li><a href="https://github.com/slochower/smirnoff-host-guest-manuscript/issues/47">Update GAFF v2.1 statistics in the text #47</a></li>
<li><a href="https://github.com/slochower/smirnoff-host-guest-manuscript/issues/49">Consider adding discussion of Kendall’s tau #49</a></li>
<li><a href="https://github.com/slochower/smirnoff-host-guest-manuscript/issues/50">Update SI figure captions with letter numbering #50</a></li>
<li><a href="https://github.com/slochower/smirnoff-host-guest-manuscript/issues/51">Refine language around “fewer parameters” claims #51</a></li>
<li><a href="https://github.com/slochower/smirnoff-host-guest-manuscript/issues/56">Cleanup table headers #56</a></li>
</ul>
<h2 id="abstract" class="page_break_before">Abstract</h2>
<p>Designing ligands that bind their target biomolecules with high affinity and specificity is a key step in small-molecule drug discovery, but accurately predicting protein-ligand binding free energies remains challenging. Key sources of errors in the calculations include inadequate sampling of conformational space, ambiguous protonation states, and errors in force fields. Noncovalent complexes between a host molecule with a binding cavity and a drug-like guest molecules have emerged as powerful model systems. As model systems, host-guest complexes reduce many of the errors in more complex protein-ligand binding systems, as their small size greatly facilitates conformational sampling, and one can choose systems that avoid ambiguities in protonation states. These features, combined with their ease of experimental characterization, make host-guest systems ideal model systems to test and ultimately optimize force fields in the context of binding thermodynamics calculations.</p>
<p>The Open Force Field Initiative aims to create a modern, open software infrastructure for automatically generating and assessing force fields using data sets. The first force field to arise out of this effort, named SMIRNOFF99Frosst, has approximately one tenth the number of parameters, in version 1.0.5, compared to typical general small molecule force fields, such as GAFF. Here, we evaluate the accuracy of this initial force field, using free energy calculations of 43 α and β-cyclodextrin host-guest pairs for which experimental thermodynamic data are available, and compare with matched calculations using two versions of GAFF. For all three force fields, we used TIP3P water and AM1-BCC charges. The calculations are performed using the attach-pull-release (APR) method as implemented in the open source package, pAPRika. For binding free energies, the root mean square error of the SMIRNOFF99Frosst calculations relative to experiment is 0.9 [0.7, 1.1] kcal/mol, while the corresponding results for GAFF 1.7 and GAFF 2.1 are 0.9 [0.7, 1.1] kcal/mol and 1.7 [1.5, 1.9] kcal/mol, respectively, with 95% confidence ranges in brackets. These results suggest that SMIRNOFF99Frosst performs competitively with existing small molecule force fields and is a parsimonious starting point for optimization.</p>
<h2 id="introduction">Introduction</h2>
<p>The accurate prediction of protein-ligand binding free energies is a key goal of computational chemistry, with key applications in early stage drug discovery. However, calculations of protein-ligand binding thermodynamics still involve a number of challenging choices, including the choice of empirical force field, specifying the protonation states of ionizable residues, adding hydrogens and otherwise adjusting the initial protein structure, and positioning the candidate ligand in the binding pocket. Predictions of protein-ligand absolute binding free energies have achieved root mean square errors around 1-2 kcal/mol for “well-behaved” systems <span class="citation" data-cites="9MLPuYAQ 1CuDE1c2r mdqdy96N">[<a href="#ref-9MLPuYAQ" role="doc-biblioref">1</a>,<a href="#ref-1CuDE1c2r" role="doc-biblioref">2</a>,<a href="#ref-mdqdy96N" role="doc-biblioref">3</a>]</span>, with deviations an order of magnitude larger for some protein families with slow degrees of freedom <span class="citation" data-cites="15iR76Uc">[<a href="#ref-15iR76Uc" role="doc-biblioref">4</a>]</span>. Retrospective relative free energy calculations on a series of congeneric ligands, using proprietary methods, have also achieved root mean square errors compared to experiment of around 1 kcal/mol <span class="citation" data-cites="HQi6ihVB B182mNck 166JLvkLU">[<a href="#ref-HQi6ihVB" role="doc-biblioref">5</a>,<a href="#ref-B182mNck" role="doc-biblioref">6</a>,<a href="#ref-166JLvkLU" role="doc-biblioref">7</a>]</span>. However, it is not possible to determine how much of the prediction error can be attributed to each of the decisions made by the modeler, as opposed to accuracy limitations of the force field.</p>
<p>By minimizing the ambiguities involved in modeling protein-ligand complexes, host-guest systems offer a way to isolate and directly probe force field error. A variety of techniques for computing absolute binding free energies in host-guest systems have shown accuracy of ~1 kcal/mol, as highlighted in the recent SAMPL5 and SAMPL6 blind challenges <span class="citation" data-cites="BGsUYQln 9MLPuYAQ">[<a href="#ref-9MLPuYAQ" role="doc-biblioref">1</a>,<a href="#ref-BGsUYQln" role="doc-biblioref">8</a>]</span>. These techniques include both quantum and classical dynamics, employing a range of energy and solvation models, with some techniques having knowledge-based steps, docking, or clustering <span class="citation" data-cites="rOVoXhRJ Dn1HX5lD kj7fJ3fg l8KUmyk3 1935a9V0d ku7PPPs ScmrI810 bKuLAjgi">[<a href="#ref-rOVoXhRJ" role="doc-biblioref">9</a>,<a href="#ref-Dn1HX5lD" role="doc-biblioref">10</a>,<a href="#ref-kj7fJ3fg" role="doc-biblioref">11</a>,<a href="#ref-l8KUmyk3" role="doc-biblioref">12</a>,<a href="#ref-1935a9V0d" role="doc-biblioref">13</a>,<a href="#ref-ku7PPPs" role="doc-biblioref">14</a>,<a href="#ref-ScmrI810" role="doc-biblioref">15</a>,<a href="#ref-bKuLAjgi" role="doc-biblioref">16</a>]</span>. The attach-pull-release (APR) method has consistently been ranked among the most reliable techniques for predicting binding thermodynamics of host-guest complexes in blind challenges <span class="citation" data-cites="BGsUYQln GA1AFcUw">[<a href="#ref-BGsUYQln" role="doc-biblioref">8</a>,<a href="#ref-GA1AFcUw" role="doc-biblioref">17</a>]</span>. In APR, the reversible work of transferring the guest from the binding site to solution, via a physical pathway, is computed using a series of umbrella sampling windows. Simulating each window and integrating over the partial derivative of the restraint energy with respect to the restraint target, in each window, is used to generate a potential of mean force along the pulling coordinate, yielding the binding free energy at standard state, ΔG° after applying an analytic correction to account for the effective concentration of the guest during the simulation <span class="citation" data-cites="li8h8Q8W">[<a href="#ref-li8h8Q8W" role="doc-biblioref">18</a>]</span>. Furthermore, subtracting the mean potential energies obtained from long simulations of the solvated bound complex and the solvated dissociated complex yields the binding enthalpy, ΔH <span class="citation" data-cites="1HWPx2yMR">[<a href="#ref-1HWPx2yMR" role="doc-biblioref">19</a>]</span>. Together, ΔG° and ΔH can be combined to determine the binding entropy at standard state, ΔS°. Thus, APR provides the complete thermodynamic signature of a host-guest binding reaction: ΔG°, ΔH, and −TΔS°.</p>
<p>Cyclodextrins, in particular, are ideal host molecules for testing computational methods. They are neutral across a broad pH range, with well-characterized structures <span class="citation" data-cites="l02WNlWU">[<a href="#ref-l02WNlWU" role="doc-biblioref">20</a>]</span>, and bind both small molecule fragments and drug-like guest molecules with reasonable affinity, from near -1 kcal/mol to about -5 kcal/mol in the present work <span class="citation" data-cites="q7znTABh">[<a href="#ref-q7znTABh" role="doc-biblioref">21</a>]</span>, and with higher affinity for some cyclodextrin derivatives <span class="citation" data-cites="q7znTABh">[<a href="#ref-q7znTABh" role="doc-biblioref">21</a>]</span>. Moreover, cyclodextrins are stable in a wide range of experimental conditions and their high millimolar aqueous solubility allows a range of different experimental techniques to be used to measure their binding to guests <span class="citation" data-cites="19cBeLx8d">[<a href="#ref-19cBeLx8d" role="doc-biblioref">22</a>]</span>. Here, we report the calculation of binding free energies, enthalpies, and entropies of small guest molecules with functional groups often found in drugs to α- and β-cyclodextrin host molecules, converged to within 0.1 kcal/mol statistical uncertainty, using the APR method. These calculations offer an opportunity to benchmark—and ultimately optimize—new and existing force fields.</p>
<p>The first force field produced by the Open Force Field Initiative, SMIRNOFF99Frosst v1.0.5, was released in late 2018 <span class="citation" data-cites="1HYTTY1PU OhpH7vfg">[<a href="#ref-1HYTTY1PU" role="doc-biblioref">23</a>,<a href="#ref-OhpH7vfg" role="doc-biblioref">24</a>]</span>. It is derived from AMBER parm99 <span class="citation" data-cites="13wrQoS3l">[<a href="#ref-13wrQoS3l" role="doc-biblioref">25</a>]</span> and Merck’s parm@Frosst <span class="citation" data-cites="168lWg0SB">[<a href="#ref-168lWg0SB" role="doc-biblioref">26</a>]</span>. Instead of relying on atom types to assign force field parameters to compounds, which is the procedure followed by the <code>LEaP</code> program used to assign parameters to molecules in AmberTools <span class="citation" data-cites="ltrN7ofw">[<a href="#ref-ltrN7ofw" role="doc-biblioref">27</a>]</span>, SMIRNOFF99Frosst and the Open Force Field Toolkit use separately defined local chemical environments for each atom, bond, angle, and dihedral, to apply force field parameters specified by SMIRKS strings <span class="citation" data-cites="pOsXFMux">[<a href="#ref-pOsXFMux" role="doc-biblioref">28</a>]</span>. This process simplifies and effectively uncouples the parameters for each term in the force field. For example, the addition of a new Lennard-Jones parameter does not require creating a new atom type that forces the addition of new bonded, angle, and dihedral parameters. This approach leads to a much leaner force field specification; there are over 3000 lines of parameters in GAFF v1.7 <span class="citation" data-cites="YmRgHfeU">[<a href="#ref-YmRgHfeU" role="doc-biblioref">29</a>]</span>, over 6000 lines of parameters in GAFF v2.1, and just 322 lines of parameters in SMIRNOFF99Frosst v1.0.5 <span class="citation" data-cites="4LS7HWPh">[<a href="#ref-4LS7HWPh" role="doc-biblioref">30</a>]</span>. It is important to note that SMIRNOFF99Frosst is not yet optimized at this stage, only compressed; subsequent work will focus on optimizing SMIRNOFF99Frosst and other SMIRNOFF-family force fields to fit quantum and experimental data <span class="citation" data-cites="XzEPM0UV">[<a href="#ref-XzEPM0UV" role="doc-biblioref">31</a>]</span>. In the following text, SMIRNOFF99Frosst refers to version 1.0.5 of the force field, unless otherwise noted.</p>
<p>Thus far, SMIRNOFF99Frosst has been tested on hydration free energies of 642 small molecules and the densities and dielectric constants of 45 pure organic liquids <span class="citation" data-cites="1HYTTY1PU">[<a href="#ref-1HYTTY1PU" role="doc-biblioref">23</a>]</span>. Here, we benchmark SMIRNOFF99Frosst, GAFF v1.7, and GAFF v2.1 using noncovalent binding thermodynamics for 43 host-guest complexes (including two hosts and 33 unique guests) for which experimental thermodynamics data are available, representing three different functional group moieties. We first compare the results of SMIRNOFF99Frosst with those of the conventional force fields GAFF v1.7 and GAFF v2.1, based on calculations of experimental binding free energies, enthalpies, and entropies. We then characterize the differences in host conformations sampled by SMIRNOFF99Frosst compared to the other two force fields.</p>
<h2 id="methods">Methods</h2>
<h3 id="choice-of-host-guest-systems">Choice of host-guest systems</h3>
<p>In this study, we report the binding thermodynamics of 43 host-guest complexes (Figure <a href="#fig:host-guest-pairs">1</a> and Table <a href="#tbl:host-guests">1</a>) computed using three different force fields. The complexes consist of either α- or β-cyclodextrin as host molecules and a series of small molecule guests containing ammonium, carboxylate, or cyclic alcohol functional groups. The cyclodextrins in the current study are cyclic polymers consisting of six (αCD) or seven (βCD) glucose monomers in the shape of a truncated cone. The equilibrium constants and standard molar enthalpies of binding for these 43 complexes have been measured using isothermal titration calorimetry (ITC) at pH = 6.90 and T = 298 K, and nuclear magnetic resonance spectroscopy (NMR) at pH = 7.0 and T = 298 ± 1 K <span class="citation" data-cites="1236RpaUv">[<a href="#ref-1236RpaUv" role="doc-biblioref">32</a>]</span>. Calculations on these host-guest systems have been performed previously <span class="citation" data-cites="HVgz5rZq">[<a href="#ref-HVgz5rZq" role="doc-biblioref">33</a>]</span>, and, as in the prior study, we considered only a single stereoisomer for the 1-methylammonium guests because it was not clear whether a mixture or a pure solution was used in Rekharsky, et al. <span class="citation" data-cites="1236RpaUv">[<a href="#ref-1236RpaUv" role="doc-biblioref">32</a>]</span>, and the ΔG° difference between each stereoisomer is expected to be &lt; 0.1 kcal/mol <span class="citation" data-cites="g7xxBB7m">[<a href="#ref-g7xxBB7m" role="doc-biblioref">34</a>]</span>.</p>
<figure>
<img src="images/host-guest-pairs.png" id="fig:host-guest-pairs" alt="" /><figcaption><span>Figure 1:</span> Structures of the two cyclodextrin hosts and 33 guest molecules in this study which together comprise 43 unique host-guest pairs. The simulation “residue name” is written beneath each guest.</figcaption>
</figure>
<a name="tbl:host-guests"></a>
<table>
<caption><span>Table 1:</span> The 43 unique host-guest combinations used in this study. The formal charge of each guest is listed in brackets. The guest names correspond to Tables 1 and 2 in Rekharsky et al. <span class="citation" data-cites="1236RpaUv">[<a href="#ref-1236RpaUv" role="doc-biblioref">32</a>]</span>. <sup>a</sup> Only the <em>R</em> enantiomer was considered. <sup>b</sup> Only the <em>S</em> enantiomer was considered. </caption>
<colgroup>
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
<col style="width: 20%" />
</colgroup>
<thead>
<tr class="header">
<th>Host-guest ID</th>
<th>Host</th>
<th>Guest</th>
<th>Charge</th>
<th>SMILES</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>a-bam</td>
<td>αCD</td>
<td>1-butylamine</td>
<td>+1</td>
<td><!-- style="min-width: 500px" --> <code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="even">
<td>a-nmb</td>
<td>αCD</td>
<td>n-methylbutylamine</td>
<td>+1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])[N+]([H])([H])C([H])([H])[H]</code></td>
</tr>
<tr class="odd">
<td>a-mba</td>
<td>αCD</td>
<td>1-methylbutylamine<sup>a</sup></td>
<td>+1</td>
<td><code>[H][C@@](C([H])([H])[H])(C([H])([H])C([H])([H])C([H])([H])[H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="even">
<td>a-pam</td>
<td>αCD</td>
<td>1-pentylamine</td>
<td>+1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="odd">
<td>a-ham</td>
<td>αCD</td>
<td>1-hexylamine</td>
<td>+1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="even">
<td>a-nmh</td>
<td>αCD</td>
<td>n-methylhexylamine</td>
<td>+1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[N+]([H])([H])C([H])([H])[H]</code></td>
</tr>
<tr class="odd">
<td>a-mha</td>
<td>αCD</td>
<td>1-methylhexylamine<sup>a</sup></td>
<td>+1</td>
<td><code>[H][C@@](C([H])([H])[H])(C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="even">
<td>a-hpa</td>
<td>αCD</td>
<td>1-heptylamine</td>
<td>+1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="odd">
<td>a-mhp</td>
<td>αCD</td>
<td>1-methylheptylamine<sup>b</sup></td>
<td>+1</td>
<td><code>[H][C@](C([H])([H])[H])(C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="even">
<td>a-oam</td>
<td>αCD</td>
<td>1-octylamine</td>
<td>+1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="odd">
<td>b-ham</td>
<td>βCD</td>
<td>1-hexylamine</td>
<td>+1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="even">
<td>b-mha</td>
<td>βCD</td>
<td>1-methylhexylamine<sup>a</sup></td>
<td>+1</td>
<td><code>[H][C@@](C([H])([H])[H])(C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="odd">
<td>b-oam</td>
<td>βCD</td>
<td>1-octylamine</td>
<td>+1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[N+]([H])([H])[H]</code></td>
</tr>
<tr class="even">
<td>a-cbu</td>
<td>αCD</td>
<td>cyclobutanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C1([H])[H])([H])O[H])([H])[H])[H]</code></td>
</tr>
<tr class="odd">
<td>a-cpe</td>
<td>αCD</td>
<td>cyclopentanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C(C1([H])[H])([H])O[H])([H])[H])([H])[H])[H]</code></td>
</tr>
<tr class="even">
<td>a-chp</td>
<td>αCD</td>
<td>cycloheptanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C(C(C(C1([H])[H])([H])[H])([H])O[H])([H])[H])([H])[H])([H])[H])[H]</code></td>
</tr>
<tr class="odd">
<td>a-coc</td>
<td>αCD</td>
<td>cyclooctanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C(C(C(C(C1([H])[H])([H])[H])([H])[H])([H])O[H])([H])[H])([H])[H])([H])[H])[H]</code></td>
</tr>
<tr class="even">
<td>b-cbu</td>
<td>βCD</td>
<td>cyclobutanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C1([H])[H])([H])O[H])([H])[H])[H]</code></td>
</tr>
<tr class="odd">
<td>b-cpe</td>
<td>βCD</td>
<td>cyclopentanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C(C1([H])[H])([H])O[H])([H])[H])([H])[H])[H]</code></td>
</tr>
<tr class="even">
<td>b-mch</td>
<td>βCD</td>
<td>1-methylcyclohexanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C(C(C1([H])[H])([H])[H])(C([H])([H])[H])O[H])([H])[H])([H])[H])[H]</code></td>
</tr>
<tr class="odd">
<td>b-m4c</td>
<td>βCD</td>
<td>cis-4-methylcyclohexanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C(C(C1([H])C([H])([H])[H])([H])[H])([H])[H])([H])O[H])([H])[H])[H]</code></td>
</tr>
<tr class="even">
<td>b-m4t</td>
<td>βCD</td>
<td>trans-4-methylcyclohexanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C(C(C1([H])C([H])([H])[H])([H])[H])([H])[H])([H])O[H])([H])[H])[H]</code></td>
</tr>
<tr class="odd">
<td>b-chp</td>
<td>βCD</td>
<td>cycloheptanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C(C(C(C1([H])[H])([H])[H])([H])O[H])([H])[H])([H])[H])([H])[H])[H]</code></td>
</tr>
<tr class="even">
<td>b-coc</td>
<td>βCD</td>
<td>cyclooctanol</td>
<td>0</td>
<td><code>[H]C1(C(C(C(C(C(C(C1([H])[H])([H])[H])([H])[H])([H])O[H])([H])[H])([H])[H])([H])[H])[H]</code></td>
</tr>
<tr class="odd">
<td>a-but</td>
<td>αCD</td>
<td>butanoate</td>
<td>-1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-]</code></td>
</tr>
<tr class="even">
<td>a-pnt</td>
<td>αCD</td>
<td>pentanoate</td>
<td>-1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-]</code></td>
</tr>
<tr class="odd">
<td>a-hex</td>
<td>αCD</td>
<td>hexanoate</td>
<td>-1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-]</code></td>
</tr>
<tr class="even">
<td>a-hx2</td>
<td>αCD</td>
<td>trans-2-hexenoate</td>
<td>-1</td>
<td><code>[H]/C(=C(/[H])\\C([H])([H])C([H])([H])C([H])([H])[H])/[C-]([O-])[O-]</code></td>
</tr>
<tr class="odd">
<td>a-hx3</td>
<td>αCD</td>
<td>trans-3-hexenoate</td>
<td>-1</td>
<td><code>[H]/C(=C(/[H])\\C([H])([H])C([H])([H])[H])/C([H])([H])[C-]([O-])[O-]</code></td>
</tr>
<tr class="even">
<td>a-hep</td>
<td>αCD</td>
<td>heptanoate</td>
<td>-1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-]</code></td>
</tr>
<tr class="odd">
<td>a-hp6</td>
<td>αCD</td>
<td>6-heptenoate</td>
<td>-1</td>
<td><code>[H]C(=C([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-])[H]</code></td>
</tr>
<tr class="even">
<td>a-oct</td>
<td>αCD</td>
<td>octanoate</td>
<td>-1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-]</code></td>
</tr>
<tr class="odd">
<td>b-pnt</td>
<td>βCD</td>
<td>pentanoate</td>
<td>-1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-]</code></td>
</tr>
<tr class="even">
<td>b-hex</td>
<td>βCD</td>
<td>hexanoate</td>
<td>-1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-]</code></td>
</tr>
<tr class="odd">
<td>b-hep</td>
<td>βCD</td>
<td>heptanoate</td>
<td>-1</td>
<td><code>[H]C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-]</code></td>
</tr>
<tr class="even">
<td>b-ben</td>
<td>βCD</td>
<td>benzoate</td>
<td>-1</td>
<td><code>[H]c1c(c(c(c(c1[H])[H])[C-]([O-])[O-])[H])[H]</code></td>
</tr>
<tr class="odd">
<td>b-pha</td>
<td>βCD</td>
<td>phenylacetate</td>
<td>-1</td>
<td><code>[H]c1c(c(c(c(c1[H])[H])C([H])([H])[C-]([O-])[O-])[H])[H]</code></td>
</tr>
<tr class="even">
<td>b-mp3</td>
<td>βCD</td>
<td>3-methylphenylacetate</td>
<td>-1</td>
<td><code>[H]c1c(c(c(c(c1[H])C([H])([H])[C-]([O-])[O-])[H])C([H])([H])[H])[H]</code></td>
</tr>
<tr class="odd">
<td>b-mp4</td>
<td>βCD</td>
<td>4-methylphenylacetate</td>
<td>-1</td>
<td><code>[H]c1c(c(c(c(c1C([H])([H])[H])[H])[H])C([H])([H])[C-]([O-])[O-])[H]</code></td>
</tr>
<tr class="even">
<td>b-mo3</td>
<td>βCD</td>
<td>3-methoxyphenylacetate</td>
<td>-1</td>
<td><code>[H]c1c(c(c(c(c1[H])OC([H])([H])[H])[H])C([H])([H])[C-]([O-])[O-])[H]</code></td>
</tr>
<tr class="odd">
<td>b-mo4</td>
<td>βCD</td>
<td>4-methoxyphenylacetate</td>
<td>-1</td>
<td><code>[H]c1c(c(c(c(c1C([H])([H])[C-]([O-])[O-])[H])[H])OC([H])([H])[H])[H]</code></td>
</tr>
<tr class="even">
<td>b-pb3</td>
<td>βCD</td>
<td>3-phenylbutanoate</td>
<td>-1</td>
<td><code>[H]c1c(c(c(c(c1[H])[H])[C@]([H])(C([H])([H])[H])C([H])([H])[C-]([O-])[O-])[H])[H]</code></td>
</tr>
<tr class="odd">
<td>b-pb4</td>
<td>βCD</td>
<td>4-phenylbutanoate</td>
<td>-1</td>
<td><code>[H]c1c(c(c(c(c1[H])[H])C([H])([H])C([H])([H])C([H])([H])[C-]([O-])[O-])[H])[H]</code></td>
</tr>
</tbody>
</table>
<h3 id="application-of-force-field-parameters">Application of force field parameters</h3>
<p>We sought to compare force fields directly and therefore attempted to minimize additional differences between the simulations with each force field. In all simulations, we applied AM1-BCC <span class="citation" data-cites="LxrgIkt0 BkLoOd0N">[<a href="#ref-LxrgIkt0" role="doc-biblioref">35</a>,<a href="#ref-BkLoOd0N" role="doc-biblioref">36</a>]</span> partial atomic charges to both the host and guest molecules using the <code>antechamber</code> program in AmberTools16 <span class="citation" data-cites="ltrN7ofw">[<a href="#ref-ltrN7ofw" role="doc-biblioref">27</a>]</span>. The host charges were calculated using a single glucose molecule with methoxy caps on the O1 and O4 alcohols (Figure <a href="#fig:atom-names">2</a>); each glucose monomer in the cyclodextrin polymer has identical charges. After removing the capping atoms, the net charge of the glucose monomer was -0.064 e. To ensure a neutrality of the glucose monomer, the charge remainder was proportionally distributed across all atoms according to the magnitude of the partial charge for each atom. The minimum and maximum charge adjustments were 0.000684 and 0.007245 e, respectively. Using the entire αCD molecule as an input to <code>antechamber</code> results in partial atomic charges that differ by at most 0.02 e, compared to using a single monomer, and requires reducing the maximum path length used to determine the equivalence of atomic charges (Figure <a href="#fig:charge-schemes">16</a>). We used TIP3P water <span class="citation" data-cites="chrX4HGh">[<a href="#ref-chrX4HGh" role="doc-biblioref">37</a>]</span> and Joung-Cheatham monovalent ion parameters <span class="citation" data-cites="uZnQBwCt">[<a href="#ref-uZnQBwCt" role="doc-biblioref">38</a>]</span> in each simulation set.</p>
<figure>
<img src="images/atom-names.png" id="fig:atom-names" style="width:100.0%" alt="" /><figcaption><span>Figure 2:</span> Atom names (A) and GAFF atom types (B) for a glucose monomer in αCD shown with two flanking monomers. The remaining three glucose monomers are hidden for clarity.</figcaption>
</figure>
<p>GAFF v1.7 bond, angle, torsion, and Lennard-Jones parameters were applied using the <code>tleap</code> program distributed with AmberTools16. GAFF v2.1 parameters were applied in an identical manner to the GAFF v1.7 parameters, using the <code>tleap</code> program distributed with AmberTools18 and substituting <code>leaprc.gaff</code> for <code>leaprc.gaff2</code> in the <code>tleap</code> input file.</p>
<p>To apply SMIRNOFF99Frosst parameters, we followed a <a href="https://github.com/slochower/smirnoff-host-guest/blob/master/02-convert-APR-files.ipynb">multistep process</a>, beginning with the AMBER-format <code>.prmtop</code> and <code>.inpcrd</code> GAFF v1.7 files. The host and guest molecules were parameterized with version 0.0.3 of the Open Force Field Toolkit, which reads molecular coordinates and topologies and creates a serialized representation of the molecular system; version 1.0.5 of the SMIRNOFF99Frosst force field; specified in version 1.0 of the SMIRNOFF format. Once parameterized with SMIRNOFF99Frosst, the topology and coordinates for the host-guest complex were combined with the solvent and ions, which retained their TIP3P water parameters and Joung-Cheatham ion parameters, respectively. This was accomplished by the <code>ParmEd</code> program <span class="citation" data-cites="T7iFp9ei">[<a href="#ref-T7iFp9ei" role="doc-biblioref">39</a>]</span>, which enables saving the OpenMM system created by the Open Force Field Toolkit in AMBER-format <code>.prmtop</code> and <code>.inpcrd</code> files. Ongoing updates to the Open Force Field Toolkit may result in changes to how this procedure is carried out in the future.</p>
<h3 id="thermodynamic-calculation">Thermodynamic calculation</h3>
<p>We used the attach-pull-release (APR) method, as implemented in the open source package pAPRika version 0.0.3, to calculate absolute binding free energies. A complete description of the APR method has been provided in the literature <span class="citation" data-cites="1935a9V0d W9FLyOZr GA1AFcUw 1HWPx2yMR">[<a href="#ref-1935a9V0d" role="doc-biblioref">13</a>,<a href="#ref-GA1AFcUw" role="doc-biblioref">17</a>,<a href="#ref-1HWPx2yMR" role="doc-biblioref">19</a>,<a href="#ref-W9FLyOZr" role="doc-biblioref">40</a>]</span>. The attachment and release phases consisted of 15 independent windows. During the attachment phase, the force constants on the host and guest are scaled by a <span class="math inline">\(\lambda\)</span> parameter that goes from <span class="math inline">\(\lambda = 0\)</span>, at which point all restraints are turned off, to <span class="math inline">\(\lambda = 1\)</span>, at which point all restraints are at their maximum force constant. The <span class="math inline">\(\lambda\)</span> windows are more densely spaced where the force constant is smaller to improve sampling along highly curved regions of the potential of mean force. These restraints include a set of distance, angle, and torsion restraints that orient the host and guest along the long axis of the simulation box. A separate set of conformational restraints were applied between neighboring glucose units of the cyclodextrin to minimize deformations of the host molecule as the guest molecule is pulled out. The conformational restraints were applied along the pseudodihedrals O5<sub>n</sub>–C1<sub>n</sub>–C1<sub>n</sub>–C4<sub>n+1</sub> and C1<sub>n</sub>–O1<sub>n</sub>–C4<sub>n+1</sub>–C5<sub>n+1</sub> to improve convergence and sampling of the bound state (Figure <a href="#fig:atom-names">2</a> for atom names). To further improve sampling of weak-binding guests, we applied a hard wall restraint that confined the guest molecule to within a sphere of 12.3 and 13.5 Å of αCD and βCD, respectively, during the bound state.</p>
<p>The release phase is the conceptual reverse of the attach phase, in which the conformational restraints on the host are gradually turned off (<span class="math inline">\(\lambda =1 \rightarrow 0\)</span>) in the absence of the guest. This explicit release phase is performed once for αCD and once for βCD, as it is independent of guest molecule. Finally, an analytic correction is performed to compute the work of moving the guest from the restricted volume enforced by the APR restraints to standard state at 1 M concentration.</p>
<p>The pulling phase consisted of 45 independent, equally spaced windows. During the pulling phase, the <span class="math inline">\(\lambda\)</span> parameter represents the target value of a distance restraint with constant force constant. This target distance is increased uniformly in 44 increments of 0.4 Å, yielding windows that separate the host and guest by 18 Å over the course of the calculation.</p>
<p>Due to the asymmetry of the primary and secondary alcohols of cyclodextrin (Figure <a href="#fig:cavity">18</a>), and the lack of symmetry of the small molecule guests, there are generally two distinct binding poses that do not interconvert during the simulation timescale. To account for this effect, we separately compute the binding free energy and enthalpy for each orientation <span class="citation" data-cites="1935a9V0d">[<a href="#ref-1935a9V0d" role="doc-biblioref">13</a>]</span> and combine the results to produce a single value for each host-guest combination using the following equation:</p>
<p><span class="math display">\[\begin{equation}
\Delta G^\circ = -RT \ln(\exp(-\beta \Delta G_\text{primary}^\circ) + \exp(-\beta \Delta G_\text{secondary}^\circ)).
\end{equation}\]</span></p>
<p>The total binding enthalpy is weighted by both the binding enthalpy and binding free energy in each orientation using the following equation:</p>
<p><span class="math display">\[\begin{equation}
\Delta H = \frac{ \Delta H_\text{primary} \exp(-\beta \Delta G_\text{primary}) + \Delta H_\text{secondary} \exp(-\beta \Delta G_\text{secondary})}{\exp(-\beta \Delta G_\text{primary}) + \exp(-\beta \Delta G_\text{secondary})}.
\end{equation}\]</span></p>
<p>In this manuscript, we refer to calculations where the guest functional group in the bound state is at the primary face of cyclodextrin with a <code>-p</code> suffix, and calculations where it is at the secondary face of cyclodextrin with a <code>-s</code> suffix..</p>
<p>Thermodynamic integration <span class="citation" data-cites="79xe5F4u">[<a href="#ref-79xe5F4u" role="doc-biblioref">41</a>]</span> and the multistate Bennett acceptance ratio estimator (MBAR) <span class="citation" data-cites="x2mTWVO">[<a href="#ref-x2mTWVO" role="doc-biblioref">42</a>]</span> were used to compute the binding free energy (ΔG°). The results presented in the main text were analyzed using thermodynamic integration to be consistent with prior analysis presented in Henriksen, <em>et al.</em> <span class="citation" data-cites="HVgz5rZq">[<a href="#ref-HVgz5rZq" role="doc-biblioref">33</a>]</span>. The binding enthalpy (ΔH) was computed as the difference in mean potential energy of the bound state (in the absence of any restraints) and the unbound state (where the guest is held far away from the host, but the conformational restraints on the host are disabled). The binding entropy (ΔS°) was computed by subtraction using ΔG° and ΔH.</p>
<h3 id="simulations">Simulations</h3>
<p>Simulations were performed with the <code>pmemd.cuda</code> module of AMBER 16 (calculations with the GAFF v1.7 force field) and AMBER 18 (calculations with the GAFF v2.1 and SMIRNOFF99Frosst force fields) molecular dynamics software <span class="citation" data-cites="tKJug83o ltrN7ofw">[<a href="#ref-ltrN7ofw" role="doc-biblioref">27</a>,<a href="#ref-tKJug83o" role="doc-biblioref">43</a>]</span>. Each window for each system was independently solvated and simulated. Simulation data for the host-guest complexes using GAFF v1.7 were taken from Henriksen, <em>et al.</em> <span class="citation" data-cites="HVgz5rZq">[<a href="#ref-HVgz5rZq" role="doc-biblioref">33</a>]</span> and are described in additional detail therein. Solvation consisted of 2000 TIP3P waters for the αCD systems and 2210 waters for the βCD systems in an orthorhombic box. The host and guest were oriented via non-interacting dummy atoms along the simulation box’s long <span class="math inline">\(z\)</span> axis, to allow use of an elongated periodic box that reduces the amount of solvent required for the calculation. Each simulation contained enough Na<sup>+</sup> or Cl<sup>-</sup> ions to neutralize the host-guest complex and an additional 50 mM NaCl to match the experimental conditions in <span class="citation" data-cites="1236RpaUv">[<a href="#ref-1236RpaUv" role="doc-biblioref">32</a>]</span>. In the GAFF simulations, hydrogen mass repartitioning <span class="citation" data-cites="IGZ99oJ8">[<a href="#ref-IGZ99oJ8" role="doc-biblioref">44</a>]</span> was used to adjust the mass of hydrogen atoms by a factor of 3 and decreasing the mass of the bound heavy atoms proportionally, keeping the total molecular weight of each molecule constant and enabling a simulation timestep of 4 fs. Hydrogen mass repartitioning produces excellent agreement on thermodynamic observables for other cyclodextrin-guest calculations, with deviations within statistical uncertainty <span class="citation" data-cites="1935a9V0d">[<a href="#ref-1935a9V0d" role="doc-biblioref">13</a>]</span>. Equilibration consisted of 50,000 steps of energy minimization, 100 ps of heating from 0 to 300 K, and then 2000 ps of additional NPT simulation. AMBER’s Langevin thermostat with a collision rate of 1 ps<sup>-1</sup>, the Monte Carlo barostat, a nonbonded cutoff of 9 Å and default PME parameters, were used for the NPT simulations. An isotropic analytic correction to the Lennard-Jones interactions is applied beyond the cutoff distance <span class="citation" data-cites="sMVo0eOR">[<a href="#ref-sMVo0eOR" role="doc-biblioref">45</a>]</span>. Production NPT simulations were run for a minimum of 2.5 ns and maximum of 50 ns per window, except for the windows used to calculate the enthalpy, which were each simulated for 1 μs. In the GAFF v1.7 and GAFF v2.1 simulations, the exact length of each window’s simulation was determined by the uncertainty in the work done in each λ window. In particular, for restraint energy <span class="math inline">\(U\)</span> in <span class="math inline">\(\lambda\)</span> window <span class="math inline">\(i\)</span>, we define the instantaneous SEM of <span class="math inline">\(\partial U/\partial λ_i\)</span> as σ(λ<sub>i</sub>), and each window (except for the windows used to calculate ΔH) was simulated until the value of <span class="math inline">\(w(\lambda_i)\)</span>, defined as</p>
<p><span id="eq:scale_w" style="display: inline-block; position: relative; width: 100%"><span class="math display">\[
    w(\lambda_i) = 
    \begin{cases}
    \sigma(\lambda_i) \, \frac{\lambda_{i+1}}{2} &amp; i = 0 \\
    \sigma(\lambda_i) \, \frac{\lambda_{i+1} - \lambda_{i-1}}{2} &amp; i \in [1, N-1] \\
    \sigma(\lambda_i) \, \frac{1 - \lambda_{i-1}}{2} &amp; i = N \\
    \end{cases}
\]</span><span style="position: absolute; right: 0em; top: 50%; line-height:0; text-align: right">(1)</span></span> </p>
<p>fell below a threshold of 0.02 kcal/mol during the attach phase and 0.1 kcal/mol during the pull phase.</p>
<p>The second term in Equation <a href="#eq:scale_w">1</a> scales the uncertainty in the work in each λ window by the nonuniform spacing of the λ windows. <span class="math inline">\(w(\lambda_i)\)</span> the approximate contribution of window λ<sub>i</sub> to the overall PMF uncertainty. Excluding the first and last window, the average window length was 11.8 ns and 5.39 ns for GAFF v1.7 and GAFF v2.1 simulations, respectively. We took a more direct approach with the SMIRNOFF99Frosst simulations, due to changes in pAPRika that allowed us to target uncertainties of the same magnitude as in the GAFF simulations, by running each window for a constant length of 10 ns, except for the first and last window which ran for 1 μs to converge ΔH for all three force fields.</p>
<h3 id="statistics">Statistics</h3>
<p>The uncertainty in the work done by each restraint in each simulation window, σ(λ<sub>i</sub>), was estimated using blocking analysis <span class="citation" data-cites="htck51Lp">[<a href="#ref-htck51Lp" role="doc-biblioref">46</a>]</span>, in a manner which has been shown to yield good agreement with uncertainties obtained from independent replicates <span class="citation" data-cites="1935a9V0d">[<a href="#ref-1935a9V0d" role="doc-biblioref">13</a>]</span>. In particular, rather than looking for a plateau in the SEM as the size of the blocks increased, as originally described by Flyvbjerg and Peterssen <span class="citation" data-cites="htck51Lp">[<a href="#ref-htck51Lp" role="doc-biblioref">46</a>]</span>, we instead use the largest standard error of the mean (SEM) obtained for any block size. This avoids the requirement of detecting a plateau and yields a more conservative estimate; i.e., a larger SEM. Then, using Gaussians with the mean and SEM of <span class="math inline">\(\frac{\partial U}{\partial \lambda}\)</span> in each window, new values of <span class="math inline">\(\frac{\partial U}{\partial \lambda}\)</span> were bootstrap sampled for each window 100,000 times and combined to create artificial data for 100,000 notional APR calculations. These were integrated across all windows with splines to generate 100,000 estimates of ΔG°. We report the mean and standard deviation of these 100,000 results as the final mean and its SEM. The SEM of ΔH was computed from the SEM of the total potential energy in each end point window, estimated using blocking analysis, added in quadrature. The standard error of the mean of −TΔS° was calculated using the uncertainties in ΔG° and ΔH added in quadrature.</p>
<p>For each force field, we computed the root mean squared error (RMSE), mean signed error (MSE), the coefficient of determination (R<sup>2</sup>), Kendall’s rank correlation coefficient (τ), and the slope and intercept of the linear regressions of the computed properties against the experimental values. The R<sup>2</sup> values for the subsets of ligand with each are also reported in the bottom right corner in each graph. Comparisons with experiment have 43 measurements, for the 43 unique host-guest complexes listed in Table <a href="#tbl:host-guests">1</a>; comparisons between force fields have 86 data points, representing the calculations for the two orientations of the guest, “p” and “s”, in the binding site (see above). The overall RMSE and R<sup>2</sup> statistics for each comparison are reported as the sample mean estimated from using all the data, with the 95% confidence interval, from bootstrapping over the set of complexes, in brackets.</p>
<h2 id="results">Results</h2>
<p>This results section is organized as follows. We first present a comparison of binding free energies (ΔG°) and binding enthalpies (ΔH) of small molecule guests to α-cyclodextrin (αCD) and β-cyclodextrin (βCD), computed with SMIRNOFF99Frosst and two versions of the General AMBER Force Field (GAFF <span class="citation" data-cites="YmRgHfeU">[<a href="#ref-YmRgHfeU" role="doc-biblioref">29</a>]</span>). We then detail how the conformational preferences of the host molecules changes between force fields and seek insight into key parameter differences between SMIRNOFF99Frosst and GAFF and their effects.</p>
<h3 id="comparison-with-experimental-binding-free-energies-enthalpies-and-entropies">Comparison with experimental binding free energies, enthalpies, and entropies</h3>
<h4 id="binding-free-energies">Binding free energies</h4>
<p>Despite having far fewer numerical parameters, SMIRNOFF99Frosst does about as well as GAFF v1.7 and arguably better than GAFF v2.1 at replicating binding free energies measured by ITC or NMR. Thus, SMIRNOFF99Frosst yields an overall ΔG° RMSE from experiment of 0.9 [0.7, 1.1] kcal/mol across the 43 host-guest systems, compared to the statistically indistinguishable 0.9 [0.7, 1.1] kcal/mol for GAFF v1.7, and distinct from 1.7 [1.5, 1.9] kcal/mol for GAFF v2.1 (where the 95% confidence interval is written in brackets) as detailed in Figure <a href="#fig:dG-dH">3</a>; Tables <a href="#tbl:statistics">2</a>, <a href="#tbl:dG-combined">5</a>.</p>
<p>On the whole, GAFF v1.7 agrees well with SMIRNOFF99Frosst (Figure <a href="#fig:dG-by-orientation">21</a>), as the RMSE and MSE between their results are 0.8 [0.6, 1.0] kcal/mol and -0.5 [-0.3, -0.7] kcal/mol. This result is not surprising as GAFF v1.7 and SMIRNOFF99Frosst may be considered sibling force fields with a common ancestor in AMBER’s parm99. Both SMIRNOFF99Frosst and GAFF v1.7 systematically underestimate the binding affinity for cyclic alcohols, with MSEs of 0.7 [0.2, 1.2] kcal/mol and 0.9 [0.4, 1.4] kcal/mol, respectively. In contrast, GAFF v2.1 significantly overestimates the binding of all compounds, leading to MSE and RMSE values of -1.6 [-1.7, -1.4] kcal/mol and 1.6 [1.4, 1.8] kcal/mol, respectively. However, GAFF v2.1 has a particularly good correlation with experiment across all functional group classes, with R² of 0.8 [0.6, 0.9], compared with 0.3 [0.1, 0.6] and 0.5 [0.3, 0.7] for SMIRNOFF99Frosst and GAFF 1.7, respectively. This may trace to differences in the host conformations sampled by GAFF v2.1, which indicate a more consistently open cyclodextrin “pocket” for guests to bind (Figure <a href="#fig:flexibility">14</a>), as detailed below.</p>
<h4 id="binding-enthalpies-and-entropies">Binding enthalpies and entropies</h4>
<p>In the case of binding enthalpies (Figure <a href="#fig:dG-dH">3</a>), SMIRNOFF99Frosst agrees the best with experiment (RMSE 1.8 [1.4, 2.3] kcal/mol), followed by GAFF v2.1 (RMSE = 2.2 [1.8, 2.7] kcal/mol), and then GAFF v1.7 (RMSE = 2.5 [2.0, 3.0] kcal/mol). In some cases, GAFF v1.7 underestimates ΔH by over 3 kcal/mol and up to 5 kcal/mol (<code>b-chp</code>). For binding entropies, GAFF v2.1 has the lowest RMSE relative to experiment (RMSE = 1.47 [1.1, 2.0] kcal/mol), followed by SMIRNOFF99Frosst (RMSE = 1.9 kcal/mol [1.5, 2.3]), and GAFF v1.7 (RMSE = 2.2 [1.7, 2.7] kcal/mol) (Figure <a href="#fig:TdS">17</a>).<br />
All force fields perform poorly at replicating −TΔS° for carboxylate guests, with RMSEs ranging from 1.8 [0.7, 3.2] kcal/mol (GAFF v2.1) to 3.0 [2.1, 3.9] kcal/mol (GAFF v1.7). All force fields also underestimate the entropic component of binding of <code>a-coc</code> (αCD:cyclooctanol) relative to experiment, by 3-5 kcal/mol. This is likely due to the poor fit of cycloctanol inside the cavity of αCD, particularly in the primary orientation (Figure <a href="#fig:by-orientation">4</a>). Overall, SMIRNOFF99Frosst and GAFF v1.7 yield rather different binding enthalpies (RMSE = 1.6 [1.3, 2.0] kcal/mol) and entropies (RMSE = 1.6 [1.2, 2.0] kcal/mol). The deviations between SMIRNOFF99Frosst and GAFF v2.1 are higher for ΔH (RMSE = 3.0 [2.5, 3.4] kcal/mol) and lower for −TΔS° (RMSE = 1.9 [1.6, 2.2] kcal/mol).</p>
<figure>
<img src="images/dG-dH.png" id="fig:dG-dH" alt="" /><figcaption><span>Figure 3:</span> Comparison of calculated absolute binding free energies (ΔG°) and binding enthalpies (ΔH) with experiment with SMIRNOFF99Frosst parameters (A, B), GAFF v1.7 parameters (C, D), or GAFF v2.1 parameters (E, F) applied to both host and guest. The orange, blue, and purple coloring distinguish the functional group of the guest as an ammonium, alcohol, or carboxylate, respectively.</figcaption>
</figure>
<a name="tbl:statistics"></a>
<table style="width:100%;">
<caption><span>Table 2:</span> Predicted thermodynamic properties for each force field relative to experiment in kcal/mol. </caption>
<colgroup>
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th></th>
<th>RMSE</th>
<th></th>
<th>MSE</th>
<th></th>
<th>R²</th>
<th></th>
<th>Slope</th>
<th></th>
<th>Intercept</th>
<th></th>
<th>Tau</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>ΔG°</td>
<td>SMIRNOFF99Frosst</td>
<td>0.91</td>
<td>[0.71, 1.13]</td>
<td>-0.01</td>
<td>[-0.28, 0.26]</td>
<td>0.34</td>
<td>[0.12, 0.56]</td>
<td>0.49</td>
<td>[0.26, 0.72]</td>
<td>-1.55</td>
<td>[-0.80, -2.29]</td>
<td>0.40</td>
<td>[0.57, 0.23]</td>
</tr>
<tr class="even">
<td>ΔG°</td>
<td>GAFF v1.7</td>
<td>0.88</td>
<td>[0.72, 1.08]</td>
<td>0.46</td>
<td>[0.23, 0.69]</td>
<td>0.54</td>
<td>[0.33, 0.71]</td>
<td>0.69</td>
<td>[0.47, 0.91]</td>
<td>-0.48</td>
<td>[0.22, -1.16]</td>
<td>0.52</td>
<td>[0.65, 0.38]</td>
</tr>
<tr class="odd">
<td>ΔG°</td>
<td>GAFF v2.1</td>
<td>1.68</td>
<td>[1.51, 1.85]</td>
<td>-1.56</td>
<td>[-1.74, -1.37]</td>
<td>0.82</td>
<td>[0.61, 0.92]</td>
<td>1.19</td>
<td>[0.96, 1.34]</td>
<td>-1.00</td>
<td>[-0.52, -1.62]</td>
<td>0.73</td>
<td>[0.82, 0.61]</td>
</tr>
<tr class="even">
<td>ΔH</td>
<td>SMIRNOFF99Frosst</td>
<td>1.85</td>
<td>[1.41, 2.30]</td>
<td>0.76</td>
<td>[0.26, 1.28]</td>
<td>0.44</td>
<td>[0.21, 0.66]</td>
<td>0.85</td>
<td>[0.54, 1.19]</td>
<td>0.41</td>
<td>[1.55, -0.50]</td>
<td>0.53</td>
<td>[0.69, 0.34]</td>
</tr>
<tr class="odd">
<td>ΔH</td>
<td>GAFF v1.7</td>
<td>2.54</td>
<td>[2.08, 3.00]</td>
<td>1.84</td>
<td>[1.31, 2.37]</td>
<td>0.39</td>
<td>[0.17, 0.62]</td>
<td>0.80</td>
<td>[0.47, 1.18]</td>
<td>1.36</td>
<td>[2.67, 0.31]</td>
<td>0.50</td>
<td>[0.65, 0.32]</td>
</tr>
<tr class="even">
<td>ΔH</td>
<td>GAFF v2.1</td>
<td>2.21</td>
<td>[1.77, 2.65]</td>
<td>-1.64</td>
<td>[-2.10, -1.20]</td>
<td>0.75</td>
<td>[0.58, 0.87]</td>
<td>1.38</td>
<td>[1.15, 1.63]</td>
<td>-0.69</td>
<td>[0.16, -1.43]</td>
<td>0.67</td>
<td>[0.79, 0.52]</td>
</tr>
<tr class="odd">
<td>−TΔS°</td>
<td>SMIRNOFF99Frosst</td>
<td>1.90</td>
<td>[1.49, 2.32]</td>
<td>-0.78</td>
<td>[-1.29, -0.24]</td>
<td>0.40</td>
<td>[0.14, 0.63]</td>
<td>0.90</td>
<td>[0.51, 1.29]</td>
<td>-0.83</td>
<td>[-0.34, -1.34]</td>
<td>0.33</td>
<td>[0.50, 0.13]</td>
</tr>
<tr class="even">
<td>−TΔS°</td>
<td>GAFF v1.7</td>
<td>2.21</td>
<td>[1.74, 2.68]</td>
<td>-1.38</td>
<td>[-1.90, -0.86]</td>
<td>0.43</td>
<td>[0.16, 0.68]</td>
<td>0.95</td>
<td>[0.54, 1.38]</td>
<td>-1.41</td>
<td>[-0.96, -1.89]</td>
<td>0.32</td>
<td>[0.50, 0.10]</td>
</tr>
<tr class="odd">
<td>−TΔS°</td>
<td>GAFF v2.1</td>
<td>1.80</td>
<td>[0.68, 3.19]</td>
<td>-0.00</td>
<td>[-0.98, 1.27]</td>
<td>0.48</td>
<td>[0.00, 0.97]</td>
<td>1.13</td>
<td>[-0.22, 1.96]</td>
<td>0.08</td>
<td>[1.14, -1.79]</td>
<td>0.46</td>
<td>[0.82, -0.02]</td>
</tr>
</tbody>
</table>
<p>Analysis of the simulations with MBAR produces very slightly improved results for SMIRNOFF99Frosst ΔG°, ΔH, and −TΔS° compared to experiment (Table <a href="#tbl:mbar-statistics">8</a>), but they do not appear to be statistically significant.</p>
<h3 id="guest-preferences-for-binding-in-the-primary-or-secondary-orientation">Guest preferences for binding in the primary or secondary orientation</h3>
<p>The asymmetry of the hosts and the guests leads to two distinct bound states for each host-guest pair: one where the functional group of the guest sits at the primary face of the host and another where the functional group of the guest sits at the secondary face (<a href="#fig:cavity">18</a>). The difference in binding free energy between these two orientations (ΔΔG<sub>orientation</sub>) can be large, at around 2 kcal/mol for SMIRNOFF99Frosst and GAFF v1.7 and 5 kcal/mol for GAFF v2.1. SMIRNOFF99Frosst predicts the largest ΔΔG<sub>orientation</sub> for the ammonium-containing butylamine and pentylamine guests with αCD (<a href="#fig:by-orientation">4</a>), with the primary orientation being more favorable. Thus, the cationic ammonium groups are predicted to prefer the narrower primary portal of the host. GAFF v1.7 predicts a large ΔΔG<sub>orientation</sub> for the cyclic alcohols cyclooctanol and cycloheptanol, with the secondary orientation having a more favorable ΔG. When GAFF v2.1 is used, the differences between primary and secondary binding range even higher, greater than 4 kcal/mol, for αCD with these two guests. This effect is due, at least in part, to steric clashes in the bound state for very large guests (Figure <a href="#fig:by-orientation">4</a>, D), especially in the narrow primary cavity of the smaller αCD. It is worth noting that the experimental measurement for the the <code>a-coc</code> (αCD:cyclooctanol) complex has very large uncertainties associated with both ΔG° and ΔH.</p>
<figure>
<img src="images/by-orientation.png" id="fig:by-orientation" alt="" /><figcaption><span>Figure 4:</span> Comparisons of binding free energy (ΔG) between guests in either the primary or secondary orientation of αCD or βCD, for SMIRNOFF99Frosst (A), GAFF v1.7 (B), or GAFF v2.1 (C). Arrows point from ΔG for the secondary to ΔG for the primary cavity. (D) An overlay of cyclooctanol bound state positions (400 snapshots over 1 μs) with αCD (left) or βCD (right) in GAFF v2.1.</figcaption>
</figure>
<h3 id="comparison-of-results-for-αcd-versus-βcd">Comparison of results for αCD versus βCD</h3>
<p>It is of interest to compare the results between αCD and βCD by focusing on the ten guests for which experimental data are available with both hosts. The SMIRNOFF99Frosst and GAFF 1.7 force fields both yield somewhat more accurate binding affinities for αCD (RMSE = 0.8 [0.5, 1.1] kcal/mol) than for βCD (RMSE = 1.0 [0.8, 1.3] kcal/mol), whereas no clear patterns is observed for GAFF v2.1 (Figure <a href="#fig:dG-dH-by-cyclodextrin">24</a>). Much as seen for the two orientations of the guest molecules within each host, GAFF v2.1 yields relatively large differences in predicted free energies for each guest between the two hosts, but it does not seem to be more accurate for either host relative to the other.</p>
<figure>
<img src="images/by-cyclodextrin.png" id="fig:by-cyclodextrin" alt="" /><figcaption><span>Figure 5:</span> Shown are the αCD and βCD binding free energies for each guest, highlighting the differences in binding to the two hosts for SMIRNOFF99Frosst (A), GAFF v1.7 (B), or GAFF v2.1 (C). The binding affinity for αCD is circled in black. Thin colored lines connect data points for the same guest. Color is used purely to distinguish among the guests.</figcaption>
</figure>
<h3 id="trends-by-guest-functional-group">Trends by guest functional group</h3>
<p>The SMIRNOFF99Frosst force field yields rather accurate binding free energies for binding of the ammonium guests (MSE = -0.1 [-0.5, 0.3] kcal/mol and RMSE = 0.7 [0.4, 1.1] kcal/mol) to both αCD and βCD (Figure <a href="#fig:ammonium">6</a> and Table <a href="#tbl:overall-ammonium">9</a>). It also replicates the experimental trends that shorter-chain molecules bind less strongly, and that each guest binds more strongly to αCD than βCD. The results are also reasonably good for the cyclic alcohols (MSE = 0.7 [0.2, 1.2] kcal/mol and RMSE = 1.1 [0.7, 1.6] kcal/mol) (Figure <a href="#fig:alcohols">7</a> and Table <a href="#tbl:overall-alcohols">11</a>), though the predicted affinities for αCD are uniformly too weak, while those for βCD are mostly too strong. Finally, SMIRNOFF99Frosst yields rather accurate binding affinities for the carboxylate guests with both αCD and βCD (MSE = -0.4 [-0.7, 0] kcal/mol and RMSE = 0.9 [0.6, 1.2] kcal/mmol) (Figure <a href="#fig:carboxylates">8</a> and Table <a href="#tbl:overall-carboxylates">10</a>).</p>
<p>GAFF v1.7 tends to predict slightly weaker binding than SMIRNOFF99Frosst, whereas GAFF v2.1 predicts much stronger binding for all classses of guest compounds (Figures <a href="#fig:additional-highlights-ammonium">25</a>, <a href="#fig:additional-highlights-alcohols">26</a>, and <a href="#fig:additional-highlights-carboxylates">27</a>).</p>
<figure>
<img src="images/ammonium.png" id="fig:ammonium" alt="" /><figcaption><span>Figure 6:</span> Binding free energy (ΔG°) comparisons showing ammonium-containing guests in color and highlighted, for αCD (A) and βCD (B). Darker colors indicate shorter chain molecules. Non-ammonium guests are shown as smaller gray circles.</figcaption>
</figure>
<figure>
<img src="images/alcohols.png" id="fig:alcohols" alt="" /><figcaption><span>Figure 7:</span> Binding free energy (ΔG°) comparisons showing alcohol-containing guests in color and highlighted, for αCD (A) and βCD (B). Darker colors indicate smaller molecules. Non-alcohol guests are shown as smaller gray circles.</figcaption>
</figure>
<figure>
<img src="images/carboxylates.png" id="fig:carboxylates" alt="" /><figcaption><span>Figure 8:</span> Binding free energy (ΔG°) comparisons showing carboxylate-containing guests in color and highlighted, for αCD (A) and βCD (B). Darker colors indicates smaller molecules. Non-carboxylate guests are shown as smaller gray circles.</figcaption>
</figure>
<h3 id="differences-in-cyclodextrin-force-field-parameters-between-smirnoff99frosst-and-gaff">Differences in cyclodextrin force field parameters between SMIRNOFF99Frosst and GAFF</h3>
<p>We now summarize differences among the parameters assigned to the host αCD by SMIRNOFF99Frosst, a descendant of parm99 and parm@Frosst; GAFF v1.7 (released circa March 2015 according to <code>gaff.dat</code> distributed with AMBER16); and GAFF v2.1 (which has not yet been published). On going from GAFF v1.7 to GAFF v2.1, the bond and angle parameters were updated to reproduce small molecule geometries obtained from high-level quantum mechanical calculations and vibrational spectra of over 600 molecules; the torsion parameters were optimized to reproduce the potential energy surfaces of torsion angles in 400 model compounds; and the Lennard-Jones coefficients were redeveloped to reproduce interaction energies and pure liquid properties, as specified in the footer of <code>gaff2.dat</code> provided with AmberTools18. Note that chemically analogous atoms, bonds, angles and torsions in αCD and βCD are assigned identical parameters.</p>
<h4 id="lennard-jones">Lennard-Jones</h4>
<p>The SMIRNOFF99Frosst and GAFF v1.7 force fields assign identical σ and ε parameters to the atoms of αCD. Note, that hydroxyl hydrogens are assigned σ = 0 Å and ε = 0 kcal/mol in both GAFF v1.7 and SMIRNOFF99Frosst v1.0.5, but later versions of SMIRNOFF99Frosst, produced after the calculations in the current manuscript, adopt <a href="https://github.com/openforcefield/smirnoff99Frosst/blob/4ca43af6241a97edb356586f3f83ac21afcdff65/smirnoff99Frosst/smirnoff99Frosst.offxml#L315">small σ and ε</a> values based on a similiar atom type in parm@Frosst <span class="citation" data-cites="Q2dwH7Dq cT3I9uDT 10E0XdByV">[<a href="#ref-Q2dwH7Dq" role="doc-biblioref">47</a>,<a href="#ref-cT3I9uDT" role="doc-biblioref">48</a>,<a href="#ref-10E0XdByV" role="doc-biblioref">49</a>]</span>. The GAFF v2.1 parameters differ in assigning shallower wells for oxygens and larger σ values for the hydroxyl hydrogens (Figure <a href="#fig:LJ">9</a>).</p>
<figure>
<img src="images/LJ.png" id="fig:LJ" style="width:100.0%" alt="" /><figcaption><span>Figure 9:</span> A comparison of Lennard-Jones nonbonded σ (A) and ε (B) parameters for SMIRNOFF99Frosst and GAFF v2.1. Values that differ by more than 10% are labeled in red. Atom names refer to Figure <a href="#fig:atom-names">2</a>.</figcaption>
</figure>
<h4 id="bond-stretches">Bond stretches</h4>
<p>Equilibrium bond lengths are very similar among the three force fields (Figure <a href="#fig:bond-req">28</a>), but there are noticeable differences among the force constants (Figure <a href="#fig:bonds">10</a>) Thus, compared to GAFF v1.7, SMIRNOFF99Frosst tends to have slightly larger bond force constants, except for the O–H hydroxyl bond force constant, which is much stronger. In GAFF v2.1, the O–H hydroxyl bond force constant is very close to that of SMIRNOFF99Frosst, but the carbon-oxygen bond constants are distinctly weaker.</p>
<figure>
<img src="images/bonds.png" id="fig:bonds" style="width:100.0%" alt="" /><figcaption><span>Figure 10:</span> A comparison of bond force constants between SMIRNOFF99Frosst and GAFF v1.7 (A), or SMIRNOFF99Frosst and GAFF v2.1 (B). Values that differ by more than 10% are labeled in red. Atom names refer to Figure <a href="#fig:atom-names">2</a>.</figcaption>
</figure>
<h4 id="bond-angles">Bond angles</h4>
<p>Relative to GAFF v1.7 and GAFF v2.1, SMIRNOFF99Frosst has fewer unique angle parameters applied to αCD; several distinct parameters appear to be compressed into a single force constant, around 50 kcal/mol/rad<sup>2</sup> (Figure <a href="#fig:angles">11</a>). These parameters correspond to C–C–C, C–O–C, O–C–O angles. The C–C–C angles are primarily around the ring of the glucose monomer. The C–O–C angles are both around the ring and between monomers (e.g., C<sub>1</sub>–O<sub>1</sub>–C<sub>4</sub> and C<sub>1</sub>–O<sub>5</sub>–C<sub>5</sub>). Weaker force constants for these parameters in GAFF v1.7 compared to GAFF v2.1 may lead to increased flexibility.</p>
<figure>
<img src="images/angles.png" id="fig:angles" style="width:100.0%" alt="" /><figcaption><span>Figure 11:</span> A comparison of angle force constants between SMIRNOFF99Frosst and GAFF v1.7 (A) or SMIRNOFF99Frosst and GAFF v2.1 (B). A comparison of equilibrium angle values SMIRNOFF99Frosst and GAFF v1.7 (C) or SMIRNOFF99Frosst and GAFF v2.1 (D). Values that differ by more than 10% are labeled in red. Precise atom names have been omitted to compress multiple angles with the same parameter values into a single label.</figcaption>
</figure>
<h4 id="dihedral-parameters">Dihedral parameters</h4>
<p>The dihedral parameters in SMIRNOFF99Frosst and GAFF v1.7 are extremely similar—where differences in barrier heights occur, they are in the hundredths or thousandths of 1 kcal/mol—with the exception of the H<sub>1</sub>–C<sub>1</sub>–C<sub>2</sub>–O<sub>2</sub> parameter (Figure <a href="#fig:atom-names">2</a>). For this dihedral, which corresponds to GAFF atom types <code>h2-c3-c3-oh</code> and SMIRKS pattern <code>[#1:1]-[#6X4:2]-[#6X4:3]-[#8X2:4]</code>), SMIRNOFF99Frosst applies a single term with periodicity = 1 and GAFF v1.7 applies a single term with periodicity = 3 (Table <a href="#tbl:S99-vs-GAFF-v1.7">12</a>, Figures <a href="#fig:dihedral">12</a>).</p>
<figure>
<img src="images/H1-C2-C2-O2.png" id="fig:dihedral" style="width:100.0%" alt="" /><figcaption><span>Figure 12:</span> (A) The atoms in the H<sub>1</sub>–C<sub>1</sub>–C<sub>2</sub>–O<sub>2</sub> dihedral marked in purple on a glucose monomer in cyclodextrin. (B) The dihedral energy term applied to H<sub>1</sub>–C<sub>1</sub>–C<sub>2</sub>–O<sub>2</sub> in SMIRNOFF99Frosst and GAFF v1.7. Atom names refer to Figure <a href="#fig:atom-names">2</a>.</figcaption>
</figure>
<p>The dihedral parameters in GAFF v2.1 differ from those in SMIRNOFF99Frosst in a number of ways. There are several dihedrals that have a different number of terms (Table <a href="#tbl:S99-vs-GAFF-v2.1-missing">13</a>). This is partly due to the addition of dihedral terms with a barrier height of exactly 0.00 kcal/mol in GAFF, which are used to override wildcard parameters that might match the same atom types. For example, GAFF v2.1 applies a three term energy function to the atom types <code>c3-os-c3-c3</code>, whereas SMIRNOFF99Frosst employs a two term energy function for the hydroxyl rotation SMIRKS pattern <code>[#6X4:1]-[#6X4:2]-[#8X2H0:3]-[#6X4:4]</code>, but only the terms with periodicity 2 and 3 have nonzero barrier heights in GAFF v2.1. Similarly, SMIRNOFF99Frosst uses two nonzero terms to model the potential barrier for the SMIRKS pattern <code>[#6X4:1]-[#6X4:2]-[#8X2H1:3]-[#1:4]</code>, yet GAFF v2.1 applies a single term with a barrier height of exactly 0.00 kcal/mol for this rotation (atom types <code>c3-c3-oh-ho</code>). The fact that GAFF employs dihedral terms with zero amplitude terms highlights the complexity that would be required to optimize existing force fields that have accumulated legacy parameters needed to maintain backwards compatibility with older force fields and simulation codes.</p>
<p>In other cases, SMIRNOFF99Frosst and GAFF v2.1 have disagreements on the barrier height after matching the periodicity and phase for a given dihedral. For example, the amplitudes for the O<sub>1</sub>–C<sub>1</sub>–O<sub>5</sub>–C<sub>5</sub> dihedral are 1.35 kcal/mol and 0.97 kcal/mol for SMIRNOFF99Frosst and GAFF v2.1, respectively, for the term with periodicity = 1, whereas the amplitudes are 0.85 kcal/mol and 1.24 kcal/mol for SMIRNOFF99Frosst and GAFF v2.1, respectively, for the term with periodicity = 2. It is notable that the barrier heights in GAFF v2.1 are similiar in magnitude to those in SMIRNOFF99Frosst, yet GAFF v2.1 produces much more rigid structures (Table <a href="#tbl:S99-vs-GAFF-v2.1">3</a>, Figure <a href="#fig:flexibility">14</a>), as detailed in the following section. Moreoever, many of the dihedrals that act between a pair of neighboring glucose monomers (i.e., inter-residue dihedrals) in cyclodextrin differ in their periodicies, phases, and amplitudes between SMIRNOFF99Frosst and GAFF v2.1 (Table <a href="#tbl:S99-vs-GAFF-v2.1-inter">4</a>, Figure <a href="#fig:interdihedrals">13</a>). The dihedral acting on atoms O1<sub>n</sub>–C4<sub>n+1</sub>–C5<sub>n+1</sub>–O5<sub>n+1</sub> is quite significantly different, with multiple minima and and barrier heights. This dihedral partially controls the rotation of glucose monomers towards or away from the interior of the cyclodextrin cavity. Surprisingly, glucose monomers in GAFF v2.1 penetrate the open cavity much less frequently than in SMIRNOFF99Frosst, despite the lower and broader dihedral energy in GAFF v2.1.</p>
<a name="tbl:S99-vs-GAFF-v2.1"></a>
<table>
<caption><span>Table 3:</span> Dihedral barrier height differences between SMIRNOFF99Frosst and GAFF v2.1 for cases where the phase and periodicity of the energy term match but the barrier height does not. Atom names refer to Figure <a href="#fig:atom-names">2</a>. </caption>
<thead>
<tr class="header">
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th>SMIRNOFF99Frosst</th>
<th>GAFF v2.1</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SMIRKS</td>
<td>Atom 1</td>
<td>Atom 2</td>
<td>Atom 3</td>
<td>Atom 4</td>
<td>Per</td>
<td>Phase</td>
<td>Height (kcal/mol)</td>
<td>Height (kcal/mol)</td>
<td></td>
</tr>
<tr class="even">
<td>[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]</td>
<td>C1</td>
<td>C2</td>
<td>C3</td>
<td>C4</td>
<td>1</td>
<td>0</td>
<td>0.20</td>
<td>0.11</td>
<td></td>
</tr>
<tr class="odd">
<td>[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]</td>
<td>C1</td>
<td>C2</td>
<td>C3</td>
<td>C4</td>
<td>2</td>
<td>0</td>
<td>0.25</td>
<td>0.29</td>
<td></td>
</tr>
<tr class="even">
<td>[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]</td>
<td>C1</td>
<td>C2</td>
<td>C3</td>
<td>C4</td>
<td>3</td>
<td>0</td>
<td>0.18</td>
<td>0.13</td>
<td></td>
</tr>
<tr class="odd">
<td>[*:1]-[#6X4:2]-[#6X4:3]-[*:4]</td>
<td>C1</td>
<td>C2</td>
<td>C3</td>
<td>O3</td>
<td>3</td>
<td>0</td>
<td>0.16</td>
<td>0.21</td>
<td></td>
</tr>
<tr class="even">
<td>[*:1]-[#6X4:2]-[#8X2H0:3]-[*:4]</td>
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>H5</td>
<td>3</td>
<td>0</td>
<td>0.38</td>
<td>0.34</td>
<td></td>
</tr>
<tr class="odd">
<td>[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]</td>
<td>C2</td>
<td>C3</td>
<td>C4</td>
<td>C5</td>
<td>1</td>
<td>0</td>
<td>0.20</td>
<td>0.11</td>
<td></td>
</tr>
<tr class="even">
<td>[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]</td>
<td>C2</td>
<td>C3</td>
<td>C4</td>
<td>C5</td>
<td>2</td>
<td>0</td>
<td>0.25</td>
<td>0.29</td>
<td></td>
</tr>
<tr class="odd">
<td>[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]</td>
<td>C2</td>
<td>C3</td>
<td>C4</td>
<td>C5</td>
<td>3</td>
<td>0</td>
<td>0.18</td>
<td>0.13</td>
<td></td>
</tr>
<tr class="even">
<td>[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]</td>
<td>C3</td>
<td>C4</td>
<td>C5</td>
<td>C6</td>
<td>1</td>
<td>0</td>
<td>0.20</td>
<td>0.11</td>
<td></td>
</tr>
<tr class="odd">
<td>[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]</td>
<td>C3</td>
<td>C4</td>
<td>C5</td>
<td>C6</td>
<td>2</td>
<td>0</td>
<td>0.25</td>
<td>0.29</td>
<td></td>
</tr>
<tr class="even">
<td>[#6X4:1]-[#6X4:2]-[#6X4:3]-[#6X4:4]</td>
<td>C3</td>
<td>C4</td>
<td>C5</td>
<td>C6</td>
<td>3</td>
<td>0</td>
<td>0.18</td>
<td>0.13</td>
<td></td>
</tr>
<tr class="odd">
<td>[*:1]-[#6X4:2]-[#6X4:3]-[*:4]</td>
<td>C4</td>
<td>C5</td>
<td>C6</td>
<td>O6</td>
<td>3</td>
<td>0</td>
<td>0.16</td>
<td>0.21</td>
<td></td>
</tr>
<tr class="even">
<td>[#1:1]-[#6X4:2]-[#6X4:3]-[#1:4]</td>
<td>H1</td>
<td>C1</td>
<td>C2</td>
<td>H2</td>
<td>3</td>
<td>0</td>
<td>0.15</td>
<td>0.16</td>
<td></td>
</tr>
<tr class="odd">
<td>[#1:1]-[#6X4:2]-[#6X4:3]-[#1:4]</td>
<td>H2</td>
<td>C2</td>
<td>C3</td>
<td>H3</td>
<td>3</td>
<td>0</td>
<td>0.15</td>
<td>0.16</td>
<td></td>
</tr>
<tr class="even">
<td>[*:1]-[#6X4:2]-[#8X2:3]-[#1:4]</td>
<td>H2</td>
<td>C2</td>
<td>O2</td>
<td>HO2</td>
<td>3</td>
<td>0</td>
<td>0.17</td>
<td>0.11</td>
<td></td>
</tr>
<tr class="odd">
<td>[#1:1]-[#6X4:2]-[#6X4:3]-[#1:4]</td>
<td>H3</td>
<td>C3</td>
<td>C4</td>
<td>H4</td>
<td>3</td>
<td>0</td>
<td>0.15</td>
<td>0.16</td>
<td></td>
</tr>
<tr class="even">
<td>[*:1]-[#6X4:2]-[#8X2:3]-[#1:4]</td>
<td>H3</td>
<td>C3</td>
<td>O3</td>
<td>HO3</td>
<td>3</td>
<td>0</td>
<td>0.17</td>
<td>0.11</td>
<td></td>
</tr>
<tr class="odd">
<td>[#1:1]-[#6X4:2]-[#6X4:3]-[#1:4]</td>
<td>H4</td>
<td>C4</td>
<td>C5</td>
<td>H5</td>
<td>3</td>
<td>0</td>
<td>0.15</td>
<td>0.16</td>
<td></td>
</tr>
<tr class="even">
<td>[#1:1]-[#6X4:2]-[#6X4:3]-[#1:4]</td>
<td>H5</td>
<td>C5</td>
<td>C6</td>
<td>H61</td>
<td>3</td>
<td>0</td>
<td>0.15</td>
<td>0.16</td>
<td></td>
</tr>
<tr class="odd">
<td>[#1:1]-[#6X4:2]-[#6X4:3]-[#1:4]</td>
<td>H5</td>
<td>C5</td>
<td>C6</td>
<td>H62</td>
<td>3</td>
<td>0</td>
<td>0.15</td>
<td>0.16</td>
<td></td>
</tr>
<tr class="even">
<td>[#6X4:1]-[#8X2:2]-[#6X4:3]-[#8X2:4]</td>
<td>O1</td>
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>1</td>
<td>0</td>
<td>1.35</td>
<td>0.97</td>
<td></td>
</tr>
<tr class="odd">
<td>[#6X4:1]-[#8X2:2]-[#6X4:3]-[#8X2:4]</td>
<td>O1</td>
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>2</td>
<td>0</td>
<td>0.85</td>
<td>1.24</td>
<td></td>
</tr>
<tr class="even">
<td>[#6X4:1]-[#8X2:2]-[#6X4:3]-[#8X2:4]</td>
<td>O1</td>
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>3</td>
<td>0</td>
<td>0.10</td>
<td>0.00</td>
<td></td>
</tr>
<tr class="odd">
<td>[*:1]-[#6X4:2]-[#6X4:3]-[*:4]</td>
<td>O2</td>
<td>C2</td>
<td>C3</td>
<td>C4</td>
<td>3</td>
<td>0</td>
<td>0.16</td>
<td>0.21</td>
<td></td>
</tr>
<tr class="even">
<td>[#8X2:1]-[#6X4:2]-[#6X4:3]-[#8X2:4]</td>
<td>O2</td>
<td>C2</td>
<td>C3</td>
<td>O3</td>
<td>2</td>
<td>0</td>
<td>1.18</td>
<td>1.13</td>
<td></td>
</tr>
<tr class="odd">
<td>[#8X2:1]-[#6X4:2]-[#6X4:3]-[#8X2:4]</td>
<td>O2</td>
<td>C2</td>
<td>C3</td>
<td>O3</td>
<td>3</td>
<td>0</td>
<td>0.14</td>
<td>0.90</td>
<td></td>
</tr>
<tr class="even">
<td>[*:1]-[#6X4:2]-[#6X4:3]-[*:4]</td>
<td>O3</td>
<td>C3</td>
<td>C4</td>
<td>C5</td>
<td>3</td>
<td>0</td>
<td>0.16</td>
<td>0.21</td>
<td></td>
</tr>
<tr class="odd">
<td>[*:1]-[#6X4:2]-[#8X2:3]-[#1:4]</td>
<td>H61</td>
<td>C6</td>
<td>O6</td>
<td>HO6</td>
<td>3</td>
<td>0</td>
<td>0.17</td>
<td>0.11</td>
<td></td>
</tr>
<tr class="even">
<td>[*:1]-[#6X4:2]-[#8X2:3]-[#1:4]</td>
<td>H62</td>
<td>C6</td>
<td>O6</td>
<td>HO6</td>
<td>3</td>
<td>0</td>
<td>0.17</td>
<td>0.11</td>
<td></td>
</tr>
</tbody>
</table>
<a name="tbl:S99-vs-GAFF-v2.1-inter"></a>
<table style="width:100%;">
<caption><span>Table 4:</span> Inter-residue dihedral parameter differences between SMIRNOFF99Frosst and GAFF v2.1. Atom names refer to Figure <a href="#fig:atom-names">2</a>. NP: not present. </caption>
<colgroup>
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th>SMIRNOFF99Frosst</th>
<th>GAFF v2.1</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>ID</td>
<td>Atom 1</td>
<td>Res 1</td>
<td>Atom 2</td>
<td>Res 2</td>
<td>Atom 3</td>
<td>Res 3</td>
<td>Atom 4</td>
<td>Res 4</td>
<td>Per</td>
<td>Phase</td>
<td>Height (kcal/mol)</td>
<td>Height (kcal/mol)</td>
</tr>
<tr class="even">
<td>1</td>
<td>C1</td>
<td><em>n</em></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C3</td>
<td><em>n+1</em></td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.00</td>
</tr>
<tr class="odd">
<td></td>
<td>C1</td>
<td><em>n</em></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C3</td>
<td><em>n+1</em></td>
<td>2</td>
<td>0</td>
<td>0.10</td>
<td>0.16</td>
</tr>
<tr class="even">
<td></td>
<td>C1</td>
<td><em>n</em></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C3</td>
<td><em>n+1</em></td>
<td>3</td>
<td>0</td>
<td>0.38</td>
<td>0.24</td>
</tr>
<tr class="odd">
<td>2</td>
<td>C1</td>
<td><em>n</em></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C5</td>
<td><em>n+1</em></td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.00</td>
</tr>
<tr class="even">
<td></td>
<td>C1</td>
<td><em>n</em></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C5</td>
<td><em>n+1</em></td>
<td>2</td>
<td>0</td>
<td>0.10</td>
<td>0.16</td>
</tr>
<tr class="odd">
<td></td>
<td>C1</td>
<td><em>n</em></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C5</td>
<td><em>n+1</em></td>
<td>3</td>
<td>0</td>
<td>0.38</td>
<td>0.24</td>
</tr>
<tr class="even">
<td>3</td>
<td>C2</td>
<td><em>n</em></td>
<td>C1</td>
<td><em>n+1</em></td>
<td>O1</td>
<td><em>n+1</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.00</td>
</tr>
<tr class="odd">
<td></td>
<td>C2</td>
<td><em>n</em></td>
<td>C1</td>
<td><em>n+1</em></td>
<td>O1</td>
<td><em>n+1</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>2</td>
<td>0</td>
<td>0.10</td>
<td>0.16</td>
</tr>
<tr class="even">
<td></td>
<td>C2</td>
<td><em>n</em></td>
<td>C1</td>
<td><em>n+1</em></td>
<td>O1</td>
<td><em>n+1</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>3</td>
<td>0</td>
<td>0.38</td>
<td>0.24</td>
</tr>
<tr class="odd">
<td>4</td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C3</td>
<td><em>n+1</em></td>
<td>O3</td>
<td><em>n+1</em></td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.02</td>
</tr>
<tr class="even">
<td></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C3</td>
<td><em>n+1</em></td>
<td>O3</td>
<td><em>n+1</em></td>
<td>2</td>
<td>0</td>
<td>1.18</td>
<td>0.00</td>
</tr>
<tr class="odd">
<td></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C3</td>
<td><em>n+1</em></td>
<td>O3</td>
<td><em>n+1</em></td>
<td>3</td>
<td>0</td>
<td>0.14</td>
<td>1.01</td>
</tr>
<tr class="even">
<td>5</td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C5</td>
<td><em>n+1</em></td>
<td>O5</td>
<td><em>n+1</em></td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.17</td>
</tr>
<tr class="odd">
<td></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C5</td>
<td><em>n+1</em></td>
<td>O5</td>
<td><em>n+1</em></td>
<td>2</td>
<td>0</td>
<td>1.18</td>
<td>0.00</td>
</tr>
<tr class="even">
<td></td>
<td>O1</td>
<td><em>n</em></td>
<td>C4</td>
<td><em>n+1</em></td>
<td>C5</td>
<td><em>n+1</em></td>
<td>O5</td>
<td><em>n+1</em></td>
<td>3</td>
<td>0</td>
<td>0.14</td>
<td>0.00</td>
</tr>
</tbody>
</table>
<figure>
<img src="images/interdihedrals.png" id="fig:interdihedrals" style="width:100.0%" alt="" /><figcaption><span>Figure 13:</span> The dihedral energy term applied to three inter-residue dihedrals in SMIRNOFF99Frosst and GAFF v2.1. Atom names refer to Figure <a href="#fig:atom-names">2</a>.</figcaption>
</figure>
<p>There are no improper dihedrals in αCD or βCD, nor any of the guests.</p>
<h3 id="structural-consequences-of-the-force-field-parameter-differences">Structural consequences of the force field parameter differences</h3>
<p>We observed a substantial difference between the conformational flexibility of the uncomplexed cyclodextrins in solution when simulated with GAFF v2.1 versus SMIRNOFF99Frosst. With SMIRNOFF99Frosst and GAFF v1.7, the average RMSD of βCD, relative to the initial structure, is between 2.0–2.5 Å over 43 μs of unrestrained simulation, while with GAFF v2.1, the average RMSD is &lt;1.0 Å (Figure <a href="#fig:flexibility">14</a>). Not only are the RMSDs greater for SMIRNOFF99Frosst and GAFF v1.7, but there is greater variance in their RMSDs compared to GAFF v2.1, indicating greater flexibility. This large difference in structural fluctuations is clearly visible in the structure overlays also shown in the figure, which shows that GAFF v2.1 is the only one of the three force fields that leads to maintainance of a clearly defined binding cavity. In this respect, it is similar to the q4md-CD force field <span class="citation" data-cites="j7JJedX4">[<a href="#ref-j7JJedX4" role="doc-biblioref">50</a>]</span>, which was designed specifically for cyclodextrins and which also maintains a relatively well-defined cavity <span class="citation" data-cites="HVgz5rZq">[<a href="#ref-HVgz5rZq" role="doc-biblioref">33</a>]</span>.</p>
<figure>
<img src="images/cyclodextrin-flexibility.png" id="fig:flexibility" style="width:100.0%" alt="" /><figcaption><span>Figure 14:</span> Top: Root mean square deviation (RMSD in Å) of free βCD in the three force fields. Each RMSD is calculated relative to a gas-phase minimization of βCD with GAFF v1.7. A 1000 frame moving average is plotted in red. Middle: top-view of the open cavity of βCD with no guest (200 snapshots over 1 μs). Bottom: side-view of the open cavity. The carbons are colored blue in SMIRNOFF99Frosst, green in GAFF v1.7, and purple in GAFF v2.1. Hydrogen atoms have been hidden for clarity.</figcaption>
</figure>
<p>This difference may be further analyzed by considering the “flip” pseudodihedral O2<sub>n</sub>–C1<sub>n</sub>–C4<sub>n+1</sub>–O3<sub>n+1</sub>, which characterizes the orientation of glucose monomers relative to their neighbors. An angle of 0° corresponds approximately to a glucose that forms part of a cylindrical wall of the binding cavity, while an angle of ± 90° indicates a glucose that has flipped to put its plane parallel to the top and bottom of the cylinder, partly filling the cavity. This dihedral is tightly distributed in GAFF v2.1, with all seven instances having a Gaussian-like distribution centered around -10° (Figure <a href="#fig:pseudodihedral">15</a>, A). GAFF v1.7 and SMIRNOFF99Frosst display, a mixed population of monomers both aligned with, and perpendicular to, the cyclodextrin cavity. In particular, during a single 1 μs simulation, each monomer will sample conformations at 0° and ±90°, as indicated by the timeseries in Figure <a href="#fig:pseudodihedral">15</a>, B. As detailed in the Discussion, the less flexible representation afforded by GAFF v2.1 agrees better with available NMR and crystallographic data.</p>
<figure>
<img src="images/cyclodextrin-pseudodihedral.png" id="fig:pseudodihedral" style="width:100.0%" alt="" /><figcaption><span>Figure 15:</span> (A) Population histograms of the pseudodihedral in free βCD, averaged over 43 μs, for each force field; one curve is drawn for each pseudodihedral in βCD. Representative structures for SMIRNOFF99Frosst and GAFF v2.1 are indicated by the black arrows. (B) Timeseries of a psuedodihedral in a GAFF v2.1 simulation (red), or SMIRNOFF99Frosst simulation (blue).</figcaption>
</figure>
<h2 id="discussion">Discussion</h2>
<p>As a terse representation of a GAFF-like force field, SMIRNOFF99Frosst performs remarkably well. Despite having far fewer parameters than GAFF v1.7 and GAFF v2.1, SMIRNOFF99Frosst performs as well as GAFF v1.7 and arguably better than GAFF v2.1 on estimated binding free energies of small molecules to αCD and βCD, based on the mean signed error relative to experiment. Moreover, SMIRNOFF99Frosst performs better than either GAFF v1.7 or GAFF v2.1 on predicted binding enthalpies, with a mean signed error less than 1 kcal/mol. It should be noted that the binding free energy and enthalpy root mean squared errors (RMSE) and mean signed errors (MSE) for GAFF v2.1 are not substantially worse than those of SMIRNOFF99Frosst, and GAFF v2.1 has statistically significant better correlations with the the experimental data. GAFF v2.1 has excellent agreement with experiment on predicted binding entropy, followed by SMIRNOFF99Frosst and then GAFF v1.7. Taken together, these results support the notion that a force field with many fewer parameters can provide competitive performance. The reduction in the number of parameters, and the simplification of the force field specification, will make it easier to iteratively refine and optimize SMIRNOFF99Frosst against experimental data and the results of quantum mechanical calculations.</p>
<p>However, both SMIRNOFF99Frosst and GAFF v1.7 result in excessively flexible representations of the cyclodextrin hosts, as detailed below. Cézard, <em>et al.</em> present strong NMR evidence that the vicinal <sup>3</sup>J H5–H6′ (atom names H5–H62 in Figure <a href="#fig:atom-names">2</a>) and <sup>3</sup>J H5-H6″(atom names H5–H61) coupling show minimal fluctuation in distance over a number of timescales, suggesting little change in the population of rotamers <span class="citation" data-cites="j7JJedX4">[<a href="#ref-j7JJedX4" role="doc-biblioref">50</a>]</span>. This is also evident in X-ray structures, where the rigidity of the cyclodextrin ring is retained as long as water is present in the cavity and the torsional angles between adjacent glucose units show little variance (0.3–0.6°) across different crystal structures <span class="citation" data-cites="j3qbz56S">[<a href="#ref-j3qbz56S" role="doc-biblioref">51</a>]</span>. The combination of X-ray and NMR data suggest that the specialized q4md-CD <span class="citation" data-cites="j7JJedX4">[<a href="#ref-j7JJedX4" role="doc-biblioref">50</a>]</span> force field, and the rigid GAFF v2.1 <span class="citation" data-cites="HVgz5rZq">[<a href="#ref-HVgz5rZq" role="doc-biblioref">33</a>]</span> force field, better model the flexibility of the CD cavity. The CHARMM36 force field displays similar structural dynamics to q4md-CD, with certain GROMOS force fields even more rigid than those <span class="citation" data-cites="jdi722iE">[<a href="#ref-jdi722iE" role="doc-biblioref">52</a>]</span>. The present results suggest that, as SMIRNOFF99Frosst is further developed, it will be important to include sugars and other carbohydrates in the training sets used to develop parameters. Unfortunately, it may be challenging to find the types of high quality experimental data typically used to train force fields—heats of vaporization, heats of mixing, hydration free energies, and partition coefficients, among others—for biologically relevant sugars. Proper accounting of sugars, and protein-sugar interactions, will be especially useful for modeling physiologically relevant protein structures, such as proteoglycans and glycopeptides.</p>
<p>The greater rigidity of the cyclodextrins when simulated with GAFF v2.1 may contribute to its tendency to generate greater binding affinities and more negative enthalpies than the other two force fields, as a more rigid host may avoid an energy penalty associated with flipping the glucose residues out of the binding cavity to accommodate a guest molecule. The better preorganized cavity might also relate to the uniformly higher correlations between calculation and experiment for GAFF v2.1. On the other hand, it is perhaps unexpected that this force field which best represents the conformational preferences of the cyclodextrin yields consistently too negative binding free energies and enthalpies. It is worth noting the magnitude of these effects will depend on the guest parameters, as well as water model and ion parameters as well.</p>
<p>More broadly, the results presented in this manuscript further demonstrate that host-guest binding thermodynamics can be used to benchmark force fields, to help diagnose issues with parameters applied to specific functional groups, and to suggest directions for improvements. We are therefore continuing to build out experimental host-guest datasets tuned for this purpose, and to further streamline host-guest binding thermodynamics calculations so that they may be incorporated into automated tools to optimize force field parameters.</p>
<h2 id="code-and-data-availability">Code and data availability</h2>
<ul>
<li><a href="https://github.com/slochower/smirnoff-host-guest">GitHub repository</a> used to convert AMBER input files from GAFF force field to SMIRNOFF99Frosst.</li>
<li><a href="https://github.com/slochower/smirnoff-host-guest-simulations-data">GitHub repository</a> for setting up the attach-pull-release calculations using <code>paprika</code> version 0.0.3.</li>
<li><a href="https://github.com/slochower/smirnoff-host-guest-simulations">GitHub repository</a> for analyzing the simulations and generating the plots in this manuscript.</li>
<li><a href="https://github.com/openforcefield/openforcefield">GitHub repository</a> for the Open Force Field group containing the toolkit and force field XML file.</li>
</ul>
<h2 id="author-contributions">Author contributions</h2>
<p>Conceptualization: DRS, NMH, DLM, JDC, MKG; Methodology: DRS, NMH; Software: DRS, NMH; Formal Analysis: DRS, NMH, JDC, MKG; Investigation: DRS, NMH; Resources: MKG, JDC; Data Curation: DRS, NMH; Writing-Original Draft: DRS; Writing - Review and Editing: DRS, NMH, JDC, MKG, LPW; Visualization: DRS; Supervision: DLM, JDC, MKG; Project Administration: MKG; Funding Acquisition: MKG.</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>This work was funded in part by grant GM061300 to MKG from the National Institute of General Medical Sciences of the NIH. JDC was funded in part by grants R01 GM121505 and R01 GM124270 from the National Institute of General Medical Sciences of the NIH and P30 CA008748 from the National Cancer Institute of the NIH. The contents of this publication are solely the responsibility of the authors and do not necessarily represent the official views of the NIH.</p>
<h2 id="disclosures">Disclosures</h2>
<p>The authors declare the following competing financial interest(s): MKG has an equity interest in and is a cofounder and scientific advisor of VeraChem LLC. JDC is a member of the Scientific Advisory Board of OpenEye Scientific Software. The Chodera laboratory receives or has received funding from multiple sources, including the National Institutes of Health, the National Science Foundation, the Parker Institute for Cancer Immunotherapy, Relay Therapeutics, Entasis Therapeutics, Silicon Therapeutics, EMD Serono (Merck KGaA), AstraZeneca, XtalPi, the Molecular Sciences Software Institute, the Starr Cancer Consortium, the Open Force Field Consortium, Cycle for Survival, a Louis V. Gerstner Young Investigator Award, the Einstein Foundation, and the Sloan Kettering Institute. A complete funding history for the Chodera lab can be found at http://choderalab.org/funding.</p>
<h2 id="list-of-abbreviations">List of abbreviations</h2>
<p>APR, attach-pull-release; CD, cyclodextrin; GAFF, Generalized AMBER Force Field</p>
<h2 id="references" class="page_break_before">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" role="doc-bibliography">
<div id="ref-9MLPuYAQ">
<p>1. <strong>Overview of the SAMPL6 host–guest binding affinity prediction challenge</strong><br />
Andrea Rizzi, Steven Murkli, John N. McNeill, Wei Yao, Matthew Sullivan, Michael K. Gilson, Michael W. Chiu, Lyle Isaacs, Bruce C. Gibb, David L. Mobley, John D. Chodera<br />
<em>Journal of Computer-Aided Molecular Design</em> (2018-10) <a href="https://doi.org/gfpzh5">https://doi.org/gfpzh5</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-018-0170-6">10.1007/s10822-018-0170-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30415285">30415285</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6301044">PMC6301044</a></p>
</div>
<div id="ref-1CuDE1c2r">
<p>2. <strong>Attach-Pull-Release Calculations of Ligand Binding and Conformational Changes on the First BRD4 Bromodomain</strong><br />
Germano Heinzelmann, Niel M. Henriksen, Michael K. Gilson<br />
<em>Journal of Chemical Theory and Computation</em> (2017-05-31) <a href="https://doi.org/gbpj4m">https://doi.org/gbpj4m</a><br />
DOI: <a href="https://doi.org/10.1021/acs.jctc.7b00275">10.1021/acs.jctc.7b00275</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28564537">28564537</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5541932">PMC5541932</a></p>
</div>
<div id="ref-mdqdy96N">
<p>3. <strong>Computation of protein–ligand binding free energies using quantum mechanical bespoke force fields</strong><br />
Daniel J. Cole, Israel Cabeza de Vaca, William L. Jorgensen<br />
<em>MedChemComm</em> (2019) <a href="https://doi.org/gfz353">https://doi.org/gfz353</a><br />
DOI: <a href="https://doi.org/10.1039/c9md00017h">10.1039/c9md00017h</a></p>
</div>
<div id="ref-15iR76Uc">
<p>4. <strong>Predictions of Ligand Selectivity from Absolute Binding Free Energy Calculations</strong><br />
Matteo Aldeghi, Alexander Heifetz, Michael J. Bodkin, Stefan Knapp, Philip C. Biggin<br />
<em>Journal of the American Chemical Society</em> (2017-01-09) <a href="https://doi.org/f9kr6h">https://doi.org/f9kr6h</a><br />
DOI: <a href="https://doi.org/10.1021/jacs.6b11467">10.1021/jacs.6b11467</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28009512">28009512</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5253712">PMC5253712</a></p>
</div>
<div id="ref-HQi6ihVB">
<p>5. <strong>Accurate and Reliable Prediction of Relative Ligand Binding Potency in Prospective Drug Discovery by Way of a Modern Free-Energy Calculation Protocol and Force Field</strong><br />
Lingle Wang, Yujie Wu, Yuqing Deng, Byungchan Kim, Levi Pierce, Goran Krilov, Dmitry Lupyan, Shaughnessy Robinson, Markus K. Dahlgren, Jeremy Greenwood, … Robert Abel<br />
<em>Journal of the American Chemical Society</em> (2015-02-12) <a href="https://doi.org/f64pdz">https://doi.org/f64pdz</a><br />
DOI: <a href="https://doi.org/10.1021/ja512751q">10.1021/ja512751q</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25625324">25625324</a></p>
</div>
<div id="ref-B182mNck">
<p>6. <strong>OPLS3e: Extending Force Field Coverage for Drug-Like Small Molecules</strong><br />
Katarina Roos, Chuanjie Wu, Wolfgang Damm, Mark Reboul, James M. Stevenson, Chao Lu, Markus K. Dahlgren, Sayan Mondal, Wei Chen, Lingle Wang, … Edward D. Harder<br />
<em>Journal of Chemical Theory and Computation</em> (2019-02-15) <a href="https://doi.org/gfvpnf">https://doi.org/gfvpnf</a><br />
DOI: <a href="https://doi.org/10.1021/acs.jctc.8b01026">10.1021/acs.jctc.8b01026</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30768902">30768902</a></p>
</div>
<div id="ref-166JLvkLU">
<p>7. <strong>Validation of AMBER/GAFF for Relative Free Energy Calculations</strong><br />
Lin Song, Tai-Sung Lee, Chun Zhu, Darrin M. York, Kenneth M. Merz Jr.<br />
<em>American Chemical Society (ACS)</em> (2019-02-04) <a href="https://doi.org/gf22kq">https://doi.org/gf22kq</a><br />
DOI: <a href="https://doi.org/10.26434/chemrxiv.7653434">10.26434/chemrxiv.7653434</a></p>
</div>
<div id="ref-BGsUYQln">
<p>8. <strong>Overview of the SAMPL5 host–guest challenge: Are we doing better?</strong><br />
Jian Yin, Niel M. Henriksen, David R. Slochower, Michael R. Shirts, Michael W. Chiu, David L. Mobley, Michael K. Gilson<br />
<em>Journal of Computer-Aided Molecular Design</em> (2016-09-22) <a href="https://doi.org/f9m82x">https://doi.org/f9m82x</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-016-9974-4">10.1007/s10822-016-9974-4</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27658802">27658802</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5241188">PMC5241188</a></p>
</div>
<div id="ref-rOVoXhRJ">
<p>9. <strong>The Movable Type Method Applied to Protein–Ligand Binding</strong><br />
Zheng Zheng, Melek N. Ucisik, Kenneth M. Merz<br />
<em>Journal of Chemical Theory and Computation</em> (2013-11-26) <a href="https://doi.org/f5kjt2">https://doi.org/f5kjt2</a><br />
DOI: <a href="https://doi.org/10.1021/ct4005992">10.1021/ct4005992</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24535920">24535920</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3924725">PMC3924725</a></p>
</div>
<div id="ref-Dn1HX5lD">
<p>10. <strong>Blinded predictions of standard binding free energies: lessons learned from the SAMPL6 challenge</strong><br />
Michail Papadourakis, Stefano Bosisio, Julien Michel<br />
<em>Journal of Computer-Aided Molecular Design</em> (2018-08-29) <a href="https://doi.org/gfpbrc">https://doi.org/gfpbrc</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-018-0154-6">10.1007/s10822-018-0154-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30159717">30159717</a></p>
</div>
<div id="ref-kj7fJ3fg">
<p>11. <strong>Resolving the problem of trapped water in binding cavities: prediction of host–guest binding free energies in the SAMPL5 challenge by funnel metadynamics</strong><br />
Soumendranath Bhakat, Pär Söderhjelm<br />
<em>Journal of Computer-Aided Molecular Design</em> (2016-08-29) <a href="https://doi.org/f9m894">https://doi.org/f9m894</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-016-9948-6">10.1007/s10822-016-9948-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27573983">27573983</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5239820">PMC5239820</a></p>
</div>
<div id="ref-l8KUmyk3">
<p>12. <strong>Absolute binding free energies for octa-acids and guests in SAMPL5</strong><br />
Florentina Tofoleanu, Juyong Lee, Frank C. Pickard IV, Gerhard König, Jing Huang, Minkyung Baek, Chaok Seok, Bernard R. Brooks<br />
<em>Journal of Computer-Aided Molecular Design</em> (2016-09-30) <a href="https://doi.org/f9m9c2">https://doi.org/f9m9c2</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-016-9965-5">10.1007/s10822-016-9965-5</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27696242">27696242</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6472255">PMC6472255</a></p>
</div>
<div id="ref-1935a9V0d">
<p>13. <strong>Computational Calorimetry: High-Precision Calculation of Host–Guest Binding Thermodynamics</strong><br />
Niel M. Henriksen, Andrew T. Fenley, Michael K. Gilson<br />
<em>Journal of Chemical Theory and Computation</em> (2015-08-26) <a href="https://doi.org/f7q3mj">https://doi.org/f7q3mj</a><br />
DOI: <a href="https://doi.org/10.1021/acs.jctc.5b00405">10.1021/acs.jctc.5b00405</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26523125">26523125</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4614838">PMC4614838</a></p>
</div>
<div id="ref-ku7PPPs">
<p>14. <strong>Prediction of CB[8] host–guest binding free energies in SAMPL6 using the double-decoupling method</strong><br />
Kyungreem Han, Phillip S. Hudson, Michael R. Jones, Naohiro Nishikawa, Florentina Tofoleanu, Bernard R. Brooks<br />
<em>Journal of Computer-Aided Molecular Design</em> (2018-08-06) <a href="https://doi.org/gfpzp2">https://doi.org/gfpzp2</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-018-0144-8">10.1007/s10822-018-0144-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30084077">30084077</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6347468">PMC6347468</a></p>
</div>
<div id="ref-ScmrI810">
<p>15. <strong>Comparison of the umbrella sampling and the double decoupling method in binding free energy predictions for SAMPL6 octa-acid host–guest challenges</strong><br />
Naohiro Nishikawa, Kyungreem Han, Xiongwu Wu, Florentina Tofoleanu, Bernard R. Brooks<br />
<em>Journal of Computer-Aided Molecular Design</em> (2018-10) <a href="https://doi.org/gfz352">https://doi.org/gfz352</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-018-0166-2">10.1007/s10822-018-0166-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30324304">30324304</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6413509">PMC6413509</a></p>
</div>
<div id="ref-bKuLAjgi">
<p>16. <strong>Detailed potential of mean force studies on host–guest systems from the SAMPL6 challenge</strong><br />
Lin Frank Song, Nupur Bansal, Zheng Zheng, Kenneth M. Merz<br />
<em>Journal of Computer-Aided Molecular Design</em> (2018-08-24) <a href="https://doi.org/gfpw5t">https://doi.org/gfpw5t</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-018-0153-7">10.1007/s10822-018-0153-7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30143917">30143917</a></p>
</div>
<div id="ref-GA1AFcUw">
<p>17. <strong>The SAMPL4 host–guest blind prediction challenge: an overview</strong><br />
Hari S. Muddana, Andrew T. Fenley, David L. Mobley, Michael K. Gilson<br />
<em>Journal of Computer-Aided Molecular Design</em> (2014-03-06) <a href="https://doi.org/f5585s">https://doi.org/f5585s</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-014-9735-1">10.1007/s10822-014-9735-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24599514">24599514</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053502">PMC4053502</a></p>
</div>
<div id="ref-li8h8Q8W">
<p>18. <strong>The statistical-thermodynamic basis for computation of binding affinities: a critical review</strong><br />
M. K. Gilson, J. A. Given, B. L. Bush, J. A. McCammon<br />
<em>Biophysical Journal</em> (1997-03) <a href="https://doi.org/dkv3sk">https://doi.org/dkv3sk</a><br />
DOI: <a href="https://doi.org/10.1016/s0006-3495(97)78756-3">10.1016/s0006-3495(97)78756-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/9138555">9138555</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1184492">PMC1184492</a></p>
</div>
<div id="ref-1HWPx2yMR">
<p>19. <strong>Bridging Calorimetry and Simulation through Precise Calculations of Cucurbituril–Guest Binding Enthalpies</strong><br />
Andrew T. Fenley, Niel M. Henriksen, Hari S. Muddana, Michael K. Gilson<br />
<em>Journal of Chemical Theory and Computation</em> (2014-08) <a href="https://doi.org/f6hv4x">https://doi.org/f6hv4x</a><br />
DOI: <a href="https://doi.org/10.1021/ct5004109">10.1021/ct5004109</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25221445">25221445</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4159218">PMC4159218</a></p>
</div>
<div id="ref-l02WNlWU">
<p>20. <strong>Structures of the Common Cyclodextrins and Their Larger AnaloguesBeyond the Doughnut</strong><br />
Wolfram Saenger, Joël Jacob, Katrin Gessler, Thomas Steiner, Daniel Hoffmann, Haruyo Sanbe, Kyoko Koizumi, Steven M. Smith, Takeshi Takaha<br />
<em>Chemical Reviews</em> (1998-07) <a href="https://doi.org/dgz7hm">https://doi.org/dgz7hm</a><br />
DOI: <a href="https://doi.org/10.1021/cr9700181">10.1021/cr9700181</a></p>
</div>
<div id="ref-q7znTABh">
<p>21. <strong>Toward Expanded Diversity of Host–Guest Interactions via Synthesis and Characterization of Cyclodextrin Derivatives</strong><br />
K. Kellett, S. A. Kantonen, B. M. Duggan, M. K. Gilson<br />
<em>Journal of Solution Chemistry</em> (2018-06-04) <a href="https://doi.org/gfp949">https://doi.org/gfp949</a><br />
DOI: <a href="https://doi.org/10.1007/s10953-018-0769-1">10.1007/s10953-018-0769-1</a></p>
</div>
<div id="ref-19cBeLx8d">
<p>22. <strong>Cyclodextrins and their uses: a review</strong><br />
E. M.Martin Del Valle<br />
<em>Process Biochemistry</em> (2004-05) <a href="https://doi.org/bxthjg">https://doi.org/bxthjg</a><br />
DOI: <a href="https://doi.org/10.1016/s0032-9592(03)00258-9">10.1016/s0032-9592(03)00258-9</a></p>
</div>
<div id="ref-1HYTTY1PU">
<p>23. <strong>Escaping Atom Types in Force Fields Using Direct Chemical Perception</strong><br />
David L. Mobley, Caitlin C. Bannan, Andrea Rizzi, Christopher I. Bayly, John D. Chodera, Victoria T. Lim, Nathan M. Lim, Kyle A. Beauchamp, David R. Slochower, Michael R. Shirts, … Peter K. Eastman<br />
<em>Journal of Chemical Theory and Computation</em> (2018-10-11) <a href="https://doi.org/gffnf3">https://doi.org/gffnf3</a><br />
DOI: <a href="https://doi.org/10.1021/acs.jctc.8b00640">10.1021/acs.jctc.8b00640</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30351006">30351006</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6245550">PMC6245550</a></p>
</div>
<div id="ref-OhpH7vfg">
<p>24. <strong>Open Force Field Initiative</strong><br />
The Open Force Field Initiative<br />
<a href="https://openforcefield.org/">https://openforcefield.org/</a></p>
</div>
<div id="ref-13wrQoS3l">
<p>25. <strong>A Modified Version of the Cornellet al.Force Field with Improved Sugar Pucker Phases and Helical Repeat</strong><br />
Thomas E. Cheatham III, Piotr Cieplak, Peter A. Kollman<br />
<em>Journal of Biomolecular Structure and Dynamics</em> (1999-02) <a href="https://doi.org/gfzp4j">https://doi.org/gfzp4j</a><br />
DOI: <a href="https://doi.org/10.1080/07391102.1999.10508297">10.1080/07391102.1999.10508297</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10217454">10217454</a></p>
</div>
<div id="ref-168lWg0SB">
<p>26. <strong>An Informal AMBER Small Molecule Force Field: parm@Frosst</strong><a href="http://www.ccl.net/cca/data/parm_at_Frosst/">http://www.ccl.net/cca/data/parm_at_Frosst/</a></p>
</div>
<div id="ref-ltrN7ofw">
<p>27. <strong>AMBER 2018</strong><br />
D. A. Case, I. Y. Ben-Shalom, S. R. Brozell, D. S. Cerutti, T. E. III Cheatham, V. W. D. Cruzeiro, T. A. Darden, R. E. Duke, D. Ghoreishi, M. K. Gilson, … P. A. Kollman</p>
</div>
<div id="ref-pOsXFMux">
<p>28. <strong>Daylight&gt;SMIRKS Tutorial</strong><a href="https://daylight.com/dayhtml_tutorials/languages/smirks/">https://daylight.com/dayhtml_tutorials/languages/smirks/</a></p>
</div>
<div id="ref-YmRgHfeU">
<p>29. <strong>Development and testing of a general amber force field</strong><br />
Junmei Wang, Romain M. Wolf, James W. Caldwell, Peter A. Kollman, David A. Case<br />
<em>Journal of Computational Chemistry</em> (2004) <a href="https://doi.org/cdmcnb">https://doi.org/cdmcnb</a><br />
DOI: <a href="https://doi.org/10.1002/jcc.20035">10.1002/jcc.20035</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15116359">15116359</a></p>
</div>
<div id="ref-4LS7HWPh">
<p>30. <strong>A general small molecule force field descended from AMBER99 and parm@Frosst, available in the SMIRNOFF format: openforcefield/smirnoff99Frosst</strong><br />
Open Force Field Initiative<br />
(2019-07-09) <a href="https://github.com/openforcefield/smirnoff99Frosst">https://github.com/openforcefield/smirnoff99Frosst</a></p>
</div>
<div id="ref-XzEPM0UV">
<p>31. <strong>Open Force Field Roadmap</strong><br />
Michael R Shirts, John Damon Chodera, David L Mobley, Michael K Gilson, Lee-Ping Wang<br />
<em>Unpublished</em> (2019) <a href="https://doi.org/gf2stw">https://doi.org/gf2stw</a><br />
DOI: <a href="https://doi.org/10.13140/rg.2.2.27587.86562">10.13140/rg.2.2.27587.86562</a></p>
</div>
<div id="ref-1236RpaUv">
<p>32. <strong>Thermodynamic and Nuclear Magnetic Resonance Study of the Reactions of α- and β-Cyclodextrin with Acids, Aliphatic Amines, and Cyclic Alcohols</strong><br />
Mikhail V. Rekharsky, Martin P. Mayhew, Robert N. Goldberg, Philip D. Ross, Yuko Yamashoji, Yoshihisa Inoue<br />
<em>The Journal of Physical Chemistry B</em> (1997-01) <a href="https://doi.org/cmwfwn">https://doi.org/cmwfwn</a><br />
DOI: <a href="https://doi.org/10.1021/jp962715n">10.1021/jp962715n</a></p>
</div>
<div id="ref-HVgz5rZq">
<p>33. <strong>Evaluating Force Field Performance in Thermodynamic Calculations of Cyclodextrin Host–Guest Binding: Water Models, Partial Charges, and Host Force Field Parameters</strong><br />
Niel M. Henriksen, Michael K. Gilson<br />
<em>Journal of Chemical Theory and Computation</em> (2017-07-11) <a href="https://doi.org/gd2z2t">https://doi.org/gd2z2t</a><br />
DOI: <a href="https://doi.org/10.1021/acs.jctc.7b00359">10.1021/acs.jctc.7b00359</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28696692">28696692</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5606194">PMC5606194</a></p>
</div>
<div id="ref-g7xxBB7m">
<p>34. <strong>Chiral Recognition Thermodynamics of β-Cyclodextrin:  The Thermodynamic Origin of Enantioselectivity and the Enthalpy−Entropy Compensation Effect</strong><br />
Mikhail Rekharsky, Yoshihisa Inoue<br />
<em>Journal of the American Chemical Society</em> (2000-05) <a href="https://doi.org/c2tvdg">https://doi.org/c2tvdg</a><br />
DOI: <a href="https://doi.org/10.1021/ja9921118">10.1021/ja9921118</a></p>
</div>
<div id="ref-LxrgIkt0">
<p>35. <strong>Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</strong><br />
Araz Jakalian, David B. Jack, Christopher I. Bayly<br />
<em>Journal of Computational Chemistry</em> (2002-10-18) <a href="https://doi.org/cktk6g">https://doi.org/cktk6g</a><br />
DOI: <a href="https://doi.org/10.1002/jcc.10128">10.1002/jcc.10128</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12395429">12395429</a></p>
</div>
<div id="ref-BkLoOd0N">
<p>36. <strong>Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</strong><br />
Araz Jakalian, Bruce L. Bush, David B. Jack, Christopher I. Bayly<br />
<em>Journal of Computational Chemistry</em> (2000-01-30) <a href="https://doi.org/cvvpkv">https://doi.org/cvvpkv</a><br />
DOI: <a href="https://doi.org/10.1002/(sici)1096-987x(20000130)21:2%3C132::aid-jcc5%3E3.0.co;2-p">10.1002/(sici)1096-987x(20000130)21:2&lt;132::aid-jcc5&gt;3.0.co;2-p</a></p>
</div>
<div id="ref-chrX4HGh">
<p>37. <strong>Comparison of simple potential functions for simulating liquid water</strong><br />
William L. Jorgensen, Jayaraman Chandrasekhar, Jeffry D. Madura, Roger W. Impey, Michael L. Klein<br />
<em>The Journal of Chemical Physics</em> (1983-07-15) <a href="https://doi.org/dg9sq8">https://doi.org/dg9sq8</a><br />
DOI: <a href="https://doi.org/10.1063/1.445869">10.1063/1.445869</a></p>
</div>
<div id="ref-uZnQBwCt">
<p>38. <strong>Determination of Alkali and Halide Monovalent Ion Parameters for Use in Explicitly Solvated Biomolecular Simulations</strong><br />
In Suk Joung, Thomas E. Cheatham III<br />
<em>The Journal of Physical Chemistry B</em> (2008-07) <a href="https://doi.org/cgwnj7">https://doi.org/cgwnj7</a><br />
DOI: <a href="https://doi.org/10.1021/jp8001614">10.1021/jp8001614</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18593145">18593145</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2652252">PMC2652252</a></p>
</div>
<div id="ref-T7iFp9ei">
<p>39. <strong>Lessons learned from comparing molecular dynamics engines on the SAMPL5 dataset</strong><br />
Michael R. Shirts, Christoph Klein, Jason M. Swails, Jian Yin, Michael K. Gilson, David L. Mobley, David A. Case, Ellen D. Zhong<br />
<em>Journal of Computer-Aided Molecular Design</em> (2016-10-27) <a href="https://doi.org/f9m3wn">https://doi.org/f9m3wn</a><br />
DOI: <a href="https://doi.org/10.1007/s10822-016-9977-1">10.1007/s10822-016-9977-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27787702">27787702</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5581938">PMC5581938</a></p>
</div>
<div id="ref-W9FLyOZr">
<p>40. <strong>Overcoming dissipation in the calculation of standard binding free energies by ligand extraction</strong><br />
Camilo Velez-Vega, Michael K. Gilson<br />
<em>Journal of Computational Chemistry</em> (2013-08) <a href="https://doi.org/gbdv9w">https://doi.org/gbdv9w</a><br />
DOI: <a href="https://doi.org/10.1002/jcc.23398">10.1002/jcc.23398</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24038118">24038118</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3932244">PMC3932244</a></p>
</div>
<div id="ref-79xe5F4u">
<p>41. <strong>Statistical Mechanics of Fluid Mixtures</strong><br />
John G. Kirkwood<br />
<em>The Journal of Chemical Physics</em> (1935-05) <a href="https://doi.org/djkdtx">https://doi.org/djkdtx</a><br />
DOI: <a href="https://doi.org/10.1063/1.1749657">10.1063/1.1749657</a></p>
</div>
<div id="ref-x2mTWVO">
<p>42. <strong>Statistically optimal analysis of samples from multiple equilibrium states</strong><br />
Michael R. Shirts, John D. Chodera<br />
<em>The Journal of Chemical Physics</em> (2008-09-28) <a href="https://doi.org/cvzgk7">https://doi.org/cvzgk7</a><br />
DOI: <a href="https://doi.org/10.1063/1.2978177">10.1063/1.2978177</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19045004">19045004</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2671659">PMC2671659</a></p>
</div>
<div id="ref-tKJug83o">
<p>43. <strong>The Amber Molecular Dynamics Package</strong>(2019) <a href="http://www.ambermd.org">http://www.ambermd.org</a></p>
</div>
<div id="ref-IGZ99oJ8">
<p>44. <strong>Long-Time-Step Molecular Dynamics through Hydrogen Mass Repartitioning</strong><br />
Chad W. Hopkins, Scott Le Grand, Ross C. Walker, Adrian E. Roitberg<br />
<em>Journal of Chemical Theory and Computation</em> (2015-03-30) <a href="https://doi.org/f697mk">https://doi.org/f697mk</a><br />
DOI: <a href="https://doi.org/10.1021/ct5010406">10.1021/ct5010406</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26574392">26574392</a></p>
</div>
<div id="ref-sMVo0eOR">
<p>45. <strong>Nonlinear scaling schemes for Lennard-Jones interactions in free energy calculations</strong><br />
Thomas Steinbrecher, David L. Mobley, David A. Case<br />
<em>The Journal of Chemical Physics</em> (2007-12-07) <a href="https://doi.org/fc7n55">https://doi.org/fc7n55</a><br />
DOI: <a href="https://doi.org/10.1063/1.2799191">10.1063/1.2799191</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18067350">18067350</a></p>
</div>
<div id="ref-htck51Lp">
<p>46. <strong>Error estimates on averages of correlated data</strong><br />
H. Flyvbjerg, H. G. Petersen<br />
<em>The Journal of Chemical Physics</em> (1989-07) <a href="https://doi.org/bjpm5j">https://doi.org/bjpm5j</a><br />
DOI: <a href="https://doi.org/10.1063/1.457480">10.1063/1.457480</a></p>
</div>
<div id="ref-Q2dwH7Dq">
<p>47. <strong>Add hydroxyl hydrogen radii, remove generics, update for 1.0.7 release by davidlmobley · Pull Request #74 · openforcefield/smirnoff99Frosst</strong><br />
GitHub<br />
<a href="https://github.com/openforcefield/smirnoff99Frosst/pull/74">https://github.com/openforcefield/smirnoff99Frosst/pull/74</a></p>
</div>
<div id="ref-cT3I9uDT">
<p>48. <strong>Remove generics, add hydrogen radii, rename ffxml files by davidlmobley · Pull Request #101 · openforcefield/openforcefield</strong><br />
GitHub<br />
<a href="https://github.com/openforcefield/openforcefield/pull/101">https://github.com/openforcefield/openforcefield/pull/101</a></p>
</div>
<div id="ref-10E0XdByV">
<p>49. <strong>Adjust hydroxyl hydrogen to have a small radius, requires more research · Issue #61 · openforcefield/smirnoff99Frosst</strong><br />
GitHub<br />
<a href="https://github.com/openforcefield/smirnoff99Frosst/issues/61">https://github.com/openforcefield/smirnoff99Frosst/issues/61</a></p>
</div>
<div id="ref-j7JJedX4">
<p>50. <strong>Molecular dynamics studies of native and substituted cyclodextrins in different media: 1. Charge derivation and force field performances</strong><br />
Christine Cézard, Xavier Trivelli, Frédéric Aubry, Florence Djedaïni-Pilard, François-Yves Dupradeau<br />
<em>Physical Chemistry Chemical Physics</em> (2011) <a href="https://doi.org/cv3hss">https://doi.org/cv3hss</a><br />
DOI: <a href="https://doi.org/10.1039/c1cp20854c">10.1039/c1cp20854c</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21792425">21792425</a></p>
</div>
<div id="ref-j3qbz56S">
<p>51. <strong>Topography of cyclodextrin inclusion complexes. 8. Crystal and molecular structure of the .alpha.-cyclodextrin-methanol-pentahydrate complex. Disorder in a hydrophobic cage</strong><br />
B. Hingerty, W. Saenger<br />
<em>Journal of the American Chemical Society</em> (1976-05) <a href="https://doi.org/cmfz5v">https://doi.org/cmfz5v</a><br />
DOI: <a href="https://doi.org/10.1021/ja00427a050">10.1021/ja00427a050</a></p>
</div>
<div id="ref-jdi722iE">
<p>52. <strong>Validation and Comparison of Force Fields for Native Cyclodextrins in Aqueous Solution</strong><br />
Julia Gebhardt, Catharina Kleist, Sven Jakobtorweihen, Niels Hansen<br />
<em>The Journal of Physical Chemistry B</em> (2018-01-24) <a href="https://doi.org/gcv3xq">https://doi.org/gcv3xq</a><br />
DOI: <a href="https://doi.org/10.1021/acs.jpcb.7b11808">10.1021/acs.jpcb.7b11808</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29287148">29287148</a></p>
</div>
</div>
<h2 id="SI" class="page_break_before">Supporting Information</h2>
<figure>
<img src="images/charge-schemes.png" id="fig:charge-schemes" style="width:100.0%" alt="" /><figcaption><span>Figure 16:</span> Comparison of AM1-BCC partial atomic charges assigned by running <code>antechamber</code> on a single glucose monomer (A) or on an entire αCD molecule (B) with the option <code>-pl 10</code> to specify the maximum path length used to determine the equivalence of atomic charges.</figcaption>
</figure>
<figure>
<img src="images/TdS.png" id="fig:TdS" style="width:100.0%" alt="" /><figcaption><span>Figure 17:</span> Comparison of calculated absolute binding entropies (−TΔS) with experiment with SMIRNOFF99Frosst parameters (top), GAFF v1.7 parameters (middle), or GAFF v2.1 parameters (bottom) applied to both host and guest. The orange, blue, and purple coloring distinguish the functional group of the guest as an ammonium, alcohol, or carboxylate, respectively.</figcaption>
</figure>
<figure>
<img src="images/alphaCD-structure.png" id="fig:cavity" style="width:3.5in" alt="" /><figcaption><span>Figure 18:</span> The primary (left) and secondary (right) face of αCD.</figcaption>
</figure>
<figure>
<img src="images/by-orientation-comparison.png" id="fig:by-orientation-comparison" style="width:100.0%" alt="" /><figcaption><span>Figure 19:</span> Binding free energies (ΔG) with the primary orientation results colored in blue and secondary orientation results colored in green.</figcaption>
</figure>
<figure>
<img src="images/by-orientation-highlight.png" id="fig:by-orientation-highlight" style="width:100.0%" alt="" /><figcaption><span>Figure 20:</span> Binding free energies (ΔG) replotted from Figure <a href="#fig:by-orientation-comparison">19</a> with points whose difference in binding free energy along either axis is greater than 2 kcal/mol shown in color. Arrows point from primary to secondary. <strong>The arrow direction should match Figure <a href="#fig:by-orientation">4</a>, but it doesn’t.</strong></figcaption>
</figure>
<figure>
<img src="images/dG-by-orientation.png" id="fig:dG-by-orientation" style="width:100.0%" alt="" /><figcaption><span>Figure 21:</span> Comparison of calculated absolute binding free energies (ΔG°) between force field combinations. The orange, blue, and purple coloring distinguish the functional group of the guest as an ammonium, alcohol, or carboxylate, respectively.</figcaption>
</figure>
<figure>
<img src="images/dH-by-orientation.png" id="fig:dH-by-orientation" style="width:100.0%" alt="" /><figcaption><span>Figure 22:</span> Comparison of calculated absolute binding free enthalpies (ΔH) between force field combinations. The orange, blue, and purple coloring distinguish the functional group of the guest as an ammonium, alcohol, or carboxylate, respectively.</figcaption>
</figure>
<figure>
<img src="images/TdS-by-orientation.png" id="fig:TdS-by-orientation" style="width:100.0%" alt="" /><figcaption><span>Figure 23:</span> Comparison of calculated absolute binding entropies (−TΔS°) between force field combinations. The orange, blue, and purple coloring distinguish the functional group of the guest as an ammonium, alcohol, or carboxylate, respectively.</figcaption>
</figure>
<figure>
<img src="images/dG-dH-by-cyclodextrin.png" id="fig:dG-dH-by-cyclodextrin" style="width:100.0%" alt="" /><figcaption><span>Figure 24:</span> Binding free energies (ΔG°) replotted from Figure <a href="#fig:dG-dH">3</a>, with αCD points colored in blue and βCD points in grey (left) or αCD points in grey with βCD points colored in green (right).</figcaption>
</figure>
<figure>
<img src="images/additional-highlights-ammonium.png" id="fig:additional-highlights-ammonium" style="width:100.0%" alt="" /><figcaption><span>Figure 25:</span> Binding free energy (ΔG°) comparisons showing ammonium guests in color and highlighted.</figcaption>
</figure>
<figure>
<img src="images/additional-highlights-alcohols.png" id="fig:additional-highlights-alcohols" style="width:100.0%" alt="" /><figcaption><span>Figure 26:</span> Binding free energy (ΔG°) comparisons showing alcohols guests in color and highlighted.</figcaption>
</figure>
<figure>
<img src="images/additional-highlights-carboxylates.png" id="fig:additional-highlights-carboxylates" style="width:100.0%" alt="" /><figcaption><span>Figure 27:</span> Binding free energy (ΔG°) comparisons showing carboxylates guests in color and highlighted.</figcaption>
</figure>
<figure>
<img src="images/bond-req.png" id="fig:bond-req" style="width:100.0%" alt="" /><figcaption><span>Figure 28:</span> A comparison of bond equilibrium lengths for SMIRNOFF99Frosst, GAFF v1.7, and GAFF v2.1. Atom names refer to Figure <a href="#fig:atom-names">2</a>.</figcaption>
</figure>
<a name="tbl:dG-combined"></a>
<table>
<caption><span>Table 5:</span> Experimental and predicted binding free energies (ΔG°). </caption>
<thead>
<tr class="header">
<th>System</th>
<th>Experimental</th>
<th></th>
<th>SMIRNOFF99Frosst</th>
<th></th>
<th>GAFF v1.7</th>
<th></th>
<th>GAFF v2.1</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td></td>
<td>Mean</td>
<td>SEM</td>
<td>Mean</td>
<td>SEM</td>
<td>Mean</td>
<td>SEM</td>
<td>Mean</td>
<td>SEM</td>
</tr>
<tr class="even">
<td>a-bam</td>
<td>-1.58</td>
<td>0.02</td>
<td>-3.25</td>
<td>0.44</td>
<td>-0.82</td>
<td>0.21</td>
<td>-2.93</td>
<td>0.23</td>
</tr>
<tr class="odd">
<td>a-but</td>
<td>-1.51</td>
<td>0.04</td>
<td>-1.49</td>
<td>0.27</td>
<td>-1.09</td>
<td>0.20</td>
<td>-3.14</td>
<td>0.22</td>
</tr>
<tr class="even">
<td>a-cbu</td>
<td>-2.02</td>
<td>0.02</td>
<td>-1.33</td>
<td>0.19</td>
<td>-0.89</td>
<td>0.22</td>
<td>-3.73</td>
<td>0.21</td>
</tr>
<tr class="odd">
<td>a-chp</td>
<td>-2.51</td>
<td>0.06</td>
<td>-2.38</td>
<td>0.28</td>
<td>-1.69</td>
<td>0.24</td>
<td>-4.11</td>
<td>0.23</td>
</tr>
<tr class="even">
<td>a-coc</td>
<td>-3.23</td>
<td>1.14</td>
<td>-1.78</td>
<td>0.29</td>
<td>-1.86</td>
<td>0.24</td>
<td>-3.35</td>
<td>0.24</td>
</tr>
<tr class="odd">
<td>a-cpe</td>
<td>-2.13</td>
<td>0.02</td>
<td>-1.59</td>
<td>0.25</td>
<td>-1.50</td>
<td>0.29</td>
<td>-3.79</td>
<td>0.22</td>
</tr>
<tr class="even">
<td>a-ham</td>
<td>-3.53</td>
<td>0.00</td>
<td>-3.43</td>
<td>0.30</td>
<td>-3.02</td>
<td>0.19</td>
<td>-5.99</td>
<td>0.17</td>
</tr>
<tr class="odd">
<td>a-hep</td>
<td>-3.99</td>
<td>0.01</td>
<td>-3.95</td>
<td>0.21</td>
<td>-3.93</td>
<td>0.20</td>
<td>-6.23</td>
<td>0.17</td>
</tr>
<tr class="even">
<td>a-hex</td>
<td>-3.38</td>
<td>0.01</td>
<td>-2.70</td>
<td>0.21</td>
<td>-2.92</td>
<td>0.21</td>
<td>-5.27</td>
<td>0.20</td>
</tr>
<tr class="odd">
<td>a-hp6</td>
<td>-3.60</td>
<td>0.00</td>
<td>-3.32</td>
<td>0.23</td>
<td>-3.37</td>
<td>0.18</td>
<td>-5.41</td>
<td>0.18</td>
</tr>
<tr class="even">
<td>a-hpa</td>
<td>-4.14</td>
<td>0.00</td>
<td>-3.02</td>
<td>0.32</td>
<td>-3.16</td>
<td>0.22</td>
<td>-6.03</td>
<td>0.22</td>
</tr>
<tr class="odd">
<td>a-hx2</td>
<td>-3.34</td>
<td>0.01</td>
<td>-2.74</td>
<td>0.20</td>
<td>-2.60</td>
<td>0.19</td>
<td>-4.79</td>
<td>0.18</td>
</tr>
<tr class="even">
<td>a-hx3</td>
<td>-3.01</td>
<td>0.01</td>
<td>-2.39</td>
<td>0.25</td>
<td>-1.58</td>
<td>0.23</td>
<td>-3.94</td>
<td>0.23</td>
</tr>
<tr class="odd">
<td>a-mba</td>
<td>-1.76</td>
<td>0.02</td>
<td>-1.22</td>
<td>0.30</td>
<td>-0.89</td>
<td>0.25</td>
<td>-3.17</td>
<td>0.23</td>
</tr>
<tr class="even">
<td>a-mha</td>
<td>-3.60</td>
<td>0.00</td>
<td>-3.60</td>
<td>0.29</td>
<td>-2.89</td>
<td>0.17</td>
<td>-5.55</td>
<td>0.22</td>
</tr>
<tr class="odd">
<td>a-mhp</td>
<td>-4.17</td>
<td>0.00</td>
<td>-3.98</td>
<td>0.29</td>
<td>-3.82</td>
<td>0.19</td>
<td>-6.23</td>
<td>0.21</td>
</tr>
<tr class="even">
<td>a-nmb</td>
<td>-1.69</td>
<td>0.02</td>
<td>-1.95</td>
<td>0.42</td>
<td>-0.83</td>
<td>0.18</td>
<td>-2.74</td>
<td>0.21</td>
</tr>
<tr class="odd">
<td>a-nmh</td>
<td>-3.52</td>
<td>0.01</td>
<td>-4.15</td>
<td>0.59</td>
<td>-2.92</td>
<td>0.18</td>
<td>-5.56</td>
<td>0.19</td>
</tr>
<tr class="even">
<td>a-oam</td>
<td>-4.61</td>
<td>0.01</td>
<td>-4.68</td>
<td>0.49</td>
<td>-4.33</td>
<td>0.17</td>
<td>-6.99</td>
<td>0.19</td>
</tr>
<tr class="odd">
<td>a-oct</td>
<td>-4.62</td>
<td>0.02</td>
<td>-4.64</td>
<td>0.30</td>
<td>-4.85</td>
<td>0.24</td>
<td>-6.81</td>
<td>0.19</td>
</tr>
<tr class="even">
<td>a-pam</td>
<td>-2.72</td>
<td>0.00</td>
<td>-2.66</td>
<td>0.77</td>
<td>-1.53</td>
<td>0.18</td>
<td>-4.00</td>
<td>0.23</td>
</tr>
<tr class="odd">
<td>a-pnt</td>
<td>-2.60</td>
<td>0.01</td>
<td>-2.56</td>
<td>0.23</td>
<td>-1.74</td>
<td>0.19</td>
<td>-4.14</td>
<td>0.19</td>
</tr>
<tr class="even">
<td>b-ben</td>
<td>-1.64</td>
<td>0.02</td>
<td>-2.85</td>
<td>0.62</td>
<td>-1.83</td>
<td>0.29</td>
<td>-2.45</td>
<td>0.17</td>
</tr>
<tr class="odd">
<td>b-cbu</td>
<td>-1.55</td>
<td>0.17</td>
<td>-1.88</td>
<td>0.20</td>
<td>-1.64</td>
<td>0.36</td>
<td>-2.77</td>
<td>0.17</td>
</tr>
<tr class="even">
<td>b-chp</td>
<td>-4.56</td>
<td>0.01</td>
<td>-3.08</td>
<td>0.25</td>
<td>-2.79</td>
<td>0.34</td>
<td>-6.27</td>
<td>0.23</td>
</tr>
<tr class="odd">
<td>b-coc</td>
<td>-4.97</td>
<td>0.04</td>
<td>-3.28</td>
<td>0.23</td>
<td>-3.36</td>
<td>0.26</td>
<td>-7.13</td>
<td>0.22</td>
</tr>
<tr class="even">
<td>b-cpe</td>
<td>-3.05</td>
<td>0.01</td>
<td>-3.57</td>
<td>0.34</td>
<td>-3.55</td>
<td>0.31</td>
<td>-5.93</td>
<td>0.27</td>
</tr>
<tr class="odd">
<td>b-ham</td>
<td>-2.49</td>
<td>0.08</td>
<td>-2.52</td>
<td>0.20</td>
<td>-2.01</td>
<td>0.26</td>
<td>-4.14</td>
<td>0.19</td>
</tr>
<tr class="even">
<td>b-hep</td>
<td>-3.39</td>
<td>0.18</td>
<td>-3.41</td>
<td>0.28</td>
<td>-3.34</td>
<td>0.35</td>
<td>-4.15</td>
<td>0.23</td>
</tr>
<tr class="odd">
<td>b-hex</td>
<td>-2.28</td>
<td>0.03</td>
<td>-2.93</td>
<td>0.25</td>
<td>-2.47</td>
<td>0.27</td>
<td>-3.59</td>
<td>0.17</td>
</tr>
<tr class="even">
<td>b-m4c</td>
<td>-4.32</td>
<td>0.01</td>
<td>-2.89</td>
<td>0.24</td>
<td>-2.68</td>
<td>0.29</td>
<td>-5.64</td>
<td>0.22</td>
</tr>
<tr class="odd">
<td>b-m4t</td>
<td>-4.54</td>
<td>0.01</td>
<td>-3.82</td>
<td>0.19</td>
<td>-3.50</td>
<td>0.26</td>
<td>-6.33</td>
<td>0.17</td>
</tr>
<tr class="even">
<td>b-mch</td>
<td>-4.18</td>
<td>0.01</td>
<td>-3.69</td>
<td>0.22</td>
<td>-3.31</td>
<td>0.26</td>
<td>-6.07</td>
<td>0.17</td>
</tr>
<tr class="odd">
<td>b-mha</td>
<td>-2.56</td>
<td>0.07</td>
<td>-3.46</td>
<td>0.24</td>
<td>-2.14</td>
<td>0.28</td>
<td>-4.66</td>
<td>0.18</td>
</tr>
<tr class="even">
<td>b-mo3</td>
<td>-2.16</td>
<td>0.01</td>
<td>-2.87</td>
<td>0.38</td>
<td>-2.73</td>
<td>0.41</td>
<td>-2.79</td>
<td>0.20</td>
</tr>
<tr class="odd">
<td>b-mo4</td>
<td>-2.51</td>
<td>0.01</td>
<td>-4.19</td>
<td>0.41</td>
<td>-3.10</td>
<td>0.31</td>
<td>-3.49</td>
<td>0.21</td>
</tr>
<tr class="even">
<td>b-mp3</td>
<td>-1.46</td>
<td>0.04</td>
<td>-3.03</td>
<td>0.27</td>
<td>-2.48</td>
<td>0.28</td>
<td>-3.00</td>
<td>0.19</td>
</tr>
<tr class="odd">
<td>b-mp4</td>
<td>-2.19</td>
<td>0.01</td>
<td>-3.02</td>
<td>0.32</td>
<td>-2.77</td>
<td>0.34</td>
<td>-3.06</td>
<td>0.21</td>
</tr>
<tr class="even">
<td>b-oam</td>
<td>-3.59</td>
<td>0.12</td>
<td>-3.35</td>
<td>0.28</td>
<td>-2.60</td>
<td>0.30</td>
<td>-5.25</td>
<td>0.23</td>
</tr>
<tr class="odd">
<td>b-pb3</td>
<td>-3.52</td>
<td>0.01</td>
<td>-3.49</td>
<td>0.32</td>
<td>-2.87</td>
<td>0.30</td>
<td>-4.58</td>
<td>0.17</td>
</tr>
<tr class="even">
<td>b-pb4</td>
<td>-3.60</td>
<td>0.02</td>
<td>-3.62</td>
<td>0.33</td>
<td>-3.34</td>
<td>0.35</td>
<td>-4.71</td>
<td>0.23</td>
</tr>
<tr class="odd">
<td>b-pha</td>
<td>-1.70</td>
<td>0.05</td>
<td>-3.24</td>
<td>0.31</td>
<td>-2.55</td>
<td>0.29</td>
<td>-3.98</td>
<td>0.19</td>
</tr>
<tr class="even">
<td>b-pnt</td>
<td>-1.27</td>
<td>0.32</td>
<td>-2.22</td>
<td>0.25</td>
<td>-1.73</td>
<td>0.29</td>
<td>-2.00</td>
<td>0.16</td>
</tr>
</tbody>
</table>
<a name="tbl:dH-combined"></a>
<table>
<caption><span>Table 6:</span> Experimental and predicted binding enthalpies (ΔH). </caption>
<thead>
<tr class="header">
<th>System</th>
<th>Experimental</th>
<th></th>
<th>SMIRNOFF99Frosst</th>
<th></th>
<th>GAFF v1.7</th>
<th></th>
<th>GAFF v2.1</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td></td>
<td>Mean</td>
<td>SEM</td>
<td>Mean</td>
<td>SEM</td>
<td>Mean</td>
<td>SEM</td>
<td>Mean</td>
<td>SEM</td>
</tr>
<tr class="even">
<td>a-bam</td>
<td>-2.17</td>
<td>0.05</td>
<td>-0.43</td>
<td>0.28</td>
<td>-0.84</td>
<td>0.59</td>
<td>-3.05</td>
<td>0.38</td>
</tr>
<tr class="odd">
<td>a-but</td>
<td>-2.53</td>
<td>0.12</td>
<td>-0.76</td>
<td>0.59</td>
<td>-1.08</td>
<td>0.37</td>
<td>-4.91</td>
<td>0.42</td>
</tr>
<tr class="even">
<td>a-cbu</td>
<td>-2.75</td>
<td>0.05</td>
<td>-2.08</td>
<td>0.21</td>
<td>-0.71</td>
<td>0.49</td>
<td>-4.94</td>
<td>0.29</td>
</tr>
<tr class="odd">
<td>a-chp</td>
<td>-2.99</td>
<td>0.23</td>
<td>-3.42</td>
<td>0.39</td>
<td>-2.33</td>
<td>0.28</td>
<td>-5.27</td>
<td>0.35</td>
</tr>
<tr class="even">
<td>a-coc</td>
<td>-0.93</td>
<td>0.32</td>
<td>-3.80</td>
<td>0.45</td>
<td>-2.93</td>
<td>0.32</td>
<td>-6.17</td>
<td>0.32</td>
</tr>
<tr class="odd">
<td>a-cpe</td>
<td>-2.74</td>
<td>0.02</td>
<td>-1.93</td>
<td>0.30</td>
<td>-1.06</td>
<td>0.42</td>
<td>-4.86</td>
<td>0.29</td>
</tr>
<tr class="even">
<td>a-ham</td>
<td>-4.19</td>
<td>0.02</td>
<td>-4.02</td>
<td>0.33</td>
<td>-2.33</td>
<td>0.30</td>
<td>-6.91</td>
<td>0.29</td>
</tr>
<tr class="odd">
<td>a-hep</td>
<td>-4.19</td>
<td>0.09</td>
<td>-4.72</td>
<td>0.33</td>
<td>-4.05</td>
<td>0.36</td>
<td>-8.68</td>
<td>0.24</td>
</tr>
<tr class="even">
<td>a-hex</td>
<td>-3.40</td>
<td>0.02</td>
<td>-4.33</td>
<td>0.30</td>
<td>-2.95</td>
<td>0.30</td>
<td>-7.43</td>
<td>0.31</td>
</tr>
<tr class="odd">
<td>a-hp6</td>
<td>-4.48</td>
<td>0.02</td>
<td>-4.86</td>
<td>0.31</td>
<td>-3.73</td>
<td>0.21</td>
<td>-8.24</td>
<td>0.31</td>
</tr>
<tr class="even">
<td>a-hpa</td>
<td>-4.66</td>
<td>0.02</td>
<td>-4.47</td>
<td>0.36</td>
<td>-2.65</td>
<td>0.30</td>
<td>-7.38</td>
<td>0.26</td>
</tr>
<tr class="odd">
<td>a-hx2</td>
<td>-4.12</td>
<td>0.06</td>
<td>-4.24</td>
<td>0.31</td>
<td>-2.35</td>
<td>0.32</td>
<td>-6.56</td>
<td>0.30</td>
</tr>
<tr class="even">
<td>a-hx3</td>
<td>-3.36</td>
<td>0.05</td>
<td>-2.25</td>
<td>0.55</td>
<td>-2.80</td>
<td>0.32</td>
<td>-5.51</td>
<td>0.28</td>
</tr>
<tr class="odd">
<td>a-mba</td>
<td>-2.68</td>
<td>0.07</td>
<td>-0.95</td>
<td>0.41</td>
<td>-0.32</td>
<td>0.37</td>
<td>-3.11</td>
<td>0.36</td>
</tr>
<tr class="even">
<td>a-mha</td>
<td>-4.28</td>
<td>0.02</td>
<td>-3.31</td>
<td>0.50</td>
<td>-2.16</td>
<td>0.25</td>
<td>-6.40</td>
<td>0.34</td>
</tr>
<tr class="odd">
<td>a-mhp</td>
<td>-4.74</td>
<td>0.02</td>
<td>-4.89</td>
<td>0.23</td>
<td>-3.41</td>
<td>0.23</td>
<td>-8.12</td>
<td>0.28</td>
</tr>
<tr class="even">
<td>a-nmb</td>
<td>-2.57</td>
<td>0.06</td>
<td>-1.10</td>
<td>0.28</td>
<td>0.03</td>
<td>0.23</td>
<td>-3.34</td>
<td>0.27</td>
</tr>
<tr class="odd">
<td>a-nmh</td>
<td>-4.20</td>
<td>0.08</td>
<td>-4.20</td>
<td>0.48</td>
<td>-2.54</td>
<td>0.24</td>
<td>-6.74</td>
<td>0.30</td>
</tr>
<tr class="even">
<td>a-oam</td>
<td>-5.46</td>
<td>0.03</td>
<td>-4.93</td>
<td>0.28</td>
<td>-3.73</td>
<td>0.33</td>
<td>-8.02</td>
<td>0.28</td>
</tr>
<tr class="odd">
<td>a-oct</td>
<td>-4.89</td>
<td>0.03</td>
<td>-6.08</td>
<td>0.21</td>
<td>-4.69</td>
<td>0.29</td>
<td>-9.53</td>
<td>0.30</td>
</tr>
<tr class="even">
<td>a-pam</td>
<td>-3.28</td>
<td>0.02</td>
<td>-1.72</td>
<td>0.61</td>
<td>-0.84</td>
<td>0.28</td>
<td>-4.45</td>
<td>0.33</td>
</tr>
<tr class="odd">
<td>a-pnt</td>
<td>-2.75</td>
<td>0.01</td>
<td>-2.05</td>
<td>0.37</td>
<td>-1.62</td>
<td>0.33</td>
<td>-5.99</td>
<td>0.30</td>
</tr>
<tr class="even">
<td>b-ben</td>
<td>-2.51</td>
<td>0.08</td>
<td>-0.45</td>
<td>0.56</td>
<td>-0.76</td>
<td>0.82</td>
<td>-1.30</td>
<td>0.26</td>
</tr>
<tr class="odd">
<td>b-cbu</td>
<td>0.88</td>
<td>0.17</td>
<td>0.05</td>
<td>0.83</td>
<td>-0.19</td>
<td>0.46</td>
<td>0.87</td>
<td>0.29</td>
</tr>
<tr class="even">
<td>b-chp</td>
<td>-2.96</td>
<td>0.01</td>
<td>0.88</td>
<td>0.40</td>
<td>1.82</td>
<td>0.61</td>
<td>-4.39</td>
<td>0.31</td>
</tr>
<tr class="odd">
<td>b-coc</td>
<td>-3.92</td>
<td>0.06</td>
<td>0.80</td>
<td>0.47</td>
<td>0.45</td>
<td>0.98</td>
<td>-5.57</td>
<td>0.44</td>
</tr>
<tr class="even">
<td>b-cpe</td>
<td>-1.09</td>
<td>0.01</td>
<td>1.86</td>
<td>0.37</td>
<td>3.62</td>
<td>0.65</td>
<td>-1.32</td>
<td>0.30</td>
</tr>
<tr class="odd">
<td>b-ham</td>
<td>0.60</td>
<td>0.05</td>
<td>1.66</td>
<td>0.68</td>
<td>2.29</td>
<td>0.78</td>
<td>0.42</td>
<td>0.33</td>
</tr>
<tr class="even">
<td>b-hep</td>
<td>0.42</td>
<td>0.04</td>
<td>-0.05</td>
<td>0.38</td>
<td>1.91</td>
<td>0.29</td>
<td>-0.92</td>
<td>0.27</td>
</tr>
<tr class="odd">
<td>b-hex</td>
<td>1.31</td>
<td>0.04</td>
<td>0.41</td>
<td>0.65</td>
<td>1.30</td>
<td>0.60</td>
<td>-0.21</td>
<td>0.27</td>
</tr>
<tr class="even">
<td>b-m4c</td>
<td>-2.27</td>
<td>0.01</td>
<td>2.18</td>
<td>0.48</td>
<td>2.62</td>
<td>0.55</td>
<td>-3.13</td>
<td>0.30</td>
</tr>
<tr class="odd">
<td>b-m4t</td>
<td>-2.17</td>
<td>0.02</td>
<td>1.26</td>
<td>0.45</td>
<td>2.49</td>
<td>0.51</td>
<td>-3.11</td>
<td>0.29</td>
</tr>
<tr class="even">
<td>b-mch</td>
<td>-2.29</td>
<td>0.03</td>
<td>-0.79</td>
<td>1.08</td>
<td>2.27</td>
<td>0.73</td>
<td>-3.37</td>
<td>0.34</td>
</tr>
<tr class="odd">
<td>b-mha</td>
<td>0.47</td>
<td>0.03</td>
<td>0.53</td>
<td>0.55</td>
<td>2.48</td>
<td>0.64</td>
<td>0.28</td>
<td>0.29</td>
</tr>
<tr class="even">
<td>b-mo3</td>
<td>-2.93</td>
<td>0.03</td>
<td>-2.66</td>
<td>0.46</td>
<td>-0.59</td>
<td>0.45</td>
<td>-2.50</td>
<td>0.28</td>
</tr>
<tr class="odd">
<td>b-mo4</td>
<td>-1.96</td>
<td>0.01</td>
<td>-2.69</td>
<td>0.58</td>
<td>-0.91</td>
<td>0.33</td>
<td>-3.05</td>
<td>0.29</td>
</tr>
<tr class="even">
<td>b-mp3</td>
<td>-2.75</td>
<td>0.13</td>
<td>-1.09</td>
<td>0.68</td>
<td>0.68</td>
<td>0.48</td>
<td>-2.64</td>
<td>0.31</td>
</tr>
<tr class="odd">
<td>b-mp4</td>
<td>-2.89</td>
<td>0.05</td>
<td>-2.84</td>
<td>0.76</td>
<td>0.78</td>
<td>0.60</td>
<td>-2.34</td>
<td>0.28</td>
</tr>
<tr class="even">
<td>b-oam</td>
<td>-0.48</td>
<td>0.03</td>
<td>0.98</td>
<td>0.41</td>
<td>2.66</td>
<td>0.43</td>
<td>-0.52</td>
<td>0.34</td>
</tr>
<tr class="odd">
<td>b-pb3</td>
<td>-2.25</td>
<td>0.01</td>
<td>-1.59</td>
<td>0.94</td>
<td>1.78</td>
<td>0.36</td>
<td>-2.24</td>
<td>0.30</td>
</tr>
<tr class="even">
<td>b-pb4</td>
<td>-2.82</td>
<td>0.01</td>
<td>-0.02</td>
<td>0.62</td>
<td>-1.44</td>
<td>0.78</td>
<td>-3.70</td>
<td>0.29</td>
</tr>
<tr class="odd">
<td>b-pha</td>
<td>-1.79</td>
<td>0.11</td>
<td>-1.10</td>
<td>0.69</td>
<td>-1.34</td>
<td>0.97</td>
<td>-3.45</td>
<td>0.36</td>
</tr>
<tr class="even">
<td>b-pnt</td>
<td>1.89</td>
<td>0.53</td>
<td>-0.79</td>
<td>1.01</td>
<td>-0.51</td>
<td>0.84</td>
<td>0.40</td>
<td>0.31</td>
</tr>
</tbody>
</table>
<a name="tbl:TdS-combined"></a>
<table>
<caption><span>Table 7:</span> Experimental and predicted binding entropies (−TΔS°). </caption>
<thead>
<tr class="header">
<th>System</th>
<th>Experimental</th>
<th></th>
<th>SMIRNOFF99Frosst</th>
<th></th>
<th>GAFF v1.7</th>
<th></th>
<th>GAFF v2.1</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td></td>
<td>Mean</td>
<td>SEM</td>
<td>Mean</td>
<td>SEM</td>
<td>Mean</td>
<td>SEM</td>
<td>Mean</td>
<td>SEM</td>
</tr>
<tr class="even">
<td>a-bam</td>
<td>0.59</td>
<td>0.05</td>
<td>-2.82</td>
<td>0.53</td>
<td>0.02</td>
<td>0.63</td>
<td>0.11</td>
<td>0.45</td>
</tr>
<tr class="odd">
<td>a-but</td>
<td>1.02</td>
<td>0.13</td>
<td>-0.73</td>
<td>0.65</td>
<td>-0.01</td>
<td>0.42</td>
<td>1.77</td>
<td>0.47</td>
</tr>
<tr class="even">
<td>a-cbu</td>
<td>0.73</td>
<td>0.05</td>
<td>0.75</td>
<td>0.28</td>
<td>-0.17</td>
<td>0.54</td>
<td>1.21</td>
<td>0.36</td>
</tr>
<tr class="odd">
<td>a-chp</td>
<td>0.48</td>
<td>0.24</td>
<td>1.04</td>
<td>0.48</td>
<td>0.63</td>
<td>0.37</td>
<td>1.16</td>
<td>0.41</td>
</tr>
<tr class="even">
<td>a-coc</td>
<td>-2.30</td>
<td>1.18</td>
<td>2.02</td>
<td>0.53</td>
<td>1.07</td>
<td>0.40</td>
<td>2.82</td>
<td>0.40</td>
</tr>
<tr class="odd">
<td>a-cpe</td>
<td>0.61</td>
<td>0.03</td>
<td>0.33</td>
<td>0.39</td>
<td>-0.44</td>
<td>0.51</td>
<td>1.07</td>
<td>0.37</td>
</tr>
<tr class="even">
<td>a-ham</td>
<td>0.66</td>
<td>0.02</td>
<td>0.59</td>
<td>0.44</td>
<td>-0.68</td>
<td>0.36</td>
<td>0.92</td>
<td>0.34</td>
</tr>
<tr class="odd">
<td>a-hep</td>
<td>0.20</td>
<td>0.09</td>
<td>0.77</td>
<td>0.39</td>
<td>0.12</td>
<td>0.41</td>
<td>2.45</td>
<td>0.29</td>
</tr>
<tr class="even">
<td>a-hex</td>
<td>0.02</td>
<td>0.02</td>
<td>1.62</td>
<td>0.37</td>
<td>0.03</td>
<td>0.36</td>
<td>2.17</td>
<td>0.37</td>
</tr>
<tr class="odd">
<td>a-hp6</td>
<td>0.88</td>
<td>0.02</td>
<td>1.54</td>
<td>0.38</td>
<td>0.37</td>
<td>0.27</td>
<td>2.84</td>
<td>0.36</td>
</tr>
<tr class="even">
<td>a-hpa</td>
<td>0.52</td>
<td>0.02</td>
<td>1.45</td>
<td>0.48</td>
<td>-0.50</td>
<td>0.37</td>
<td>1.35</td>
<td>0.34</td>
</tr>
<tr class="odd">
<td>a-hx2</td>
<td>0.78</td>
<td>0.06</td>
<td>1.49</td>
<td>0.37</td>
<td>-0.25</td>
<td>0.37</td>
<td>1.77</td>
<td>0.35</td>
</tr>
<tr class="even">
<td>a-hx3</td>
<td>0.35</td>
<td>0.05</td>
<td>-0.13</td>
<td>0.61</td>
<td>1.23</td>
<td>0.40</td>
<td>1.58</td>
<td>0.36</td>
</tr>
<tr class="odd">
<td>a-mba</td>
<td>0.92</td>
<td>0.07</td>
<td>-0.27</td>
<td>0.51</td>
<td>-0.57</td>
<td>0.44</td>
<td>-0.06</td>
<td>0.43</td>
</tr>
<tr class="even">
<td>a-mha</td>
<td>0.68</td>
<td>0.02</td>
<td>-0.29</td>
<td>0.58</td>
<td>-0.73</td>
<td>0.30</td>
<td>0.85</td>
<td>0.40</td>
</tr>
<tr class="odd">
<td>a-mhp</td>
<td>0.57</td>
<td>0.02</td>
<td>0.92</td>
<td>0.37</td>
<td>-0.41</td>
<td>0.30</td>
<td>1.89</td>
<td>0.35</td>
</tr>
<tr class="even">
<td>a-nmb</td>
<td>0.88</td>
<td>0.06</td>
<td>-0.85</td>
<td>0.50</td>
<td>-0.86</td>
<td>0.29</td>
<td>0.61</td>
<td>0.35</td>
</tr>
<tr class="odd">
<td>a-nmh</td>
<td>0.68</td>
<td>0.08</td>
<td>0.05</td>
<td>0.76</td>
<td>-0.38</td>
<td>0.30</td>
<td>1.18</td>
<td>0.36</td>
</tr>
<tr class="even">
<td>a-oam</td>
<td>0.85</td>
<td>0.03</td>
<td>0.25</td>
<td>0.57</td>
<td>-0.60</td>
<td>0.37</td>
<td>1.02</td>
<td>0.34</td>
</tr>
<tr class="odd">
<td>a-oct</td>
<td>0.27</td>
<td>0.04</td>
<td>1.44</td>
<td>0.37</td>
<td>-0.16</td>
<td>0.38</td>
<td>2.72</td>
<td>0.36</td>
</tr>
<tr class="even">
<td>a-pam</td>
<td>0.56</td>
<td>0.02</td>
<td>-0.94</td>
<td>0.98</td>
<td>-0.68</td>
<td>0.34</td>
<td>0.45</td>
<td>0.40</td>
</tr>
<tr class="odd">
<td>a-pnt</td>
<td>0.15</td>
<td>0.01</td>
<td>-0.51</td>
<td>0.43</td>
<td>-0.12</td>
<td>0.38</td>
<td>1.84</td>
<td>0.36</td>
</tr>
<tr class="even">
<td>b-ben</td>
<td>0.87</td>
<td>0.08</td>
<td>-2.40</td>
<td>0.83</td>
<td>-1.07</td>
<td>0.87</td>
<td>-1.15</td>
<td>0.31</td>
</tr>
<tr class="odd">
<td>b-cbu</td>
<td>-2.43</td>
<td>0.24</td>
<td>-1.93</td>
<td>0.85</td>
<td>-1.45</td>
<td>0.58</td>
<td>-3.64</td>
<td>0.34</td>
</tr>
<tr class="even">
<td>b-chp</td>
<td>-1.60</td>
<td>0.01</td>
<td>-3.96</td>
<td>0.47</td>
<td>-4.61</td>
<td>0.69</td>
<td>-1.87</td>
<td>0.39</td>
</tr>
<tr class="odd">
<td>b-coc</td>
<td>-1.05</td>
<td>0.07</td>
<td>-4.08</td>
<td>0.53</td>
<td>-3.81</td>
<td>1.02</td>
<td>-1.56</td>
<td>0.49</td>
</tr>
<tr class="even">
<td>b-cpe</td>
<td>-1.96</td>
<td>0.01</td>
<td>-5.43</td>
<td>0.51</td>
<td>-7.17</td>
<td>0.72</td>
<td>-4.60</td>
<td>0.40</td>
</tr>
<tr class="odd">
<td>b-ham</td>
<td>-3.09</td>
<td>0.09</td>
<td>-4.19</td>
<td>0.71</td>
<td>-4.29</td>
<td>0.82</td>
<td>-4.56</td>
<td>0.38</td>
</tr>
<tr class="even">
<td>b-hep</td>
<td>-3.81</td>
<td>0.18</td>
<td>-3.36</td>
<td>0.47</td>
<td>-5.25</td>
<td>0.45</td>
<td>-3.23</td>
<td>0.35</td>
</tr>
<tr class="odd">
<td>b-hex</td>
<td>-3.59</td>
<td>0.05</td>
<td>-3.34</td>
<td>0.70</td>
<td>-3.77</td>
<td>0.66</td>
<td>-3.38</td>
<td>0.32</td>
</tr>
<tr class="even">
<td>b-m4c</td>
<td>-2.05</td>
<td>0.01</td>
<td>-5.07</td>
<td>0.54</td>
<td>-5.29</td>
<td>0.62</td>
<td>-2.51</td>
<td>0.37</td>
</tr>
<tr class="odd">
<td>b-m4t</td>
<td>-2.37</td>
<td>0.02</td>
<td>-5.08</td>
<td>0.49</td>
<td>-5.99</td>
<td>0.57</td>
<td>-3.22</td>
<td>0.33</td>
</tr>
<tr class="even">
<td>b-mch</td>
<td>-1.89</td>
<td>0.03</td>
<td>-2.90</td>
<td>1.10</td>
<td>-5.58</td>
<td>0.78</td>
<td>-2.70</td>
<td>0.38</td>
</tr>
<tr class="odd">
<td>b-mha</td>
<td>-3.03</td>
<td>0.08</td>
<td>-4.00</td>
<td>0.60</td>
<td>-4.62</td>
<td>0.69</td>
<td>-4.94</td>
<td>0.34</td>
</tr>
<tr class="even">
<td>b-mo3</td>
<td>0.77</td>
<td>0.03</td>
<td>-0.20</td>
<td>0.60</td>
<td>-2.14</td>
<td>0.61</td>
<td>-0.29</td>
<td>0.34</td>
</tr>
<tr class="odd">
<td>b-mo4</td>
<td>-0.55</td>
<td>0.01</td>
<td>-1.50</td>
<td>0.71</td>
<td>-2.19</td>
<td>0.46</td>
<td>-0.45</td>
<td>0.36</td>
</tr>
<tr class="even">
<td>b-mp3</td>
<td>1.29</td>
<td>0.14</td>
<td>-1.94</td>
<td>0.74</td>
<td>-3.16</td>
<td>0.56</td>
<td>-0.35</td>
<td>0.37</td>
</tr>
<tr class="odd">
<td>b-mp4</td>
<td>0.70</td>
<td>0.05</td>
<td>-0.19</td>
<td>0.83</td>
<td>-3.55</td>
<td>0.69</td>
<td>-0.72</td>
<td>0.35</td>
</tr>
<tr class="even">
<td>b-oam</td>
<td>-3.11</td>
<td>0.12</td>
<td>-4.33</td>
<td>0.50</td>
<td>-5.26</td>
<td>0.53</td>
<td>-4.73</td>
<td>0.41</td>
</tr>
<tr class="odd">
<td>b-pb3</td>
<td>-1.27</td>
<td>0.01</td>
<td>-1.90</td>
<td>0.99</td>
<td>-4.64</td>
<td>0.47</td>
<td>-2.34</td>
<td>0.34</td>
</tr>
<tr class="even">
<td>b-pb4</td>
<td>-0.78</td>
<td>0.02</td>
<td>-3.60</td>
<td>0.71</td>
<td>-1.90</td>
<td>0.85</td>
<td>-1.01</td>
<td>0.37</td>
</tr>
<tr class="odd">
<td>b-pha</td>
<td>0.09</td>
<td>0.12</td>
<td>-2.14</td>
<td>0.76</td>
<td>-1.21</td>
<td>1.01</td>
<td>-0.52</td>
<td>0.41</td>
</tr>
<tr class="even">
<td>b-pnt</td>
<td>-3.16</td>
<td>0.62</td>
<td>-1.43</td>
<td>1.04</td>
<td>-1.22</td>
<td>0.89</td>
<td>-2.40</td>
<td>0.35</td>
</tr>
</tbody>
</table>
<a name="tbl:mbar-statistics"></a>
<table style="width:100%;">
<caption><span>Table 8:</span> Predicted thermodynamic properties for SMIRNOFF99Frosst relative to experiment in kcal/mol, analyzed using MBAR. </caption>
<colgroup>
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th></th>
<th>RMSE</th>
<th></th>
<th>MSE</th>
<th></th>
<th>R²</th>
<th></th>
<th>Slope</th>
<th></th>
<th>Intercept</th>
<th></th>
<th>Tau</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>ΔG°</td>
<td>SMIRNOFF99Frosst</td>
<td>0.80</td>
<td>[0.62, 1.00]</td>
<td>-0.04</td>
<td>[-0.28, 0.20]</td>
<td>0.67</td>
<td>[0.22, 0.65]</td>
<td>0.53</td>
<td>[0.33, 0.73]</td>
<td>-1.45</td>
<td>[-0.80, -2.07]</td>
<td>0.48</td>
<td>[0.32, 0.62]</td>
</tr>
<tr class="even">
<td>ΔH</td>
<td>SMIRNOFF99Frosst</td>
<td>1.83</td>
<td>[1.37, 2.28]</td>
<td>0.73</td>
<td>[0.26, 1.24]</td>
<td>0.44</td>
<td>[0.21, 0.66]</td>
<td>0.84</td>
<td>[0.54, 1.18]</td>
<td>0.36</td>
<td>[-0.53, 1.47]</td>
<td>0.52</td>
<td>[0.34, 0.68]</td>
</tr>
<tr class="odd">
<td>−TΔS°</td>
<td>SMIRNOFF99Frosst</td>
<td>1.84</td>
<td>[1.44, 2.25]</td>
<td>-0.76</td>
<td>[-1.26, -0.26]</td>
<td>0.40</td>
<td>[0.15, 0.63]</td>
<td>0.87</td>
<td>[0.50, 1.24]</td>
<td>-0.84</td>
<td>[-1.33, -0.36]</td>
<td>0.32</td>
<td>[0.11, -0.50]</td>
</tr>
</tbody>
</table>
<a name="tbl:overall-ammonium"></a>
<table style="width:100%;">
<caption><span>Table 9:</span> Predicted thermodynamic properties for each force field relative to experiment on ammonium guests. </caption>
<colgroup>
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th></th>
<th>RMSE</th>
<th></th>
<th>MSE</th>
<th></th>
<th>R²</th>
<th></th>
<th>Slope</th>
<th></th>
<th>Intercept</th>
<th></th>
<th>Tau</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>ΔG°</td>
<td>SMIRNOFF99Frosst</td>
<td>0.76</td>
<td>[0.43, 1.11]</td>
<td>-0.10</td>
<td>[-0.54, 0.31]</td>
<td>0.48</td>
<td>[0.07, 0.84]</td>
<td>0.69</td>
<td>[0.19, 1.16]</td>
<td>-1.06</td>
<td>[0.54, -2.77]</td>
<td>0.44</td>
<td>[0.75, 0.04]</td>
</tr>
<tr class="even">
<td>ΔG°</td>
<td>GAFF v1.7</td>
<td>0.77</td>
<td>[0.59, 0.95]</td>
<td>0.69</td>
<td>[0.51, 0.88]</td>
<td>0.90</td>
<td>[0.76, 0.98]</td>
<td>1.08</td>
<td>[0.88, 1.26]</td>
<td>0.95</td>
<td>[1.56, 0.32]</td>
<td>0.74</td>
<td>[0.91, 0.50]</td>
</tr>
<tr class="odd">
<td>ΔG°</td>
<td>GAFF v2.1</td>
<td>1.85</td>
<td>[1.59, 2.09]</td>
<td>-1.79</td>
<td>[-2.04, -1.53]</td>
<td>0.93</td>
<td>[0.83, 0.98]</td>
<td>1.32</td>
<td>[1.13, 1.51]</td>
<td>-0.80</td>
<td>[-0.20, -1.46]</td>
<td>0.76</td>
<td>[0.92, 0.53]</td>
</tr>
<tr class="even">
<td>ΔH</td>
<td>SMIRNOFF99Frosst</td>
<td>1.15</td>
<td>[0.77, 1.51]</td>
<td>0.83</td>
<td>[0.39, 1.27]</td>
<td>0.89</td>
<td>[0.76, 0.97]</td>
<td>1.15</td>
<td>[0.89, 1.53]</td>
<td>1.31</td>
<td>[2.81, 0.38]</td>
<td>0.78</td>
<td>[0.92, 0.56]</td>
</tr>
<tr class="odd">
<td>ΔH</td>
<td>GAFF v1.7</td>
<td>2.12</td>
<td>[1.77, 2.47]</td>
<td>2.02</td>
<td>[1.67, 2.37]</td>
<td>0.92</td>
<td>[0.80, 0.98]</td>
<td>1.09</td>
<td>[0.86, 1.35]</td>
<td>2.29</td>
<td>[3.34, 1.39]</td>
<td>0.75</td>
<td>[0.90, 0.54]</td>
</tr>
<tr class="even">
<td>ΔH</td>
<td>GAFF v2.1</td>
<td>1.90</td>
<td>[1.31, 2.43]</td>
<td>-1.51</td>
<td>[-2.15, -0.88]</td>
<td>0.96</td>
<td>[0.91, 0.99]</td>
<td>1.54</td>
<td>[1.38, 1.83]</td>
<td>0.09</td>
<td>[1.18, -0.44]</td>
<td>0.81</td>
<td>[0.95, 0.62]</td>
</tr>
<tr class="odd">
<td>−TΔS°</td>
<td>SMIRNOFF99Frosst</td>
<td>1.47</td>
<td>[0.90, 2.10]</td>
<td>-0.93</td>
<td>[-1.59, -0.31]</td>
<td>0.65</td>
<td>[0.13, 0.91]</td>
<td>0.99</td>
<td>[0.58, 1.35]</td>
<td>-0.88</td>
<td>[-0.09, -1.66]</td>
<td>0.26</td>
<td>[0.64, -0.26]</td>
</tr>
<tr class="even">
<td>−TΔS°</td>
<td>GAFF v1.7</td>
<td>1.45</td>
<td>[1.14, 1.79]</td>
<td>-1.33</td>
<td>[-1.66, -1.00]</td>
<td>0.88</td>
<td>[0.18, 0.97]</td>
<td>1.04</td>
<td>[-0.02, 1.37]</td>
<td>-1.27</td>
<td>[-0.55, -1.62]</td>
<td>0.28</td>
<td>[0.64, -0.21]</td>
</tr>
<tr class="odd">
<td>−TΔS°</td>
<td>GAFF v2.1</td>
<td>1.04</td>
<td>[0.67, 1.40]</td>
<td>-0.27</td>
<td>[-0.84, 0.26]</td>
<td>0.89</td>
<td>[0.29, 0.98]</td>
<td>1.36</td>
<td>[-0.53, 1.66]</td>
<td>-0.12</td>
<td>[1.16, -0.59]</td>
<td>0.23</td>
<td>[0.62, -0.26]</td>
</tr>
</tbody>
</table>
<a name="tbl:overall-carboxylates"></a>
<table style="width:100%;">
<caption><span>Table 10:</span> Predicted thermodynamic properties for each force field relative to experiment on carboxylate guests. </caption>
<colgroup>
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th></th>
<th>RMSE</th>
<th></th>
<th>MSE</th>
<th></th>
<th>R²</th>
<th></th>
<th>Slope</th>
<th></th>
<th>Intercept</th>
<th></th>
<th>Tau</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>ΔG°</td>
<td>SMIRNOFF99Frosst</td>
<td>0.87</td>
<td>[0.59, 1.16]</td>
<td>-0.36</td>
<td>[-0.74, -0.01]</td>
<td>0.34</td>
<td>[0.02, 0.68]</td>
<td>0.45</td>
<td>[0.11, 0.75]</td>
<td>-1.85</td>
<td>[-0.91, -2.83]</td>
<td>0.40</td>
<td>[0.67, 0.07]</td>
</tr>
<tr class="even">
<td>ΔG°</td>
<td>GAFF v1.7</td>
<td>0.68</td>
<td>[0.49, 0.88]</td>
<td>0.03</td>
<td>[-0.28, 0.34]</td>
<td>0.52</td>
<td>[0.16, 0.80]</td>
<td>0.68</td>
<td>[0.33, 0.97]</td>
<td>-0.84</td>
<td>[0.08, -1.75]</td>
<td>0.53</td>
<td>[0.76, 0.23]</td>
</tr>
<tr class="odd">
<td>ΔG°</td>
<td>GAFF v2.1</td>
<td>1.46</td>
<td>[1.21, 1.71]</td>
<td>-1.36</td>
<td>[-1.61, -1.10]</td>
<td>0.81</td>
<td>[0.61, 0.93]</td>
<td>1.18</td>
<td>[0.85, 1.46]</td>
<td>-0.87</td>
<td>[0.02, -1.74]</td>
<td>0.72</td>
<td>[0.87, 0.54]</td>
</tr>
<tr class="even">
<td>ΔH</td>
<td>SMIRNOFF99Frosst</td>
<td>1.41</td>
<td>[0.94, 1.93]</td>
<td>0.20</td>
<td>[-0.43, 0.84]</td>
<td>0.53</td>
<td>[0.20, 0.79]</td>
<td>0.83</td>
<td>[0.40, 1.53]</td>
<td>-0.14</td>
<td>[2.12, -1.30]</td>
<td>0.59</td>
<td>[0.80, 0.30]</td>
</tr>
<tr class="odd">
<td>ΔH</td>
<td>GAFF v1.7</td>
<td>1.95</td>
<td>[1.34, 2.55]</td>
<td>1.24</td>
<td>[0.55, 1.93]</td>
<td>0.47</td>
<td>[0.13, 0.78]</td>
<td>0.79</td>
<td>[0.32, 1.49]</td>
<td>0.82</td>
<td>[3.10, -0.54]</td>
<td>0.53</td>
<td>[0.75, 0.23]</td>
</tr>
<tr class="even">
<td>ΔH</td>
<td>GAFF v2.1</td>
<td>2.43</td>
<td>[1.75, 3.06]</td>
<td>-1.73</td>
<td>[-2.51, -0.96]</td>
<td>0.69</td>
<td>[0.49, 0.85]</td>
<td>1.40</td>
<td>[0.99, 2.29]</td>
<td>-0.66</td>
<td>[2.15, -1.61]</td>
<td>0.63</td>
<td>[0.82, 0.35]</td>
</tr>
<tr class="odd">
<td>−TΔS°</td>
<td>SMIRNOFF99Frosst</td>
<td>1.73</td>
<td>[1.17, 2.29]</td>
<td>-0.57</td>
<td>[-1.32, 0.16]</td>
<td>0.29</td>
<td>[0.02, 0.61]</td>
<td>0.62</td>
<td>[0.16, 1.09]</td>
<td>-0.68</td>
<td>[0.05, -1.43]</td>
<td>0.27</td>
<td>[0.58, -0.09]</td>
</tr>
<tr class="even">
<td>−TΔS°</td>
<td>GAFF v1.7</td>
<td>2.07</td>
<td>[1.35, 2.76]</td>
<td>-1.22</td>
<td>[-2.00, -0.46]</td>
<td>0.29</td>
<td>[0.00, 0.67]</td>
<td>0.63</td>
<td>[-0.02, 1.18]</td>
<td>-1.31</td>
<td>[-0.58, -2.09]</td>
<td>0.27</td>
<td>[0.58, -0.09]</td>
</tr>
<tr class="odd">
<td>−TΔS°</td>
<td>GAFF v2.1</td>
<td>1.46</td>
<td>[1.12, 1.77]</td>
<td>0.37</td>
<td>[-0.27, 1.00]</td>
<td>0.50</td>
<td>[0.13, 0.76]</td>
<td>0.93</td>
<td>[0.58, 1.30]</td>
<td>0.37</td>
<td>[1.07, -0.34]</td>
<td>0.37</td>
<td>[0.67, -0.01]</td>
</tr>
</tbody>
</table>
<a name="tbl:overall-alcohols"></a>
<table style="width:100%;">
<caption><span>Table 11:</span> Predicted thermodynamic properties for each force field relative to experiment on cyclic alcohol guests. </caption>
<colgroup>
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
<col style="width: 7%" />
</colgroup>
<thead>
<tr class="header">
<th></th>
<th></th>
<th>RMSE</th>
<th></th>
<th>MSE</th>
<th></th>
<th>R²</th>
<th></th>
<th>Slope</th>
<th></th>
<th>Intercept</th>
<th></th>
<th>Tau</th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>ΔG°</td>
<td>SMIRNOFF99Frosst</td>
<td>1.07</td>
<td>[0.66, 1.58]</td>
<td>0.71</td>
<td>[0.22, 1.21]</td>
<td>0.54</td>
<td>[0.09, 0.86]</td>
<td>0.55</td>
<td>[0.20, 0.84]</td>
<td>-0.84</td>
<td>[0.16, -2.09]</td>
<td>0.44</td>
<td>[0.75, 0.02]</td>
</tr>
<tr class="even">
<td>ΔG°</td>
<td>GAFF v1.7</td>
<td>1.22</td>
<td>[0.86, 1.67]</td>
<td>0.93</td>
<td>[0.45, 1.41]</td>
<td>0.56</td>
<td>[0.12, 0.89]</td>
<td>0.59</td>
<td>[0.25, 0.89]</td>
<td>-0.47</td>
<td>[0.64, -1.77]</td>
<td>0.47</td>
<td>[0.78, 0.05]</td>
</tr>
<tr class="odd">
<td>ΔG°</td>
<td>GAFF v2.1</td>
<td>1.80</td>
<td>[1.48, 2.15]</td>
<td>-1.64</td>
<td>[-2.04, -1.14]</td>
<td>0.73</td>
<td>[0.19, 0.98]</td>
<td>1.01</td>
<td>[0.49, 1.27]</td>
<td>-1.63</td>
<td>[-0.67, -3.19]</td>
<td>0.66</td>
<td>[0.89, 0.27]</td>
</tr>
<tr class="even">
<td>ΔH</td>
<td>SMIRNOFF99Frosst</td>
<td>2.88</td>
<td>[1.99, 3.68]</td>
<td>1.66</td>
<td>[0.21, 3.03]</td>
<td>0.09</td>
<td>[0.00, 0.44]</td>
<td>0.07</td>
<td>[-1.28, 1.67]</td>
<td>-0.29</td>
<td>[3.93, -4.06]</td>
<td>0.09</td>
<td>[0.56, -0.35]</td>
</tr>
<tr class="odd">
<td>ΔH</td>
<td>GAFF v1.7</td>
<td>3.63</td>
<td>[2.67, 4.47]</td>
<td>2.66</td>
<td>[1.13, 4.07]</td>
<td>0.10</td>
<td>[0.00, 0.57]</td>
<td>0.12</td>
<td>[-1.09, 2.28]</td>
<td>0.91</td>
<td>[6.66, -2.47]</td>
<td>0.14</td>
<td>[0.60, -0.31]</td>
</tr>
<tr class="even">
<td>ΔH</td>
<td>GAFF v2.1</td>
<td>2.08</td>
<td>[1.18, 3.16]</td>
<td>-1.64</td>
<td>[-2.54, -0.91]</td>
<td>0.54</td>
<td>[0.00, 0.97]</td>
<td>1.08</td>
<td>[-0.37, 1.90]</td>
<td>-1.51</td>
<td>[0.83, -5.50]</td>
<td>0.46</td>
<td>[0.89, -0.09]</td>
</tr>
<tr class="odd">
<td>−TΔS°</td>
<td>SMIRNOFF99Frosst</td>
<td>2.47</td>
<td>[1.62, 3.36]</td>
<td>-0.96</td>
<td>[-2.22, 0.52]</td>
<td>0.40</td>
<td>[0.00, 0.93]</td>
<td>1.18</td>
<td>[-0.45, 2.26]</td>
<td>-0.88</td>
<td>[0.36, -3.60]</td>
<td>0.30</td>
<td>[0.71, -0.20]</td>
</tr>
<tr class="even">
<td>−TΔS°</td>
<td>GAFF v1.7</td>
<td>3.00</td>
<td>[2.07, 3.88]</td>
<td>-1.73</td>
<td>[-3.14, -0.18]</td>
<td>0.37</td>
<td>[0.00, 0.93]</td>
<td>1.23</td>
<td>[-0.38, 2.48]</td>
<td>-1.59</td>
<td>[-0.31, -4.23]</td>
<td>0.29</td>
<td>[0.71, -0.20]</td>
</tr>
<tr class="odd">
<td>−TΔS°</td>
<td>GAFF v2.1</td>
<td>1.80</td>
<td>[0.68, 3.19]</td>
<td>-0.00</td>
<td>[-0.98, 1.27]</td>
<td>0.48</td>
<td>[0.00, 0.97]</td>
<td>1.13</td>
<td>[-0.22, 1.96]</td>
<td>0.08</td>
<td>[1.14, -1.79]</td>
<td>0.46</td>
<td>[0.82, -0.02]</td>
</tr>
</tbody>
</table>
<a name="tbl:S99-vs-GAFF-v1.7"></a>
<table>
<caption><span>Table 12:</span> Dihedral parameter differences between SMIRNOFF99Frosst and GAFF v1.7. Atom names refer to Figure <a href="#fig:atom-names">2</a>. NP: not present. </caption>
<thead>
<tr class="header">
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th>SMIRNOFF99Frosst</th>
<th>GAFF v1.7</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Atom 1</td>
<td>Atom 2</td>
<td>Atom 3</td>
<td>Atom 4</td>
<td>Per</td>
<td>Phase</td>
<td>Height (kcal/mol)</td>
<td>Height (kcal/mol)</td>
</tr>
<tr class="even">
<td>H1</td>
<td>C1</td>
<td>C2</td>
<td>O2</td>
<td>1</td>
<td>0</td>
<td>0.25</td>
<td>NP</td>
</tr>
<tr class="odd">
<td>H1</td>
<td>C1</td>
<td>C2</td>
<td>O2</td>
<td>3</td>
<td>0</td>
<td>0.00</td>
<td>0.16</td>
</tr>
</tbody>
</table>
<a name="tbl:S99-vs-GAFF-v2.1-missing"></a>
<table>
<caption><span>Table 13:</span> Dihedral parameter differences between SMIRNOFF99Frosst and GAFF v2.1, where one dihedral has fewer or more periodicity terms than the corresponding term in the other force field. Atom names refer to <a href="#fig:atom-names">2</a>. NP: not present. </caption>
<thead>
<tr class="header">
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th></th>
<th>SMIRNOFF99Frosst</th>
<th>GAFF v2.1</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Atom 1</td>
<td>Atom 2</td>
<td>Atom 3</td>
<td>Atom 4</td>
<td>Per</td>
<td>Phase</td>
<td>Height (kcal/mol)</td>
<td>Height (kcal/mol)</td>
</tr>
<tr class="even">
<td>C1</td>
<td>C2</td>
<td>O2</td>
<td>HO2</td>
<td>1</td>
<td>0</td>
<td>0.25</td>
<td>NP</td>
</tr>
<tr class="odd">
<td>C1</td>
<td>C2</td>
<td>O2</td>
<td>HO2</td>
<td>3</td>
<td>0</td>
<td>0.16</td>
<td>0.00</td>
</tr>
<tr class="even">
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>C4</td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.00</td>
</tr>
<tr class="odd">
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>C4</td>
<td>2</td>
<td>0</td>
<td>0.10</td>
<td>0.16</td>
</tr>
<tr class="even">
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>C4</td>
<td>3</td>
<td>0</td>
<td>0.38</td>
<td>0.24</td>
</tr>
<tr class="odd">
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>C6</td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.00</td>
</tr>
<tr class="even">
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>C6</td>
<td>2</td>
<td>0</td>
<td>0.10</td>
<td>0.16</td>
</tr>
<tr class="odd">
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>C6</td>
<td>3</td>
<td>0</td>
<td>0.38</td>
<td>0.24</td>
</tr>
<tr class="even">
<td>C2</td>
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.00</td>
</tr>
<tr class="odd">
<td>C2</td>
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>2</td>
<td>0</td>
<td>0.10</td>
<td>0.16</td>
</tr>
<tr class="even">
<td>C2</td>
<td>C1</td>
<td>O5</td>
<td>C5</td>
<td>3</td>
<td>0</td>
<td>0.38</td>
<td>0.24</td>
</tr>
<tr class="odd">
<td>C2</td>
<td>C3</td>
<td>O3</td>
<td>HO3</td>
<td>1</td>
<td>0</td>
<td>0.25</td>
<td>NP</td>
</tr>
<tr class="even">
<td>C2</td>
<td>C3</td>
<td>O3</td>
<td>HO3</td>
<td>3</td>
<td>0</td>
<td>0.16</td>
<td>0.00</td>
</tr>
<tr class="odd">
<td>C5</td>
<td>C6</td>
<td>O6</td>
<td>HO6</td>
<td>1</td>
<td>0</td>
<td>0.25</td>
<td>NP</td>
</tr>
<tr class="even">
<td>C5</td>
<td>C6</td>
<td>O6</td>
<td>HO6</td>
<td>3</td>
<td>0</td>
<td>0.16</td>
<td>0.00</td>
</tr>
<tr class="odd">
<td>H1</td>
<td>C1</td>
<td>C2</td>
<td>O2</td>
<td>1</td>
<td>0</td>
<td>0.25</td>
<td>NP</td>
</tr>
<tr class="even">
<td>H1</td>
<td>C1</td>
<td>C2</td>
<td>O2</td>
<td>3</td>
<td>0</td>
<td>0.00</td>
<td>0.16</td>
</tr>
<tr class="odd">
<td>O1</td>
<td>C1</td>
<td>C2</td>
<td>O2</td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.02</td>
</tr>
<tr class="even">
<td>O1</td>
<td>C1</td>
<td>C2</td>
<td>O2</td>
<td>2</td>
<td>0</td>
<td>1.18</td>
<td>0.00</td>
</tr>
<tr class="odd">
<td>O1</td>
<td>C1</td>
<td>C2</td>
<td>O2</td>
<td>3</td>
<td>0</td>
<td>0.14</td>
<td>1.01</td>
</tr>
<tr class="even">
<td>O2</td>
<td>C2</td>
<td>C1</td>
<td>O5</td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.02</td>
</tr>
<tr class="odd">
<td>O2</td>
<td>C2</td>
<td>C1</td>
<td>O5</td>
<td>2</td>
<td>0</td>
<td>1.18</td>
<td>0.00</td>
</tr>
<tr class="even">
<td>O2</td>
<td>C2</td>
<td>C1</td>
<td>O5</td>
<td>3</td>
<td>0</td>
<td>0.14</td>
<td>1.01</td>
</tr>
<tr class="odd">
<td>O5</td>
<td>C5</td>
<td>C6</td>
<td>O6</td>
<td>1</td>
<td>0</td>
<td>NP</td>
<td>0.02</td>
</tr>
<tr class="even">
<td>O5</td>
<td>C5</td>
<td>C6</td>
<td>O6</td>
<td>2</td>
<td>0</td>
<td>1.18</td>
<td>0.00</td>
</tr>
<tr class="odd">
<td>O5</td>
<td>C5</td>
<td>C6</td>
<td>O6</td>
<td>3</td>
<td>0</td>
<td>0.14</td>
<td>1.01</td>
</tr>
<tr class="even">
<td>HO2</td>
<td>O2</td>
<td>C2</td>
<td>C3</td>
<td>1</td>
<td>0</td>
<td>0.25</td>
<td>NP</td>
</tr>
<tr class="odd">
<td>HO2</td>
<td>O2</td>
<td>C2</td>
<td>C3</td>
<td>3</td>
<td>0</td>
<td>0.16</td>
<td>0.00</td>
</tr>
<tr class="even">
<td>HO3</td>
<td>O3</td>
<td>C3</td>
<td>C4</td>
<td>1</td>
<td>0</td>
<td>0.25</td>
<td>NP</td>
</tr>
<tr class="odd">
<td>HO3</td>
<td>O3</td>
<td>C3</td>
<td>C4</td>
<td>3</td>
<td>0</td>
<td>0.16</td>
<td>0.00</td>
</tr>
</tbody>
</table>
<!-- default theme -->

<style>
    /* import google fonts */
    @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
    @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

    /* -------------------------------------------------- */
    /* global */
    /* -------------------------------------------------- */

    /* all elements */
    * {
        /* force sans-serif font unless specified otherwise */
        font-family: "Helvetica", "Open Sans", sans-serif;
    }

    @media only screen {
        /* "page" element */
        body {
            position: relative;
            box-sizing: border-box;
            font-size: 12pt;
            line-height: 1.5;
            max-width: 8.5in;
            margin: 20px auto;
            padding: 40px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* "page" element */
        body {
            padding: 20px;
            margin: 0;
            border-radius: 0;
            border: none;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
            background: none;
        }
    }

    /* -------------------------------------------------- */
    /* headings */
    /* -------------------------------------------------- */

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
        margin: 20px 0;
        padding: 0;
        font-weight: bold;
    }

    /* biggest heading */
    h1 {
        margin: 40px 0;
        text-align: center;
    }

    /* second biggest heading */
    h2 {
        margin-top: 30px;
        padding-bottom: 5px;
        border-bottom: solid 1px #bdbdbd;
    }

    /* -------------------------------------------------- */
    /* manuscript header */
    /* -------------------------------------------------- */

    /* manuscript title */
    header > h1 {
        margin: 0;
    }

    /* manuscript title caption text (ie "automatically generated on") */
    header + p {
        text-align: center;
        margin-top: 10px;
    }

    /* -------------------------------------------------- */
    /* text elements */
    /* -------------------------------------------------- */

    /* links */
    a {
        color: #2196f3;
    }

    /* normal links (not empty, not button link, not syntax highlighting link) */
    a:not(:empty):not(.button):not(.sourceLine) {
        padding-left: 1px;
        padding-right: 1px;
    }

    /* superscripts and subscripts */
    sub,
    sup {
        /* prevent from affecting line height */
        line-height: 0;
    }

    /* unordered and ordered lists*/
    ul,
    ol {
        padding-left: 20px;
    }

    /* class for styling text semibold */
    .semibold {
        font-weight: 600;
    }

    /* class for styling elements horizontally left aligned */
    .left {
        display: block;
        text-align: left;
        margin-left: auto;
        margin-right: 0;
        justify-content: left;
    }

    /* class for styling elements horizontally centered */
    .center {
        display: block;
        text-align: center;
        margin-left: auto;
        margin-right: auto;
        justify-content: center;
    }

    /* class for styling elements horizontally right aligned */
    .right {
        display: block;
        text-align: right;
        margin-left: 0;
        margin-right: auto;
        justify-content: right;
    }

    /* -------------------------------------------------- */
    /* section elements */
    /* -------------------------------------------------- */

    /* horizontal divider line */
    hr {
        border: none;
        height: 1px;
        background: #bdbdbd;
    }

    /* paragraphs, horizontal dividers, figures, tables, code */
    p,
    hr,
    figure,
    table,
    pre {
        /* treat all as "paragraphs", with consistent vertical margins */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* figures */
    /* -------------------------------------------------- */

    /* figure */
    figure {
        max-width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure caption */
    figcaption {
        padding: 0;
        padding-top: 10px;
    }

    /* figure image element */
    figure img {
        max-width: 100%;
        display: block;
        margin-left: auto;
        margin-right: auto;
    }

    /* figure auto-number */
    img + figcaption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* tables */
    /* -------------------------------------------------- */

    /* table */
    table {
        border-collapse: collapse;
        border-spacing: 0;
        width: 100%;
        margin-left: auto;
        margin-right: auto;
    }

    /* table cells */
    th,
    td {
        border: solid 1px #bdbdbd;
        padding: 10px;
        /* squash table if too wide for page by forcing line breaks */
        word-break: break-all;
        word-break: break-word;
        /* decrease font size */
        font-size: 0.85em;
    }

    /* header row and even rows */
    th,
    tr:nth-child(2n) {
        background-color: #fafafa;
    }

    /* odd rows */
    tr:nth-child(2n + 1) {
        background-color: #ffffff;
    }

    /* table caption */
    caption {
        text-align: left;
        padding: 0;
        padding-bottom: 10px;
    }

    /* table auto-number */
    table > caption > span:first-of-type,
    div.table_wrapper > table > caption > span:first-of-type {
        font-weight: bold;
        margin-right: 5px;
    }

    /* -------------------------------------------------- */
    /* code */
    /* -------------------------------------------------- */

    /* multi-line code block */
    pre {
        padding: 10px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
        break-inside: avoid;
        text-align: left;
    }

    /* inline code, ie code within normal text */
    :not(pre) > code {
        padding: 0 4px;
        background-color: #eeeeee;
        color: #000000;
        border-radius: 5px;
    }

    /* code text */
    /* apply all children, to reach syntax highlighting sub-elements */
    code,
    code * {
        /* force monospace font */
        font-family: "Source Code Pro", "Courier New", monospace;
    }

    /* -------------------------------------------------- */
    /* quotes */
    /* -------------------------------------------------- */

    /* quoted text */
    blockquote {
        margin: 0;
        padding: 0;
        border-left: 4px solid #bdbdbd;
        padding-left: 16px;
        break-inside: avoid;
    }

    /* -------------------------------------------------- */
    /* banners */
    /* -------------------------------------------------- */

    /* info banners */
    .banner {
        box-sizing: border-box;
        display: block;
        position: relative;
        width: 100%;
        margin-top: 20px;
        margin-bottom: 20px;
        padding: 20px;
        text-align: center;
    }

    /* paragraph in banner */
    .banner > p {
        margin: 0;
    }

    /* -------------------------------------------------- */
    /* highlight colors */
    /* -------------------------------------------------- */

    .white {
        background: #ffffff;
    }
    .lightgrey {
        background: #eeeeee;
    }
    .grey {
        background: #757575;
    }
    .darkgrey {
        background: #424242;
    }
    .black {
        background: #000000;
    }
    .lightred {
        background: #ffcdd2;
    }
    .lightyellow {
        background: #ffecb3;
    }
    .lightgreen {
        background: #dcedc8;
    }
    .lightblue {
        background: #e3f2fd;
    }
    .lightpurple {
        background: #f3e5f5;
    }
    .red {
        background: #f44336;
    }
    .orange {
        background: #ff9800;
    }
    .yellow {
        background: #ffeb3b;
    }
    .green {
        background: #4caf50;
    }
    .blue {
        background: #2196f3;
    }
    .purple {
        background: #9c27b0;
    }
    .white,
    .lightgrey,
    .lightred,
    .lightyellow,
    .lightgreen,
    .lightblue,
    .lightpurple,
    .orange,
    .yellow,
    .white a,
    .lightgrey a,
    .lightred a,
    .lightyellow a,
    .lightgreen a,
    .lightblue a,
    .lightpurple a,
    .orange a,
    .yellow a {
        color: #000000;
    }
    .grey,
    .darkgrey,
    .black,
    .red,
    .green,
    .blue,
    .purple,
    .grey a,
    .darkgrey a,
    .black a,
    .red a,
    .green a,
    .blue a,
    .purple a {
        color: #ffffff;
    }

    /* -------------------------------------------------- */
    /* buttons */
    /* -------------------------------------------------- */

    /* class for styling links like buttons */
    .button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        margin: 5px;
        padding: 10px 20px;
        font-size: 0.75em;
        font-weight: 600;
        text-transform: uppercase;
        text-decoration: none;
        letter-spacing: 1px;
        background: none;
        color: #2196f3;
        border: solid 1px #bdbdbd;
        border-radius: 5px;
    }

    /* buttons when hovered */
    .button:hover:not([disabled]),
    .icon_button:hover:not([disabled]) {
        cursor: pointer;
        background: #f5f5f5;
    }

    /* buttons when disabled */
    .button[disabled],
    .icon_button[disabled] {
        opacity: 0.35;
        pointer-events: none;
    }

    /* class for styling buttons containg only single icon */
    .icon_button {
        display: inline-flex;
        justify-content: center;
        align-items: center;
        text-decoration: none;
        margin: 0;
        padding: 0;
        background: none;
        border-radius: 5px;
        border: none;
        width: 20px;
        height: 20px;
        min-width: 20px;
        min-height: 20px;
    }

    /* icon button inner svg image */
    .icon_button > svg {
        height: 16px;
    }

    /* -------------------------------------------------- */
    /* icons */
    /* -------------------------------------------------- */

    /* class for styling icons inline with text */
    .inline_icon {
        height: 1em;
        position: relative;
        top: 0.125em;
    }

    /* -------------------------------------------------- */
    /* print control */
    /* -------------------------------------------------- */

    @media print {
        @page {
            /* suggested printing margin */
            margin: 0.5in;
        }

        /* document and "page" elements */
        html, body {
            margin: 0;
            padding: 0;
            width: 100%;
            height: 100%;
        }

        /* "page" element */
        body {
            font-size: 11pt !important;
            line-height: 1.35;
        }

        /* all headings */
        h1,
        h2,
        h3,
        h4,
        h5,
        h6 {
            margin: 15px 0;
        }

        /* heading 1 */
        h1 {
            font-size: 1.75em;
        }

        /* heading 2 */
        h2 {
            font-size: 1.25em;
            margin-top: 0;
        }

        /* heading 3 */
        h3 {
            font-size: 1.10em;
        }

        /* figures and tables */
        figure, table {
            font-size: 0.85em;
        }

        /* table cells */
        th,
        td {
            padding: 5px;
        }

        /* shrink font awesome icons */
        i.fas,
        i.fab,
        i.far,
        i.fal {
            transform: scale(0.85);
        }

        /* decrease banner margins */
        .banner {
            margin-top: 15px;
            margin-bottom: 15px;
            padding: 15px;
        }

        /* class for centering an element vertically on its own page */
        .page_center {
            margin: auto;
            width: 100%;
            height: 100%;
            display: flex;
            align-items: center;
            vertical-align: middle;
            break-before: page;
            break-after: page;
        }

        /* always insert a page break before the element */
        .page_break_before {
            break-before: page;
        }

        /* always insert a page break after the element */
        .page_break_after {
            break-after: page;
        }

        /* avoid page break before the element */
        .page_break_before_avoid {
            break-before: avoid;
        }

        /* avoid page break after the element */
        .page_break_after_avoid {
            break-after: avoid;
        }

        /* avoid page break inside the element */
        .page_break_inside_avoid {
            break-inside: avoid;
        }
    }

    /* -------------------------------------------------- */
    /* override pandoc css quirks */
    /* -------------------------------------------------- */

    .sourceCode {
        /* prevent unsightly overflow in wide code blocks */
        overflow: auto !important;
    }

    div.sourceCode {
        /* prevent background fill on top-most code block  container */
        background: none !important;
    }

    .sourceCode * {
        /* force consistent line spacing */
        line-height: 1.5 !important;
    }

    div.sourceCode {
        /* style code block margins same as <pre> element */
        margin-top: 20px;
        margin-bottom: 20px;
    }

    /* -------------------------------------------------- */
    /* mathjax */
    /* -------------------------------------------------- */

    /* mathjax containers */
    .math.display > span:not(.MathJax_Preview) {
        /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
        display: flex !important;
        overflow-x: auto !important;
        overflow-y: hidden !important;
        justify-content: center;
        align-items: center;
        margin: 0 !important;
    }

    /* right click menu */
    .MathJax_Menu {
        border-radius: 5px !important;
        border: solid 1px #bdbdbd !important;
        box-shadow: none !important;
    }

    /* equation auto-number */
    span[id^="eq:"] > span.math.display + span {
        font-weight: 600;
    }

    /* equation */
    span[id^="eq:"] > span.math.display > span {
        /* nudge to make room for equation auto-number */
        margin-right: 40px !important;
    }

    /* -------------------------------------------------- */
    /* table scroll plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* table wrapper */
        .table_wrapper {
            /* show scrollbar on tables if necessary to prevent overflow */
            overflow: auto;
            width: 100%;
            margin: 20px 0;
        }

        /* table within table wrapper */
        .table_wrapper table,
        .table_wrapper table * {
            /* don't break table words */
            word-break: normal !important;
        }

        .table_wrapper > table {
            /* move margins from table to table_wrapper to allow margin collapsing */
            margin: 0;
        }
    }

    /* -------------------------------------------------- */
    /* anchors plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anchor button */
        .anchor {
            opacity: 0;
            margin-left: 5px;
        }

        /* anchor buttons within <h2>'s */
        h2 .anchor {
            margin-left: 10px;
        }

        /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
        *:hover > .anchor,
        .anchor:hover,
        .anchor:focus {
            opacity: 1;
        }

        /* anchor button when hovered */
        .anchor:hover {
            cursor: pointer;
        }
    }

    /* always show anchor button on devices with no mouse/hover ability */
    @media (hover: none) {
        .anchor {
            opacity: 1;
        }
    }

    /* always hide anchor button on print */
    @media only print {
        .anchor {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* accordion plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* accordion arrow button */
        .accordion_arrow {
            margin-right: 10px;
        }

        /* arrow icon when <h2> data-collapsed attribute true */
        h2[data-collapsed="true"] > .accordion_arrow > svg {
            transform: rotate(-90deg);
        }

        /* all elements (except <h2>'s) when data-collapsed attribute true */
        *:not(h2)[data-collapsed="true"] {
            display: none;
        }

        /* accordion arrow button when hovered and <h2>'s when hovered */
        .accordion_arrow:hover,
        h2[data-collapsed="true"]:hover,
        h2[data-collapsed="false"]:hover {
            cursor: pointer;
        }
    }

    /* always hide accordion arrow button on print */
    @media only print {
        .accordion_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* tooltips plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* tooltip container */
        #tooltip {
            position: absolute;
            width: 50%;
            min-width: 240px;
            max-width: 75%;
            z-index: 1;
        }

        /* tooltip content */
        #tooltip_content {
            margin-bottom: 5px;
            padding: 20px;
            border-radius: 5px;
            border: solid 1px #bdbdbd;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            background: #ffffff;
            word-break: break-word;
        }

        /* tooltip copy of paragraphs and figures */
        #tooltip_content > p,
        #tooltip_content > figure {
            margin: 0;
            max-height: 320px;
            overflow-y: auto;
        }

        /* tooltip copy of <img> */
        #tooltip_content > figure > img {
            max-height: 260px;
        }

        /* navigation bar */
        #tooltip_nav_bar {
            margin-top: 10px;
            text-align: center;
        }

        /* navigation bar previous/next buton */
        #tooltip_nav_bar > .icon_button {
            position: relative;
            top: 3px;
        }

        /* navigation bar previous button */
        #tooltip_nav_bar > .icon_button:first-of-type {
            margin-right: 5px;
        }

        /* navigation bar next button */
        #tooltip_nav_bar > .icon_button:last-of-type {
            margin-left: 5px;
        }
    }

    /* always hide tooltip on print */
    @media only print {
        #tooltip {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* jump to first plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* jump button */
        .jump_arrow {
            position: relative;
            top: 0.125em;
            margin-right: 5px;
        }
    }

    /* always hide jump button on print */
    @media only print {
        .jump_arrow {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* link highlight plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* anything with data-highlighted attribute true */
        [data-highlighted="true"] {
            background: #ffeb3b;
        }

        /* anything with data-selected attribute true */
        [data-selected="true"] {
            background: #ff8a65 !important;
        }

        /* animation definition for glow */
        @keyframes highlight_glow {
            0% {
                background: none;
            }
            10% {
                background: #bbdefb;
            }
            100% {
                background: none;
            }
        }

        /* anything with data-glow attribute true */
        [data-glow="true"] {
            animation: highlight_glow 2s;
        }
    }

    /* -------------------------------------------------- */
    /* table of contents plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* toc panel */
        #toc_panel {
            box-sizing: border-box;
            position: fixed;
            top: 0;
            left: 0;
            background: #ffffff;
            box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
            z-index: 2;
        }

        /* toc panel when closed */
        #toc_panel[data-open="false"] {
            min-width: 60px;
            width: 60px;
            height: 60px;
            border-right: solid 1px #bdbdbd;
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc panel when open */
        #toc_panel[data-open="true"] {
            min-width: 260px;
            max-width: 480px;
            /* keep panel edge consistent distance away from "page" edge */
            width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
            bottom: 0;
            border-right: solid 1px #bdbdbd;
        }

        /* toc panel header */
        #toc_header {
            box-sizing: border-box;
            display: flex;
            flex-direction: row;
            align-items: center;
            height: 60px;
            margin: 0;
            padding: 20px;
        }

        /* toc panel header when hovered */
        #toc_header:hover {
            cursor: pointer;
        }

        /* toc panel header when panel open */
        #toc_panel[data-open="true"] > #toc_header {
            border-bottom: solid 1px #bdbdbd;
        }

        /* toc open/close header button */
        #toc_button {
            margin-right: 20px;
        }

        /* hide toc list and header text when closed */
        #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
        #toc_panel[data-open="false"] > #toc_list {
            display: none;
        }

        /* toc list of entries */
        #toc_list {
            box-sizing: border-box;
            width: 100%;
            padding: 20px;
            position: absolute;
            top: calc(60px + 1px);
            bottom: 0;
            overflow: auto;
        }

        /* toc entry, link to section in document */
        .toc_link {
            display: block;
            padding: 5px;
            position: relative;
            font-weight: 600;
            text-decoration: none;
        }

        /* toc entry when hovered or when "viewed" */
        .toc_link:hover,
        .toc_link[data-viewing="true"] {
            background: #f5f5f5;
        }

        /* toc entry, level 1 indentation */
        .toc_link[data-level="1"] {
            margin-left: 0;
        }

        /* toc entry, level 2 indentation */
        .toc_link[data-level="2"] {
            margin-left: 20px;
        }

        /* toc entry, level 3 indentation */
        .toc_link[data-level="3"] {
            margin-left: 40px;
        }

        /* toc entry, level 4 indentation */
        .toc_link[data-level="4"] {
            margin-left: 60px;
        }

        /* toc entry bullets */
        #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
            position: absolute;
            left: -15px;
            top: -1px;
            font-size: 1.5em;
        }

        /* toc entry, level 2 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
            content: "\2022";
        }

        /* toc entry, level 3 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
            content: "\25AB";
        }

        /* toc entry, level 4 bullet */
        #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
            content: "-";
        }
    }

    /* when on screen < 8.5in wide */
    @media only screen and (max-width: 8.5in) {
        /* push <body> ("page") element down to make room for toc icon */
        .toc_body_nudge {
            padding-top: 60px;
        }

        /* toc icon when panel closed and not hovered */
        #toc_panel[data-open="false"]:not(:hover) {
            background: rgba(255, 255, 255, 0.75);
        }
    }

    /* always hide toc panel on print */
    @media only print {
        #toc_panel {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* lightbox plugin */
    /* -------------------------------------------------- */

    @media only screen {
        /* regular <img> in document when hovered */
        .lightbox_document_img:hover {
            cursor: pointer;
        }

        .body_no_scroll {
            overflow: hidden !important;
        }

        /* screen overlay */
        #lightbox_overlay {
            display: flex;
            flex-direction: column;
            position: fixed;
            left: 0;
            top: 0;
            right: 0;
            bottom: 0;
            background: rgba(0, 0, 0, 0.75);
            z-index: 3;
        }

        /* middle area containing lightbox image */
        #lightbox_image_container {
            flex-grow: 1;
            display: flex;
            justify-content: center;
            align-items: center;
            overflow: hidden;
            position: relative;
            padding: 20px;
        }

        /* bottom area containing caption */
        #lightbox_bottom_container {
            display: flex;
            justify-content: center;
            align-items: center;
            height: 100px;
            min-height: 100px;
            max-height: 100px;
            background: rgba(0, 0, 0, 0.5);
        }

        /* image number info text box */
        #lightbox_number_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            left: 2px;
            top: 0;
            z-index: 4;
        }

        /* zoom info text box */
        #lightbox_zoom_info {
            position: absolute;
            color: #ffffff;
            font-weight: 600;
            right: 2px;
            top: 0;
            z-index: 4;
        }

        /* copy of image caption */
        #lightbox_caption {
            box-sizing: border-box;
            display: inline-block;
            width: 100%;
            max-height: 100%;
            padding: 10px 0;
            text-align: center;
            overflow-y: auto;
            color: #ffffff;
        }

        /* navigation previous/next button */
        .lightbox_button {
            width: 100px;
            height: 100%;
            min-width: 100px;
            min-height: 100%;
            color: #ffffff;
        }

        /* navigation previous/next button when hovered */
        .lightbox_button:hover {
            background: none !important;
        }

        /* navigation button icon */
        .lightbox_button > svg {
            height: 25px;
        }

        /* figure auto-number */
        #lightbox_caption > span:first-of-type {
            font-weight: bold;
            margin-right: 5px;
        }

        /* lightbox image when hovered */
        #lightbox_img:hover {
            cursor: grab;
        }

        /* lightbox image when grabbed */
        #lightbox_img:active {
            cursor: grabbing;
        }
    }

    /* when on screen < 480px wide */
    @media only screen and (max-width: 480px) {
        /* make navigation buttons skinnier on small screens to make more room for caption text */
        .lightbox_button {
            width: 50px;
            min-width: 50px;
        }
    }

    /* always hide lightbox on print */
    @media only print {
        #lightbox_overlay {
            display: none;
        }
    }

    /* -------------------------------------------------- */
    /* hypothesis (annotations) plugin */
    /* -------------------------------------------------- */

    /* side panel */
    .annotator-frame {
        width: 280px !important;
        z-index: 0 !important;
    }

    /* match highlight color to rest of theme */
    .annotator-highlights-always-on .annotator-hl {
        background-color: #ffeb3b !important;
    }

    /* match focused color to rest of theme */
    .annotator-hl.annotator-hl-focused {
        background-color: #ff8a65 !important;
    }

    /* match bucket bar color to rest of theme */
    .annotator-bucket-bar {
        background: #f5f5f5 !important;
    }

    /* always hide toolbar and tooltip on print */
    @media only print {
        .annotator-frame {
            display: none !important;
        }

        hypothesis-adder {
            display: none !important;
        }
    }
</style>
<!-- table scroll plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows tables that are too wide to fit within
        // the page to have a scrollbar instead of being squashed to fit.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableScroll';

        // default plugin options
        const options = {
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // wrap each table in a container
            const tables = document.querySelectorAll('table');
            for (const table of tables)
                wrapElement(table).classList.add('table_wrapper');
            // table_wrapper CSS class in theme file provides scroll
            // functionality
        }

        // wrap element in div and return div
        function wrapElement(element) {
            const parent = element.parentNode;
            const wrapper = document.createElement('div');
            parent.replaceChild(wrapper, element);
            wrapper.appendChild(element);
            return wrapper;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- anchors plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds an anchor next to each of a certain type
        // of element that provides a human-readable url to that specific
        // item/position in the document (eg "manuscript.html#abstract"). It
        // also makes it such that scrolling out of view of a target removes
        // its identifier from the url.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'anchors';

        // default plugin options
        const options = {
            // which types of elements to add anchors next to, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3, figure, table',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // add anchor to each element of specified types
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements)
                addAnchor(element);

            // attach scroll listener to window
            window.addEventListener('scroll', onScroll);
        }

        // when window is scrolled
        function onScroll() {
            // if url has hash and user has scrolled out of view of hash
            // target, remove hash from url
            const target = getHashTarget();
            if (target) {
                if (
                    target.getBoundingClientRect().top > window.innerHeight ||
                    target.getBoundingClientRect().bottom < 0
                )
                    history.pushState(null, null, ' ');
            }
        }

        // add anchor to element
        function addAnchor(element) {
            let withId; // element with unique id
            let addTo; // element to add anchor button to

            // if figure or table, modify withId and addTo to get expected
            // elements
            if (element.tagName.toLowerCase() === 'figure') {
                withId = element.querySelector('img');
                addTo = element.querySelector('figcaption');
            } else if (element.tagName.toLowerCase() === 'table') {
                withId =
                    element.previousElementSibling ||
                    element.parentNode.previousElementSibling;
                addTo = element.querySelector('caption');
            }

            withId = withId || element;
            addTo = addTo || element;
            const id = withId.id || withId.name || null;

            // do not add anchor if element doesn't have assigned id.
            // id is generated by pandoc and is assumed to be unique and
            // human-readable
            if (!id)
                return;

            // create anchor button
            const anchor = document.createElement('a');
            anchor.innerHTML = document.querySelector('.icon_link').innerHTML;
            anchor.title = 'Link to this part of the document';
            anchor.classList.add('icon_button', 'anchor');
            anchor.dataset.ignore = 'true';
            anchor.href = '#' + id;
            addTo.appendChild(anchor);
        }

        // get element that is target of link or url hash
        function getHashTarget() {
            const hash = window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector(
                '[id="' + id + '"], [name="' + id + '"]'
            );
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (hash.indexOf('#fig:') === 0)
                target = target.parentNode;
            if (hash.indexOf('#tbl:') === 0)
                target = target.nextElementSibling;

            return target;
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- link icon -->

<template class="icon_link">
    <!-- modified from: https://fontawesome.com/icons/link -->
    <svg width="16" height="16" viewBox="0 0 512 512">
        <path
            fill="currentColor"
            d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
        ></path>
    </svg>
</template>
<!-- accordion plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows sections of content under <h2> headings
        // to be collapsible.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'accordion';

        // default plugin options
        const options = {
            // whether to always start expanded ('false'), always start
            // collapsed ('true'), or start collapsed when screen small ('auto')
            startCollapsed: 'auto',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <h2> heading
            const headings = document.querySelectorAll('h2');
            for (const heading of headings) {
                addArrow(heading);

                // start expanded/collapsed based on option
                if (
                    options.startCollapsed === 'true' ||
                    (options.startCollapsed === 'auto' && isSmallScreen())
                )
                    collapseHeading(heading);
                else
                    expandHeading(heading);
            }

            // attach hash change listener to window
            window.addEventListener('hashchange', onHashChange);
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                goToElement(target);
        }

        // add arrow to heading
        function addArrow(heading) {
            // add arrow button
            const arrow = document.createElement('button');
            arrow.innerHTML = document.querySelector(
                '.icon_angle_down'
            ).innerHTML;
            arrow.classList.add('icon_button', 'accordion_arrow');
            heading.insertBefore(arrow, heading.firstChild);

            // attach click listener to heading and button
            heading.addEventListener('click', onHeadingClick);
            arrow.addEventListener('click', onArrowClick);
        }

        // determine if on mobile-like device with small screen
        function isSmallScreen() {
            return Math.min(window.innerWidth, window.innerHeight) < 480;
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get element that is target of hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector(
                '[id="' + id + '"], [name="' + id + '"]'
            );
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (hash.indexOf('#fig:') === 0)
                target = target.parentNode;
            if (hash.indexOf('#tbl:') === 0)
                target = target.nextElementSibling;

            return target;
        }

        // when <h2> heading is clicked
        function onHeadingClick(event) {
            // only collapse if <h2> itself is target of click (eg, user did
            // not click on anchor within <h2>)
            if (event.target === this)
                toggleCollapse(this);
        }

        // when arrow button is clicked
        function onArrowClick() {
            toggleCollapse(this.parentNode);
        }

        // collapse section if expanded, expand if collapsed
        function toggleCollapse(heading) {
            if (heading.dataset.collapsed === 'false')
                collapseHeading(heading);
            else
                expandHeading(heading);
        }

        // elements to exclude from collapse, such as table of contents panel,
        // hypothesis panel, etc
        const exclude = '#toc_panel, div.annotator-frame, #lightbox_overlay';

        // collapse section
        function collapseHeading(heading) {
            heading.setAttribute('data-collapsed', 'true');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'true');
        }

        // expand section
        function expandHeading(heading) {
            heading.setAttribute('data-collapsed', 'false');
            const children = getChildren(heading);
            for (const child of children)
                child.setAttribute('data-collapsed', 'false');
        }

        // get list of elements between this <h2> and next <h2> or <h1>
        // ("children" of the <h2> section)
        function getChildren(heading) {
            return nextUntil(heading, 'h2, h1', exclude);
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get list of elements after a start element up to element matching
        // query
        function nextUntil(element, query, exclude) {
            const elements = [];
            while (element = element.nextElementSibling, element) {
                if (element.matches(query))
                    break;
                if (!element.matches(exclude))
                    elements.push(element);
            }
            return elements;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
    <!-- modified from: https://fontawesome.com/icons/angle-down -->
    <svg width="16" height="16" viewBox="0 0 448 512">
        <path
            fill="currentColor"
            d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
        ></path>
    </svg>
</template>
<!-- tooltips plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when the user hovers or
        // focuses a link to a citation or figure, a tooltip appears with a
        // preview of the reference content, along with arrows to navigate
        // between instances of the same reference in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tooltips';

        // default plugin options
        const options = {
            // whether user must click off to close tooltip instead of just
            // un-hovering
            clickClose: 'false',
            // delay (in ms) between opening and closing tooltip
            delay: '100',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach hover and focus listeners to link
                link.addEventListener('mouseover', onLinkHover);
                link.addEventListener('mouseleave', onLinkUnhover);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('touchend', onLinkTouch);
            }

            // attach mouse, key, and resize listeners to window
            window.addEventListener('mousedown', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('keyup', onKeyUp);
            window.addEventListener('resize', onResize);
        }

        // when link is hovered
        function onLinkHover() {
            // function to open tooltip
            const delayOpenTooltip = function() {
                openTooltip(this);
            }.bind(this);

            // run open function after delay
            this.openTooltipTimer = window.setTimeout(
                delayOpenTooltip,
                options.delay
            );
        }

        // when mouse leaves link
        function onLinkUnhover() {
            // cancel opening tooltip
            window.clearTimeout(this.openTooltipTimer);

            // don't close on unhover if option specifies
            if (options.clickClose === 'true')
                return;

            // function to close tooltip
            const delayCloseTooltip = function() {
                // if tooltip open and if mouse isn't over tooltip, close
                const tooltip = document.getElementById('tooltip');
                if (tooltip && !tooltip.matches(':hover'))
                    closeTooltip();
            };

            // run close function after delay
            this.closeTooltipTimer = window.setTimeout(
                delayCloseTooltip,
                options.delay
            );
        }

        // when link is focused (tabbed to)
        function onLinkFocus(event) {
            openTooltip(this);
        }

        // when link is touched on touch screen
        function onLinkTouch(event) {
            // attempt to force hover state on first tap always, and trigger
            // regular link click (and navigation) on second tap
            if (event.target === document.activeElement)
                event.target.click();
            else {
                document.activeElement.blur();
                event.target.focus();
            }
            if (event.cancelable)
                event.preventDefault();
            event.stopPropagation();
            return false;
        }

        // when mouse is clicked anywhere in window
        function onClick(event) {
            closeTooltip();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'tooltip_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'tooltip_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeTooltip();
                    break;
            }
        }

        // when window is resized or zoomed
        function onResize() {
            closeTooltip();
        }

        // get all links of types we wish to handle
        function getLinks() {
            const queries = [];
            // exclude buttons, anchor links, toc links, etc
            const exclude =
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            queries.push('a[href^="#ref-"]' + exclude); // citation links
            queries.push('a[href^="#fig:"]' + exclude); // figure links
            const query = queries.join(', ');
            return document.querySelectorAll(query);
        }

        // get links with same target, get index of link in set, get total
        // same links
        function getSameLinks(link) {
            const sameLinks = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    sameLinks.push(otherLink);
            }

            return {
                elements: sameLinks,
                index: sameLinks.indexOf(link),
                total: sameLinks.length
            };
        }

        // open tooltip
        function openTooltip(link) {
            // delete tooltip if it exists, start fresh
            closeTooltip();

            // make tooltip element
            const tooltip = makeTooltip(link);

            // if source couldn't be found and tooltip not made, exit
            if (!tooltip)
                return;

            // make navbar elements
            const navBar = makeNavBar(link);
            if (navBar)
                tooltip.firstElementChild.appendChild(navBar);

            // attach tooltip to page
            document.body.appendChild(tooltip);

            // position tooltip
            const position = function() {
                positionTooltip(link);
            };
            position();

            // if tooltip contains images, position again after they've loaded
            const imgs = tooltip.querySelectorAll('img');
            for (const img of imgs)
                img.addEventListener('load', position);
        }

        // close (delete) tooltip
        function closeTooltip() {
            const tooltip = document.getElementById('tooltip');
            if (tooltip)
                tooltip.remove();
        }

        // make tooltip
        function makeTooltip(link) {
            // get target element that link points to
            const source = getSource(link);

            // if source can't be found, exit
            if (!source)
                return;

            // create new tooltip
            const tooltip = document.createElement('div');
            tooltip.id = 'tooltip';
            const tooltipContent = document.createElement('div');
            tooltipContent.id = 'tooltip_content';
            tooltip.appendChild(tooltipContent);

            // make copy of source node and put in tooltip
            const sourceCopy = makeCopy(source);
            tooltipContent.appendChild(sourceCopy);

            // attach mouse event listeners
            tooltip.addEventListener('click', onTooltipClick);
            tooltip.addEventListener('mousedown', onTooltipClick);
            tooltip.addEventListener('touchstart', onTooltipClick);
            tooltip.addEventListener('mouseleave', onTooltipUnhover);

            // (for interaction with lightbox plugin)
            // transfer click on tooltip copied img to original img
            const sourceImg = source.querySelector('img');
            const sourceCopyImg = sourceCopy.querySelector('img');
            if (sourceImg && sourceCopyImg) {
                const clickImg = function() {
                    sourceImg.click();
                    closeTooltip();
                };
                sourceCopyImg.addEventListener('click', clickImg);
            }

            return tooltip;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // when tooltip is clicked
        function onTooltipClick(event) {
            // when user clicks on tooltip, stop click from transferring
            // outside of tooltip (eg, click off to close tooltip, or eg click
            // off to unhighlight same refs)
            event.stopPropagation();
        }

        // when tooltip is unhovered
        function onTooltipUnhover(event) {
            if (options.clickClose === 'true')
                return;

            // make sure new mouse/touch/focus no longer over tooltip or any
            // element within it
            const tooltip = document.getElementById('tooltip');
            if (!tooltip)
                return;
            if (this.contains(event.relatedTarget))
                return;

            closeTooltip();
        }

        // make nav bar to go betwen prev/next instances of same reference
        function makeNavBar(link) {
            // find other links to the same source
            const sameLinks = getSameLinks(link);

            // don't show nav bar when singular reference
            if (sameLinks.total <= 1)
                return;

            // find prev/next links with same target
            const prevLink = getPrevLink(link, sameLinks);
            const nextLink = getNextLink(link, sameLinks);

            // create nav bar
            const navBar = document.createElement('div');
            navBar.id = 'tooltip_nav_bar';
            const text = sameLinks.index + 1 + ' of ' + sameLinks.total;

            // create nav bar prev/next buttons
            const prevButton = document.createElement('button');
            const nextButton = document.createElement('button');
            prevButton.id = 'tooltip_prev_button';
            nextButton.id = 'tooltip_next_button';
            prevButton.title =
                'Jump to the previous occurence of this item in the document [←]';
            nextButton.title =
                'Jump to the next occurence of this item in the document [→]';
            prevButton.classList.add('icon_button');
            nextButton.classList.add('icon_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;
            navBar.appendChild(prevButton);
            navBar.appendChild(document.createTextNode(text));
            navBar.appendChild(nextButton);

            // attach click listeners to buttons
            prevButton.addEventListener('click', function() {
                onPrevNextClick(link, prevLink);
            });
            nextButton.addEventListener('click', function() {
                onPrevNextClick(link, nextLink);
            });

            return navBar;
        }

        // get previous link with same target
        function getPrevLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if < 1
            let index;
            if (sameLinks.index - 1 >= 0)
                index = sameLinks.index - 1;
            else
                index = sameLinks.total - 1;
            return sameLinks.elements[index];
        }

        // get next link with same target
        function getNextLink(link, sameLinks) {
            if (!sameLinks)
                sameLinks = getSameLinks(link);
            // wrap index to other side if > total
            let index;
            if (sameLinks.index + 1 <= sameLinks.total - 1)
                index = sameLinks.index + 1;
            else
                index = 0;
            return sameLinks.elements[index];
        }

        // get element that is target of link or url hash
        function getSource(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector(
                '[id="' + id + '"], [name="' + id + '"]'
            );
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (hash.indexOf('#ref-') === 0)
                target = target.querySelector('p');
            else if (hash.indexOf('#fig:') === 0)
                target = target.parentNode;
            else if (hash.indexOf('#tbl:') === 0)
                return;

            return target;
        }

        // when prev/next arrow button is clicked
        function onPrevNextClick(link, prevNextLink) {
            if (link && prevNextLink)
                goToElement(prevNextLink, window.innerHeight * 0.5);
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // determine position to place tooltip based on link position in
        // viewport and tooltip size
        function positionTooltip(link, left, top) {
            const tooltipElement = document.getElementById('tooltip');
            if (!tooltipElement)
                return;

            // get convenient vars for position/dimensions of
            // link/tooltip/page/view
            link = getRectInPage(link);
            const tooltip = getRectInPage(tooltipElement);
            const view = getRectInPage();

            // horizontal positioning
            if (left)
                // use explicit value
                left = left;
            else if (link.left + tooltip.width < view.right)
                // fit tooltip to right of link
                left = link.left;
            else if (link.right - tooltip.width > view.left)
                // fit tooltip to left of link
                left = link.right - tooltip.width;
            // center tooltip in view
            else
                left = (view.right - view.left) / 2 - tooltip.width / 2;

            // vertical positioning
            if (top)
                // use explicit value
                top = top;
            else if (link.top - tooltip.height > view.top)
                // fit tooltip above link
                top = link.top - tooltip.height;
            else if (link.bottom + tooltip.height < view.bottom)
                // fit tooltip below link
                top = link.bottom;
            else {
                // center tooltip in view
                top = view.top + view.height / 2 - tooltip.height / 2;
                // nudge off of link to left/right if possible
                if (link.right + tooltip.width < view.right)
                    left = link.right;
                else if (link.left - tooltip.width > view.left)
                    left = link.left - tooltip.width;
            }

            tooltipElement.style.left = left + 'px';
            tooltipElement.style.top = top + 'px';
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get position of element relative to page
        function getRectInPage(element) {
            const rect = getRectInView(element);
            const body = getRectInView(document.body);

            const newRect = {};
            newRect.left = rect.left - body.left;
            newRect.top = rect.top - body.top;
            newRect.right = rect.right - body.left;
            newRect.bottom = rect.bottom - body.top;
            newRect.width = rect.width;
            newRect.height = rect.height;

            return newRect;
        }

        // (for interaction with accordion plugin)
        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // (for interaction with accordion plugin)
        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- jump to first plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin adds a button next to each reference entry,
        // figure, and table that jumps the page to the first occurrence of a
        // link to that item in the manuscript.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'jumpToFirst';

        // default plugin options
        const options = {
            // whether to add buttons next to reference entries
            references: 'true',
            // whether to add buttons next to figures
            figures: 'true',
            // whether to add buttons next to tables
            tables: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            if (options.references !== 'false')
                makeReferenceButtons();
            if (options.figures !== 'false')
                makeFigureButtons();
            if (options.tables !== 'false')
                makeTableButtons();
        }

        // when jump button clicked
        function onButtonClick() {
            const first = getFirstOccurrence(this.dataset.id);
            if (!first)
                return;

            // update url hash so navigating "back" in history will return
            // user to jump button
            window.location.hash = this.dataset.id;
            // scroll to link
            window.setTimeout(function() {
                goToElement(first, window.innerHeight * 0.5);
            }, 0);
        }

        // get first occurence of link to item in document
        function getFirstOccurrence(id) {
            let query = 'a';
            query += '[href="#' + id + '"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelector(query);
        }

        // add button next to each reference entry
        function makeReferenceButtons() {
            const references = document.querySelectorAll('div[id^="ref-"]');
            for (const reference of references) {
                // get reference id and element to add button to
                const id = reference.id;
                const container = reference.firstElementChild;
                const first = getFirstOccurrence(id);

                // if can't find link to reference, ignore
                if (!first)
                    continue;

                // make jump button
                let button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this reference in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.innerHTML = button.outerHTML + container.innerHTML;
                button = container.firstElementChild;
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeFigureButtons() {
            const figures = document.querySelectorAll('img[id^="fig:"]');
            for (const figure of figures) {
                // get figure id and element to add button to
                const id = figure.id;
                const container = figure.nextElementSibling;
                const first = getFirstOccurrence(id);

                // if can't find link to figure, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this figure in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // add button next to each figure
        function makeTableButtons() {
            const tables = document.querySelectorAll('a[name^="tbl:"]');
            for (const table of tables) {
                // get ref id and element to add button to
                const id = table.name;
                const container = table.nextElementSibling.querySelector(
                    'caption'
                );
                const first = getFirstOccurrence(id);

                // if can't find link to table, ignore
                if (!first)
                    continue;

                // make jump button
                const button = document.createElement('button');
                button.classList.add('icon_button', 'jump_arrow');
                button.title =
                    'Jump to the first occurence of this table in the document';
                button.innerHTML = document.querySelector(
                    '.icon_angle_double_up'
                ).innerHTML;
                button.dataset.id = id;
                button.dataset.ignore = 'true';
                container.insertBefore(button, container.firstElementChild);
                button.addEventListener('click', onButtonClick);
            }
        }

        // scroll to and focus element
        function goToElement(element, offset) {
            // expand accordion section if collapsed
            expandElement(element);
            const y =
                getRectInView(element).top -
                getRectInView(document.documentElement).top -
                (offset || 0);
            // trigger any function listening for "onscroll" event
            window.dispatchEvent(new Event('scroll'));
            window.scrollTo(0, y);
            document.activeElement.blur();
            element.focus();
        }

        // get position/dimensions of element or viewport
        function getRectInView(element) {
            let rect = {};
            rect.left = 0;
            rect.top = 0;
            rect.right = document.documentElement.clientWidth;
            rect.bottom = document.documentElement.clientHeight;
            let style = {};

            if (element instanceof HTMLElement) {
                rect = element.getBoundingClientRect();
                style = window.getComputedStyle(element);
            }

            const margin = {};
            margin.left = parseFloat(style.marginLeftWidth) || 0;
            margin.top = parseFloat(style.marginTopWidth) || 0;
            margin.right = parseFloat(style.marginRightWidth) || 0;
            margin.bottom = parseFloat(style.marginBottomWidth) || 0;

            const border = {};
            border.left = parseFloat(style.borderLeftWidth) || 0;
            border.top = parseFloat(style.borderTopWidth) || 0;
            border.right = parseFloat(style.borderRightWidth) || 0;
            border.bottom = parseFloat(style.borderBottomWidth) || 0;

            const newRect = {};
            newRect.left = rect.left + margin.left + border.left;
            newRect.top = rect.top + margin.top + border.top;
            newRect.right = rect.right + margin.right + border.right;
            newRect.bottom = rect.bottom + margin.bottom + border.bottom;
            newRect.width = newRect.right - newRect.left;
            newRect.height = newRect.bottom - newRect.top;

            return newRect;
        }

        // get closest element before specified element that matches query
        function firstBefore(element, query) {
            while (
                element &&
                element !== document.body &&
                !element.matches(query)
            )
                element = element.previousElementSibling || element.parentNode;

            return element;
        }

        // check if element is part of collapsed heading
        function isCollapsed(element) {
            while (element && element !== document.body) {
                if (element.dataset.collapsed === 'true')
                    return true;
                element = element.parentNode;
            }
            return false;
        }

        // (for interaction with accordion plugin)
        // expand heading containing element if necesary
        function expandElement(element) {
            if (isCollapsed(element)) {
                const heading = firstBefore(element, 'h2');
                if (heading)
                    heading.click();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
    <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
    <svg width="16" height="16" viewBox="0 0 320 512">
        <path
            fill="currentColor"
            d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
        ></path>
    </svg>
</template>
<!-- link highlight plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user hovers or
        // focuses a link, other links that have the same target will be
        // highlighted. It also makes it such that when clicking a link, the
        // target of the link (eg reference, figure, table) is briefly
        // highlighted.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'linkHighlight';

        // default plugin options
        const options = {
            // whether to also highlight links that go to external urls
            externalLinks: 'false',
            // whether user must click off to unhighlight instead of just
            // un-hovering
            clickUnhighlight: 'false',
            // whether to also highlight links that are unique
            highlightUnique: 'true',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            const links = getLinks();
            for (const link of links) {
                // attach mouse and focus listeners to link
                link.addEventListener('mouseenter', onLinkFocus);
                link.addEventListener('focus', onLinkFocus);
                link.addEventListener('mouseleave', onLinkUnhover);
            }

            // attach click and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('hashchange', onHashChange);

            // run hash change on window load in case user has navigated
            // directly to hash
            onHashChange();
        }

        // when link is focused (tabbed to) or hovered
        function onLinkFocus() {
            highlight(this);
        }

        // when link is unhovered
        function onLinkUnhover() {
            if (options.clickUnhighlight !== 'true')
                unhighlightAll();
        }

        // when the mouse is clicked anywhere in window
        function onClick(event) {
            unhighlightAll();
        }

        // when hash (eg manuscript.html#introduction) changes
        function onHashChange() {
            const target = getHashTarget();
            if (target)
                glowElement(target);
        }

        // get element that is target of link or url hash
        function getHashTarget(link) {
            const hash = link ? link.hash : window.location.hash;
            const id = hash.slice(1);
            let target = document.querySelector(
                '[id="' + id + '"], [name="' + id + '"]'
            );
            if (!target)
                return;

            // if figure or table, modify target to get expected element
            if (hash.indexOf('#fig:') === 0)
                target = target.parentNode;
            else if (hash.indexOf('#tbl:') === 0)
                target = target.nextElementSibling.querySelector('caption');

            return target;
        }

        // start glow sequence on an element
        function glowElement(element) {
            const startGlow = function() {
                onGlowEnd();
                element.dataset.glow = 'true';
                element.addEventListener('animationend', onGlowEnd);
            };
            const onGlowEnd = function() {
                element.removeAttribute('data-glow');
                element.removeEventListener('animationend', onGlowEnd);
            };
            startGlow();
        }

        // highlight link and all others with same target
        function highlight(link) {
            // force unhighlight all to start fresh
            unhighlightAll();

            // get links with same target
            if (!link)
                return;
            const sameLinks = getSameLinks(link);

            // if link unique and option is off, exit and don't highlight
            if (sameLinks.length <= 1 && options.highlightUnique !== 'true')
                return;

            // highlight all same links, and "select" (special highlight) this
            // one
            for (const sameLink of sameLinks) {
                if (sameLink === link)
                    sameLink.setAttribute('data-selected', 'true');
                else
                    sameLink.setAttribute('data-highlighted', 'true');
            }
        }

        // unhighlight all links
        function unhighlightAll() {
            const links = getLinks();
            for (const link of links) {
                link.setAttribute('data-selected', 'false');
                link.setAttribute('data-highlighted', 'false');
            }
        }

        // get links with same target
        function getSameLinks(link) {
            const results = [];
            const links = getLinks();
            for (const otherLink of links) {
                if (
                    otherLink.getAttribute('href') === link.getAttribute('href')
                )
                    results.push(otherLink);
            }
            return results;
        }

        // get all links of types we wish to handle
        function getLinks() {
            let query = 'a';
            if (options.externalLinks !== 'true')
                query += '[href^="#"]';
            // exclude buttons, anchor links, toc links, etc
            query +=
                ':not(.button):not(.icon_button):not(.anchor):not(.toc_link)';
            return document.querySelectorAll(query);
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- table of contents plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin provides a "table of contents" (toc) panel on
        // the side of the document that allows the user to conveniently
        // navigate between sections of the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'tableOfContents';

        // default plugin options
        const options = {
            // which types of elements to add links for, in
            // "document.querySelector" format
            typesQuery: 'h1, h2, h3',
            // whether default behavior is to be closed ('false'), open
            // ('true'), or only open when screen wide enough to fit panel
            // ('auto'). note: still always starts closed when page loads.
            open: 'auto',
            // if list item is more than this many characters, text will be
            // truncated
            charLimit: '50',
            // whether or not to show bullets next to each toc item
            bullets: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // make toc panel and populate with entries (links to document
            // sections)
            const panel = makePanel();
            if (!panel)
                return;
            makeEntries(panel);
            document.body.insertBefore(panel, document.body.firstChild);

            closePanel();

            // attach click, scroll, and hash change listeners to window
            window.addEventListener('click', onClick);
            window.addEventListener('touchstart', onClick);
            window.addEventListener('scroll', onScroll);
            window.addEventListener('hashchange', onScroll);
            window.addEventListener('keyup', onKeyUp);
            onScroll();

            // add class to push document body down out of way of toc button
            document.body.classList.add('toc_body_nudge');
        }

        // determine if screen wide enough to fit toc panel
        function isSmallScreen() {
            // in default theme:
            // 816px = 8.5in = width of "page" (<body>) element
            // 260px = min width of toc panel (*2 for both sides of <body>)
            return window.innerWidth < 816 + 260 * 2;
        }

        // open/close panel based on option and screen size
        function openOrClosePanel() {
            if (
                options.open === 'true' ||
                (options.open === 'auto' && !isSmallScreen())
            )
                openPanel();
            else
                closePanel();
        }

        // when mouse is clicked anywhere in window
        function onClick() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                openOrClosePanel();
        }

        // when window is scrolled or hash changed
        function onScroll() {
            highlightViewed();
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            // close on esc
            if (event.key === 'Escape')
                closePanel();
        }

        // find entry of currently viewed document section in toc and highlight
        function highlightViewed() {
            const firstId = getFirstInView(options.typesQuery);

            // get toc entries (links), unhighlight all, then highlight viewed
            const list = document.getElementById('toc_list');
            if (!firstId || !list)
                return;
            const links = list.querySelectorAll('a');
            for (const link of links)
                link.dataset.viewing = 'false';
            const link = list.querySelector('a[href="#' + firstId + '"]');
            if (!link)
                return;
            link.dataset.viewing = 'true';
        }

        // get first or previous toc listed element in top half of view
        function getFirstInView(query) {
            // get all elements matching query and with id
            const elements = document.querySelectorAll(query);
            const elementsWithIds = [];
            for (const element of elements) {
                if (element.id)
                    elementsWithIds.push(element);
            }


            // get first or previous element in top half of view
            for (let i = 0; i < elementsWithIds.length; i++) {
                const element = elementsWithIds[i];
                const prevElement = elementsWithIds[Math.max(0, i - 1)];
                if (element.getBoundingClientRect().top >= 0) {
                    if (
                        element.getBoundingClientRect().top <
                        window.innerHeight / 2
                    )
                        return element.id;
                    else
                        return prevElement.id;
                }
            }
        }

        // make panel
        function makePanel() {
            // create panel
            const panel = document.createElement('div');
            panel.id = 'toc_panel';
            if (options.bullets === 'true')
                panel.dataset.bullets = 'true';

            // create header
            const header = document.createElement('div');
            header.id = 'toc_header';

            // create toc button
            const button = document.createElement('button');
            button.id = 'toc_button';
            button.innerHTML = document.querySelector(
                '.icon_th_list'
            ).innerHTML;
            button.classList.add('icon_button');

            // create header text
            const text = document.createElement('h3');
            text.innerHTML = 'Table of Contents';

            // create container for toc list
            const list = document.createElement('div');
            list.id = 'toc_list';

            // attach click listeners
            panel.addEventListener('click', onPanelClick);
            header.addEventListener('click', onHeaderClick);
            button.addEventListener('click', onButtonClick);

            // attach elements
            header.appendChild(button);
            header.appendChild(text);
            panel.appendChild(header);
            panel.appendChild(list);

            return panel;
        }

        // create toc entries (links) to each element of the specified types
        function makeEntries(panel) {
            const elements = document.querySelectorAll(options.typesQuery);
            for (const element of elements) {
                // do not add link if element doesn't have assigned id
                if (!element.id)
                    continue;

                // create link/list item
                const link = document.createElement('a');
                link.classList.add('toc_link');
                switch (element.tagName.toLowerCase()) {
                    case 'h1':
                        link.dataset.level = '1';
                        break;
                    case 'h2':
                        link.dataset.level = '2';
                        break;
                    case 'h3':
                        link.dataset.level = '3';
                        break;
                    case 'h4':
                        link.dataset.level = '4';
                        break;
                }
                link.title = element.innerText;
                let text = element.innerText;
                if (text.length > options.charLimit)
                    text = text.slice(0, options.charLimit) + '...';
                link.innerHTML = text;
                link.href = '#' + element.id;
                link.addEventListener('click', onLinkClick);

                // attach link
                panel.querySelector('#toc_list').appendChild(link);
            }
        }

        // when panel is clicked
        function onPanelClick(event) {
            // stop click from propagating to window/document and closing panel
            event.stopPropagation();
        }

        // when header itself is clicked
        function onHeaderClick(event) {
            togglePanel();
        }

        // when button is clicked
        function onButtonClick(event) {
            togglePanel();
            // stop header underneath button from also being clicked
            event.stopPropagation();
        }

        // when link is clicked
        function onLinkClick() {
            openOrClosePanel();
        }

        // open panel if closed, close if opened
        function togglePanel() {
            const panel = document.getElementById('toc_panel');
            if (!panel)
                return;

            if (panel.dataset.open === 'true')
                closePanel();
            else
                openPanel();
        }

        // open panel
        function openPanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'true';
        }

        // close panel
        function closePanel() {
            const panel = document.getElementById('toc_panel');
            if (panel)
                panel.dataset.open = 'false';
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- th list icon -->

<template class="icon_th_list">
    <!-- modified from: https://fontawesome.com/icons/th-list -->
    <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
        <path
            fill="currentColor"
            d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
            tabindex="-1"
        ></path>
    </svg>
</template>
<!-- lightbox plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin makes it such that when a user clicks on an
        // image, the image fills the screen and the user can pan/drag/zoom
        // the image and navigate between other images in the document.

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'lightbox';

        // default plugin options
        const options = {
            // list of possible zoom/scale factors
            zoomSteps:
                '0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1,' +
                '1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8',
            // whether to fit image to view ('fit'), display at 100% and shrink
            // if necessary ('shrink'), or always display at 100% ('100')
            defaultZoom: 'fit',
            // whether to zoom in/out toward center of view ('true') or mouse
            // ('false')
            centerZoom: 'false',
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // run through each <img> element
            const imgs = document.querySelectorAll('figure > img');
            let count = 1;
            for (const img of imgs) {
                img.classList.add('lightbox_document_img');
                img.dataset.number = count;
                img.dataset.total = imgs.length;
                img.addEventListener('click', openLightbox);
                count++;
            }

            // attach mouse and key listeners to window
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('keyup', onKeyUp);
        }

        // when mouse is moved anywhere in window
        function onWindowMouseMove(event) {
            window.mouseX = event.clientX;
            window.mouseY = event.clientY;
        }

        // when key pressed
        function onKeyUp(event) {
            if (!event || !event.key)
                return;

            switch (event.key) {
                // trigger click of prev button
                case 'ArrowLeft':
                    const prevButton = document.getElementById(
                        'lightbox_prev_button'
                    );
                    if (prevButton)
                        prevButton.click();
                    break;
                // trigger click of next button
                case 'ArrowRight':
                    const nextButton = document.getElementById(
                        'lightbox_next_button'
                    );
                    if (nextButton)
                        nextButton.click();
                    break;
                // close on esc
                case 'Escape':
                    closeLightbox();
                    break;
            }
        }

        // open lightbox
        function openLightbox() {
            const lightbox = makeLightbox(this);
            if (!lightbox)
                return;

            blurBody(lightbox);
            document.body.appendChild(lightbox);
        }

        // make lightbox
        function makeLightbox(img) {
            // delete lightbox if it exists, start fresh
            closeLightbox();

            // create screen overlay containing lightbox
            const overlay = document.createElement('div');
            overlay.id = 'lightbox_overlay';

            // create image info boxes
            const numberInfo = document.createElement('div');
            const zoomInfo = document.createElement('div');
            numberInfo.id = 'lightbox_number_info';
            zoomInfo.id = 'lightbox_zoom_info';

            // create container for image
            const imageContainer = document.createElement('div');
            imageContainer.id = 'lightbox_image_container';
            const lightboxImg = makeLightboxImg(
                img,
                imageContainer,
                numberInfo,
                zoomInfo
            );
            imageContainer.appendChild(lightboxImg);

            // create bottom container for caption and navigation buttons
            const bottomContainer = document.createElement('div');
            bottomContainer.id = 'lightbox_bottom_container';
            const caption = makeCaption(img);
            const prevButton = makePrevButton(img);
            const nextButton = makeNextButton(img);
            bottomContainer.appendChild(prevButton);
            bottomContainer.appendChild(caption);
            bottomContainer.appendChild(nextButton);

            // attach top middle and bottom to overlay
            overlay.appendChild(numberInfo);
            overlay.appendChild(zoomInfo);
            overlay.appendChild(imageContainer);
            overlay.appendChild(bottomContainer);

            return overlay;
        }

        // make <img> object that is intuitively draggable and zoomable
        function makeLightboxImg(
            sourceImg,
            container,
            numberInfoBox,
            zoomInfoBox
        ) {
            // create copy of source <img>
            const img = sourceImg.cloneNode(true);
            img.classList.remove('lightbox_document_img');
            img.removeAttribute('id');
            img.removeAttribute('width');
            img.removeAttribute('height');
            img.style.position = 'unset';
            img.style.margin = '0';
            img.style.padding = '0';
            img.style.width = '';
            img.style.height = '';
            img.style.minWidth = '';
            img.style.minHeight = '';
            img.style.maxWidth = '';
            img.style.maxHeight = '';
            img.id = 'lightbox_img';

            // build sorted list of unique zoomSteps, always including a 100%
            let zoomSteps = [];
            const optionsZooms = options.zoomSteps.split(/[^0-9.]/);
            for (const optionZoom of optionsZooms) {
                const newZoom = parseFloat(optionZoom);
                if (newZoom && !zoomSteps.includes(newZoom))
                    zoomSteps.push(newZoom);
            }
            if (!zoomSteps.includes(1))
                zoomSteps.push(1);
            zoomSteps = zoomSteps.sort(function sortNumber(a, b) {
                return a - b;
            });

            // <img> object property variables
            let zoom = 1;
            let translateX = 0;
            let translateY = 0;
            let clickMouseX = undefined;
            let clickMouseY = undefined;
            let clickTranslateX = undefined;
            let clickTranslateY = undefined;

            updateNumberInfo();

            // update image numbers displayed in info box
            function updateNumberInfo() {
                numberInfoBox.innerHTML =
                    sourceImg.dataset.number + ' of ' + sourceImg.dataset.total;
            }

            // update zoom displayed in info box
            function updateZoomInfo() {
                let zoomInfo = zoom * 100;
                if (!Number.isInteger(zoomInfo))
                    zoomInfo = zoomInfo.toFixed(2);
                zoomInfoBox.innerHTML = zoomInfo + '%';
            }

            // move to closest zoom step above current zoom
            const zoomIn = function() {
                for (const zoomStep of zoomSteps) {
                    if (zoomStep > zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                updateTransform();
            };

            // move to closest zoom step above current zoom
            const zoomOut = function() {
                zoomSteps.reverse();
                for (const zoomStep of zoomSteps) {
                    if (zoomStep < zoom) {
                        zoom = zoomStep;
                        break;
                    }
                }
                zoomSteps.reverse();

                updateTransform();
            };

            // update display of <img> based on scale/translate properties
            const updateTransform = function() {
                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                // get new width/height after scale
                const rect = img.getBoundingClientRect();
                // limit translate
                translateX = Math.max(translateX, -rect.width / 2);
                translateX = Math.min(translateX, rect.width / 2);
                translateY = Math.max(translateY, -rect.height / 2);
                translateY = Math.min(translateY, rect.height / 2);

                // set transform
                img.style.transform =
                    'translate(' +
                    (translateX || 0) +
                    'px,' +
                    (translateY || 0) +
                    'px) scale(' +
                    (zoom || 1) +
                    ')';

                updateZoomInfo();
            };

            // fit <img> to container
            const fit = function() {
                // no x/y offset, 100% zoom by default
                translateX = 0;
                translateY = 0;
                zoom = 1;

                // widths of <img> and container
                const imgWidth = img.naturalWidth;
                const imgHeight = img.naturalHeight;
                const containerWidth = parseFloat(
                    window.getComputedStyle(container).width
                );
                const containerHeight = parseFloat(
                    window.getComputedStyle(container).height
                );

                // how much zooming is needed to fit <img> to container
                const xRatio = imgWidth / containerWidth;
                const yRatio = imgHeight / containerHeight;
                const maxRatio = Math.max(xRatio, yRatio);
                const newZoom = 1 / maxRatio;

                // fit <img> to container according to option
                if (options.defaultZoom === 'shrink') {
                    if (maxRatio > 1)
                        zoom = newZoom;
                } else if (options.defaultZoom === 'fit')
                    zoom = newZoom;

                updateTransform();
            };

            // when mouse wheel is rolled anywhere in container
            const onContainerWheel = function(event) {
                if (!event)
                    return;

                // let ctrl + mouse wheel to zoom behave as normal
                if (event.ctrlKey)
                    return;

                // prevent normal scroll behavior
                event.preventDefault();
                event.stopPropagation();

                // point around which to scale img
                const viewRect = container.getBoundingClientRect();
                const viewX = (viewRect.left + viewRect.right) / 2;
                const viewY = (viewRect.top + viewRect.bottom) / 2;
                const originX = options.centerZoom === 'true' ? viewX : mouseX;
                const originY = options.centerZoom === 'true' ? viewY : mouseY;

                // get point on image under origin
                const oldRect = img.getBoundingClientRect();
                const oldPercentX = (originX - oldRect.left) / oldRect.width;
                const oldPercentY = (originY - oldRect.top) / oldRect.height;

                // increment/decrement zoom
                if (event.deltaY < 0)
                    zoomIn();
                if (event.deltaY > 0)
                    zoomOut();

                // get offset between previous image point and origin
                const newRect = img.getBoundingClientRect();
                const offsetX =
                    originX - (newRect.left + newRect.width * oldPercentX);
                const offsetY =
                    originY - (newRect.top + newRect.height * oldPercentY);

                // translate image to keep image point under origin
                translateX += offsetX;
                translateY += offsetY;

                // perform translate
                updateTransform();
            };

            // when container is clicked
            function onContainerClick(event) {
                // if container itself is target of click, and not other
                // element above it
                if (event.target === this)
                    closeLightbox();
            }

            // when mouse button is pressed on image
            const onImageMouseDown = function(event) {
                // store original mouse position relative to image
                clickMouseX = window.mouseX;
                clickMouseY = window.mouseY;
                clickTranslateX = translateX;
                clickTranslateY = translateY;
                event.stopPropagation();
                event.preventDefault();
            };

            // when mouse button is released anywhere in window
            const onWindowMouseUp = function(event) {
                // reset original mouse position
                clickMouseX = undefined;
                clickMouseY = undefined;
                clickTranslateX = undefined;
                clickTranslateY = undefined;

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mouseup', onWindowMouseUp);
            };

            // when mouse is moved anywhere in window
            const onWindowMouseMove = function(event) {
                if (
                    clickMouseX === undefined ||
                    clickMouseY === undefined ||
                    clickTranslateX === undefined ||
                    clickTranslateY === undefined
                )
                    return;

                // offset image based on original and current mouse position
                translateX = clickTranslateX + window.mouseX - clickMouseX;
                translateY = clickTranslateY + window.mouseY - clickMouseY;
                updateTransform();
                event.preventDefault();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('mousemove', onWindowMouseMove);
            };

            // when window is resized
            const onWindowResize = function(event) {
                fit();

                // remove global listener if lightbox removed from document
                if (!document.body.contains(container))
                    window.removeEventListener('resize', onWindowResize);
            };

            // attach the necessary event listeners
            img.addEventListener('dblclick', fit);
            img.addEventListener('mousedown', onImageMouseDown);
            container.addEventListener('wheel', onContainerWheel);
            container.addEventListener('mousedown', onContainerClick);
            container.addEventListener('touchstart', onContainerClick);
            window.addEventListener('mouseup', onWindowMouseUp);
            window.addEventListener('mousemove', onWindowMouseMove);
            window.addEventListener('resize', onWindowResize);

            // run fit() after lightbox atttached to document and <img> Loaded
            // so needed container and img dimensions available
            img.addEventListener('load', fit);

            return img;
        }

        // make caption
        function makeCaption(img) {
            const caption = document.createElement('div');
            caption.id = 'lightbox_caption';
            const captionSource = img.nextElementSibling;
            if (captionSource.tagName.toLowerCase() === 'figcaption') {
                const captionCopy = makeCopy(captionSource);
                caption.innerHTML = captionCopy.innerHTML;
            }

            caption.addEventListener('touchstart', function(event) {
                event.stopPropagation();
            });

            return caption;
        }

        // make carbon copy of html dom element
        function makeCopy(source) {
            const sourceCopy = source.cloneNode(true);

            // delete elements marked with ignore (eg anchor and jump buttons)
            const deleteFromCopy = sourceCopy.querySelectorAll(
                '[data-ignore="true"]'
            );
            for (const element of deleteFromCopy)
                element.remove();

            // delete certain element attributes
            const attributes = [
                'id',
                'data-collapsed',
                'data-selected',
                'data-highlighted',
                'data-glow'
            ];
            for (const attribute of attributes) {
                sourceCopy.removeAttribute(attribute);
                const elements = sourceCopy.querySelectorAll(
                    '[' + attribute + ']'
                );
                for (const element of elements)
                    element.removeAttribute(attribute);
            }

            return sourceCopy;
        }

        // make button to jump to previous image in document
        function makePrevButton(img) {
            const prevButton = document.createElement('button');
            prevButton.id = 'lightbox_prev_button';
            prevButton.title = 'Jump to the previous image in the document [←]';
            prevButton.classList.add('icon_button', 'lightbox_button');
            prevButton.innerHTML = document.querySelector(
                '.icon_caret_left'
            ).innerHTML;

            // attach click listeners to button
            prevButton.addEventListener('click', function() {
                getPrevImg(img).click();
            });

            return prevButton;
        }

        // make button to jump to next image in document
        function makeNextButton(img) {
            const nextButton = document.createElement('button');
            nextButton.id = 'lightbox_next_button';
            nextButton.title = 'Jump to the next image in the document [→]';
            nextButton.classList.add('icon_button', 'lightbox_button');
            nextButton.innerHTML = document.querySelector(
                '.icon_caret_right'
            ).innerHTML;

            // attach click listeners to button
            nextButton.addEventListener('click', function() {
                getNextImg(img).click();
            });

            return nextButton;
        }

        // get previous image in document
        function getPrevImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if < 1
            if (index - 1 >= 0)
                index--;
            else
                index = imgs.length - 1;
            return imgs[index];
        }

        // get next image in document
        function getNextImg(img) {
            const imgs = document.querySelectorAll('.lightbox_document_img');

            // find index of provided img
            let index;
            for (index = 0; index < imgs.length; index++) {
                if (imgs[index] === img)
                    break;
            }


            // wrap index to other side if > total
            if (index + 1 <= imgs.length - 1)
                index++;
            else
                index = 0;
            return imgs[index];
        }

        // close lightbox
        function closeLightbox() {
            focusBody();

            const lightbox = document.getElementById('lightbox_overlay');
            if (lightbox)
                lightbox.remove();
        }

        // make all elements behind lightbox non-focusable
        function blurBody(overlay) {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.tabIndex = -1;
            document.body.classList.add('body_no_scroll');
        }

        // make all elements focusable again
        function focusBody() {
            const all = document.querySelectorAll('*');
            for (const element of all)
                element.removeAttribute('tabIndex');
            document.body.classList.remove('body_no_scroll');
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
    <!-- modified from: https://fontawesome.com/icons/caret-left -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
        ></path>
    </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
    <!-- modified from: https://fontawesome.com/icons/caret-right -->
    <svg width="16" height="16" viewBox="0 0 192 512">
        <path
            fill="currentColor"
            d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
        ></path>
    </svg>
</template>
<!-- attributes plugin -->

<script>
    (function() {
        // /////////////////////////
        // DESCRIPTION
        // /////////////////////////

        // This Manubot plugin allows arbitrary HTML attributes to be attached
        // to (almost) any element. Place an HTML comment inside or next to the
        // desired element in the format <!-- attribute="value" -->

        // /////////////////////////
        // OPTIONS
        // /////////////////////////

        // plugin name prefix for url parameters
        const pluginName = 'attributes';

        // default plugin options
        const options = {
            // whether plugin is on or not
            enabled: 'true'
        };

        // change options above, or override with url parameter, eg:
        // 'manuscript.html?pluginName-enabled=false'

        // /////////////////////////
        // SCRIPT
        // /////////////////////////

        // start script
        function start() {
            // get list of comments in document
            const comments = findComments();

            for(const comment of comments)
                if (comment.parentElement)
                    addAttributes(
                        comment.parentElement,
                        comment.nodeValue.trim()
                    );
        }

        // add html attributes to specified element based on string of 
        // html attributes and values
        function addAttributes(element, text) {
            // regex's for finding attribute/value pairs in the format of
            // attribute="value" or attribute='value
            const regex2 = /([a-zA-Z\-]+)?=\"(.+?)\"/;
            const regex1 = /([a-zA-Z\-]+)?=\'(.+?)\'/;

            // loop through attribute/value pairs
            let match;
            let count = 0;
            while(
                match = text.match(regex2) || text.match(regex1),
                count++,
                match && count < 100
            ) {
                // get attribute and value from regex capture groups
                let attribute = match[1];
                let value = match[2];

                // remove from string
                text = text.substring(match.index + match[0].length);

                if (!attribute || !value)
                    break;

                // set attribute of parent element
                try {
                    element.setAttribute(attribute, value);
                } catch(error) {
                    console.log(error);
                }

                // special case for colspan
                if (attribute === 'colspan')
                    removeTableCells(element, value);
            }
        }

        // get list of comment elements in document
        function findComments() {
            const comments = [];

            // iterate over comment nodes in document
            function acceptNode(node) {
                return NodeFilter.FILTER_ACCEPT;
            }
            const iterator = document.createNodeIterator(
                document.body,
                NodeFilter.SHOW_COMMENT,
                acceptNode
            );
            let node;
            while(node = iterator.nextNode())
                comments.push(node);

            return comments;
        }

        // remove certain number of cells after specified cell
        function removeTableCells(cell, number) {
            number = parseInt(number);
            if (!number)
                return;

            // remove elements
            for(; number > 1; number--) {
                if (cell.nextElementSibling)
                    cell.nextElementSibling.remove();
            }
        }

        // load options from url parameters
        function loadOptions() {
            const url = window.location.search;
            const params = new URLSearchParams(url);
            for (const optionName of Object.keys(options)) {
                const paramName = pluginName + '-' + optionName;
                const param = params.get(paramName);
                if (param !== '' && param !== null)
                    options[optionName] = param;
            }
        }
        loadOptions();

        // start script when document is finished loading
        if (options.enabled === 'true')
            window.addEventListener('load', start);
    })();
</script>
<!-- math plugin configuration -->

<script type="text/x-mathjax-config">
    MathJax.Ajax.config.path["mhchem"] =
  "https://cdnjs.cloudflare.com/ajax/libs/mathjax-mhchem/3.3.2";
    MathJax.Hub.Config({
        "CommonHTML": { linebreaks: { automatic: true } },
        "HTML-CSS": { linebreaks: { automatic: true } },
        "SVG": { linebreaks: { automatic: true } },
        TeX: {
            extensions: ["[mhchem]/mhchem.js"]
          }        
    });
</script>

<!-- math plugin -->

<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML'>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'MathJax' allows the proper rendering of
    // math/equations written in LaTeX.

    // https://www.mathjax.org/
</script>
<!-- annotations plugin configuration -->

<script>
    window.hypothesisConfig = function() {
        return {
            branding: {
                accentColor: '#2196f3',
                appBackgroundColor: '#f8f8f8',
                ctaBackgroundColor: '#f8f8f8',
                ctaTextColor: '#000000',
                selectionFontFamily: 'Open Sans, Helvetica, sans serif',
                annotationFontFamily: 'Open Sans, Helvetica, sans serif'
            }
        };
    };
</script>

<!-- annotations plugin -->

<script src='https://hypothes.is/embed.js'>
    // /////////////////////////
    // DESCRIPTION
    // /////////////////////////

    // This third-party plugin 'Hypothesis' allows public annotation of the
    // manuscript.

    // https://web.hypothes.is/
</script>
<!-- analytics plugin -->

<!-- copy and paste code from Google Analytics or similar service here -->
</body>
</html>
